src/vcpkg/postbuildlint.cpp

```diff
@@ -851,6 +851,30 @@ namespace vcpkg::PostBuildLint
         return LintStatus::SUCCESS;
     }
 
+    static LintStatus check_no_absolute_paths_to(const Filesystem& fs, const Path& dir)
+    {
+        std::vector<Path> text_files = fs.get_regular_files_recursive(dir, IgnoreErrors{});
+        Util::erase_remove_if(text_files,
```

Comment:
What if there's an absolute path in a binary that _isn't_ a DLL or an SO?

Reply:
Than this file would be reported. The problem that there is no way to differentiate a binary file from a text file.

Reply:
that seems unreasonable. Can we only check in files with _specific_ extensions?

Reply:
We can do that. What should we check?
- .pc
- .h
- .cpp
- .conf
- config in filename? (from https://github.com/microsoft/vcpkg/issues/20321)

Reply:
Or we use the `file` command

Reply:
I think `.pc`, `.h`, `.cpp`, `.conf`, `.txt`, `.cmake`, and ends with `-config` are probably a good start?

Reply:
I have run `vcpkg ci` and done a search with the following result:
<details><summary></summary>

```
1122 results - 701 files

vcpkg_installed/vcpkg/info/glib_2.66.4_x64-osx.list:
  333  x64-osx/share/gdb/auto-load/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/
  334: x64-osx/share/gdb/auto-load/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/
  335: x64-osx/share/gdb/auto-load/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/glib_x64-osx/
  336: x64-osx/share/gdb/auto-load/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/glib_x64-osx/lib/
  337: x64-osx/share/gdb/auto-load/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/glib_x64-osx/lib/libglib-2.0.so.0.6600.4-gdb.py
  338: x64-osx/share/gdb/auto-load/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/glib_x64-osx/lib/libgobject-2.0.so.0.6600.4-gdb.py
  339  x64-osx/share/gettext/

vcpkg_installed/x64-osx/bin/hpxcxx:
  37  pkgconfpath += [
  38:     os.path.join("/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/hpx_x64-osx","lib","pkgconfig"),     # install directory
  39:     os.path.join("/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/hpx_x64-osx","lib64","pkgconfig"),     # install directory
  40      os.path.join(os.path.dirname(sys.argv[0]),"..","lib","pkgconfig"),

vcpkg_installed/x64-osx/debug/bin/hpxcxx:
  37  pkgconfpath += [
  38:     os.path.join("/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/hpx_x64-osx/debug","lib","pkgconfig"),     # install directory
  39:     os.path.join("/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/hpx_x64-osx/debug","lib64","pkgconfig"),     # install directory
  40      os.path.join(os.path.dirname(sys.argv[0]),"..","lib","pkgconfig"),

vcpkg_installed/x64-osx/debug/gdk-pixbuf-2.0/2.10.0/loaders.cache:
  4  #
  5: # LoaderDir = /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/gdk-pixbuf_x64-osx/debug/lib/gdk-pixbuf-2.0/2.10.0/loaders
  6  #

vcpkg_installed/x64-osx/debug/lib/botan-2.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/botan_x64-osx/debug
  2  exec_prefix=${prefix}
  3: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/botan_x64-osx/debug/lib
  4  includedir=${prefix}/include/botan-2

vcpkg_installed/x64-osx/debug/lib/libgraphite2.la:
  37  # Directory that this library needs to be installed in:
  38: libdir='/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/graphite2_x64-osx/debug/lib'
  39  

vcpkg_installed/x64-osx/debug/lib/libhdf5.settings:
  11                         Byte sex: little-endian
  12:              Installation point: /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/hdf5_x64-osx/debug
  13  

vcpkg_installed/x64-osx/debug/lib/libnetcdf.settings:
  10  Build Directory: 	/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/netcdf-c/x64-osx-dbg
  11: Install Prefix:         /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/netcdf-c_x64-osx/debug
  12  

vcpkg_installed/x64-osx/debug/lib/glibmm-2.68/proc/gmmproc:
  42    # get prefix info from configure
  43:   my $prefix = "/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/glibmm_x64-osx/debug";
  44    my $exec_prefix = "${prefix}";

vcpkg_installed/x64-osx/debug/lib/manual-link/pkgconfig/gmock_main.pc:
  1: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/gtest_x64-osx/debug/lib
  2: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/gtest_x64-osx/debug/include
  3  

vcpkg_installed/x64-osx/debug/lib/manual-link/pkgconfig/gtest_main.pc:
  1: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/gtest_x64-osx/debug/lib
  2: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/gtest_x64-osx/debug/include
  3  

vcpkg_installed/x64-osx/debug/lib/pkgconfig/ACE_ETCL_Parser.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/ace_x64-osx
  2: exec_prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/ace_x64-osx
  3: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/ace_x64-osx/debug/lib
  4: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/ace_x64-osx/include
  5  

vcpkg_installed/x64-osx/debug/lib/pkgconfig/ACE_ETCL.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/ace_x64-osx
  2: exec_prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/ace_x64-osx
  3: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/ace_x64-osx/debug/lib
  4: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/ace_x64-osx/include
  5  

vcpkg_installed/x64-osx/debug/lib/pkgconfig/ACE.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/ace_x64-osx
  2: exec_prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/ace_x64-osx
  3: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/ace_x64-osx/debug/lib
  4: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/ace_x64-osx/include
  5  

vcpkg_installed/x64-osx/debug/lib/pkgconfig/amqpcpp.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/amqpcpp_x64-osx/debug
  2  libdir=${prefix}/lib

vcpkg_installed/x64-osx/debug/lib/pkgconfig/aom.pc:
  1  # libaom pkg-config.
  2: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/aom_x64-osx/debug
  3  exec_prefix=${prefix}

vcpkg_installed/x64-osx/debug/lib/pkgconfig/armadillo.pc:
  1: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/armadillo_x64-osx/debug/lib
  2: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/armadillo_x64-osx/debug/include
  3  

vcpkg_installed/x64-osx/debug/lib/pkgconfig/arrow-csv.pc:
  17  
  18: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/arrow_x64-osx/debug/lib
  19: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/arrow_x64-osx/debug/include
  20  

vcpkg_installed/x64-osx/debug/lib/pkgconfig/arrow-filesystem.pc:
  17  
  18: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/arrow_x64-osx/debug/lib
  19: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/arrow_x64-osx/debug/include
  20  

vcpkg_installed/x64-osx/debug/lib/pkgconfig/arrow-json.pc:
  17  
  18: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/arrow_x64-osx/debug/lib
  19: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/arrow_x64-osx/debug/include
  20  

vcpkg_installed/x64-osx/debug/lib/pkgconfig/arrow.pc:
  17  
  18: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/arrow_x64-osx/debug/lib
  19: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/arrow_x64-osx/debug/include
  20  

vcpkg_installed/x64-osx/debug/lib/pkgconfig/bal.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/bde_x64-osx/debug
  2  libdir=${prefix}/lib

vcpkg_installed/x64-osx/debug/lib/pkgconfig/bbl.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/bde_x64-osx/debug
  2  libdir=${prefix}/lib

vcpkg_installed/x64-osx/debug/lib/pkgconfig/bdl.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/bde_x64-osx/debug
  2  libdir=${prefix}/lib

vcpkg_installed/x64-osx/debug/lib/pkgconfig/bigtable_client.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/blitz-static.pc:
  1  # generated by cmake. remove this line to disable regeneration
  2: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/blitz_x64-osx/debug
  3  exec_prefix=${prefix}

vcpkg_installed/x64-osx/debug/lib/pkgconfig/blitz.pc:
  1  # generated by cmake. remove this line to disable regeneration
  2: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/blitz_x64-osx/debug
  3  exec_prefix=${prefix}

vcpkg_installed/x64-osx/debug/lib/pkgconfig/blosc.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/blosc_x64-osx/debug
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/debug/lib/pkgconfig/brpc.pc:
  15  
  16: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/brpc_x64-osx/debug
  17  exec_prefix=${prefix}

vcpkg_installed/x64-osx/debug/lib/pkgconfig/bsl.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/bde_x64-osx/debug
  2  libdir=${prefix}/lib

vcpkg_installed/x64-osx/debug/lib/pkgconfig/btl.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/bde_x64-osx/debug
  2  libdir=${prefix}/lib

vcpkg_installed/x64-osx/debug/lib/pkgconfig/bullet.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/bullet3_x64-osx/debug
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/debug/lib/pkgconfig/capnp-json.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/capnproto_x64-osx/debug
  2  exec_prefix=
  3: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/capnproto_x64-osx/debug/lib
  4: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/capnproto_x64-osx/debug/include
  5  

vcpkg_installed/x64-osx/debug/lib/pkgconfig/capnp-rpc.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/capnproto_x64-osx/debug
  2  exec_prefix=
  3: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/capnproto_x64-osx/debug/lib
  4: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/capnproto_x64-osx/debug/include
  5  

vcpkg_installed/x64-osx/debug/lib/pkgconfig/capnp.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/capnproto_x64-osx/debug
  2  exec_prefix=
  3: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/capnproto_x64-osx/debug/lib
  4: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/capnproto_x64-osx/debug/include
  5  

vcpkg_installed/x64-osx/debug/lib/pkgconfig/ccd.pc:
  2  
  3: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/ccd_x64-osx/debug
  4  exec_prefix=${prefix}

vcpkg_installed/x64-osx/debug/lib/pkgconfig/charls.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/charls_x64-osx/debug
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/debug/lib/pkgconfig/check.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/check_x64-osx/debug
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/debug/lib/pkgconfig/clblast.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/clblast_x64-osx/debug
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/debug/lib/pkgconfig/cmocka.pc:
  3  Version: 1.1.5
  4: Libs: -L/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/cmocka_x64-osx/debug/lib -lcmocka
  5: Cflags: -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/cmocka_x64-osx/debug/include

vcpkg_installed/x64-osx/debug/lib/pkgconfig/Coin.pc:
   1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/coin_x64-osx/debug
   2  exec_prefix=${prefix}
   3: bindir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/coin_x64-osx/debug/bin
   4: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/coin_x64-osx/debug/lib
   5: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/coin_x64-osx/debug/include
   6: datarootdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/coin_x64-osx/debug/share
   7: datadir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/coin_x64-osx/debug/share
   8: docdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/coin_x64-osx/debug/share/doc/Coin
   9: infodir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/coin_x64-osx/debug/share/info
  10: mandir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/coin_x64-osx/debug/share/man
  11  htmldir=${docdir}/html

vcpkg_installed/x64-osx/debug/lib/pkgconfig/collada-dom-141.pc:
  2  
  3: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/collada-dom_x64-osx/debug
  4: exec_prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/collada-dom_x64-osx/debug/bin
  5: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/collada-dom_x64-osx/debug/lib
  6: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/collada-dom_x64-osx/debug/include/collada-dom2.5
  7  

vcpkg_installed/x64-osx/debug/lib/pkgconfig/collada-dom-150.pc:
  2  
  3: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/collada-dom_x64-osx/debug
  4: exec_prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/collada-dom_x64-osx/debug/bin
  5: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/collada-dom_x64-osx/debug/lib
  6: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/collada-dom_x64-osx/debug/include/collada-dom2.5
  7  

vcpkg_installed/x64-osx/debug/lib/pkgconfig/collada-dom.pc:
  2  
  3: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/collada-dom_x64-osx/debug
  4: exec_prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/collada-dom_x64-osx/debug/bin
  5: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/collada-dom_x64-osx/debug/lib
  6: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/collada-dom_x64-osx/debug/include/collada-dom2.5
  7  

vcpkg_installed/x64-osx/debug/lib/pkgconfig/cpp_redis.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/cpp-redis_x64-osx/debug
  2  libdir=${prefix}/lib

vcpkg_installed/x64-osx/debug/lib/pkgconfig/csv2.pc:
  1: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/p-ranav-csv2_x64-osx/debug/include
  2  

vcpkg_installed/x64-osx/debug/lib/pkgconfig/curlpp.pc:
  1  # This is a comment
  2: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/curlpp_x64-osx/debug
  3: exec_prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/curlpp_x64-osx/debug
  4  includedir=include

vcpkg_installed/x64-osx/debug/lib/pkgconfig/Cutelyst2Qt5Authentication.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/cutelyst2_x64-osx/debug
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/debug/lib/pkgconfig/Cutelyst2Qt5Core.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/cutelyst2_x64-osx/debug
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/debug/lib/pkgconfig/Cutelyst2Qt5Session.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/cutelyst2_x64-osx/debug
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/debug/lib/pkgconfig/Cutelyst2Qt5StaticSimple.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/cutelyst2_x64-osx/debug
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/debug/lib/pkgconfig/Cutelyst2Qt5StatusMessage.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/cutelyst2_x64-osx/debug
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/debug/lib/pkgconfig/Cutelyst2Qt5UserAgent.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/cutelyst2_x64-osx/debug
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/debug/lib/pkgconfig/Cutelyst2Qt5UtilsLangSelect.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/cutelyst2_x64-osx/debug
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/debug/lib/pkgconfig/Cutelyst2Qt5UtilsPagination.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/cutelyst2_x64-osx/debug
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/debug/lib/pkgconfig/Cutelyst2Qt5UtilsSql.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/cutelyst2_x64-osx/debug
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/debug/lib/pkgconfig/Cutelyst2Qt5UtilsValidator.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/cutelyst2_x64-osx/debug
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/debug/lib/pkgconfig/Cutelyst2Qt5ViewJSON.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/cutelyst2_x64-osx/debug
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/debug/lib/pkgconfig/Cutelyst2Qt5WSGI.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/cutelyst2_x64-osx/debug
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/debug/lib/pkgconfig/decnumber.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/bde_x64-osx/debug
  2  libdir=${prefix}/lib

vcpkg_installed/x64-osx/debug/lib/pkgconfig/dlib-1.pc:
  1: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/dlib_x64-osx/debug/lib
  2: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/dlib_x64-osx/debug/include
  3  

vcpkg_installed/x64-osx/debug/lib/pkgconfig/edlib-1.pc:
  1: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/edlib_x64-osx/debug/lib
  2: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/edlib_x64-osx/debug/include
  3  

vcpkg_installed/x64-osx/debug/lib/pkgconfig/evhtp.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libevhtp_x64-osx/debug
  2  libdir=

vcpkg_installed/x64-osx/debug/lib/pkgconfig/fcl.pc:
  1  # This file was generated by CMake for fcl
  2: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/fcl_x64-osx/debug
  3  exec_prefix=${prefix}
  4: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/fcl_x64-osx/debug/lib
  5: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/fcl_x64-osx/debug/include
  6  

vcpkg_installed/x64-osx/debug/lib/pkgconfig/fftw3.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/fftw3_x64-osx/debug
  2: exec_prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/fftw3_x64-osx/debug
  3: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/fftw3_x64-osx/debug/lib
  4: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/fftw3_x64-osx/debug/include
  5  

vcpkg_installed/x64-osx/debug/lib/pkgconfig/fftw3f.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/fftw3_x64-osx/debug
  2: exec_prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/fftw3_x64-osx/debug
  3: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/fftw3_x64-osx/debug/lib
  4: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/fftw3_x64-osx/debug/include
  5  

vcpkg_installed/x64-osx/debug/lib/pkgconfig/fftw3l.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/fftw3_x64-osx/debug
  2: exec_prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/fftw3_x64-osx/debug
  3: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/fftw3_x64-osx/debug/lib
  4: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/fftw3_x64-osx/debug/include
  5  

vcpkg_installed/x64-osx/debug/lib/pkgconfig/flann.pc:
  1  # This file was generated by CMake for flann
  2: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/flann_x64-osx/debug
  3  exec_prefix=${prefix}

vcpkg_installed/x64-osx/debug/lib/pkgconfig/flatbuffers.pc:
  1: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/flatbuffers_x64-osx/debug/lib
  2: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/flatbuffers_x64-osx/debug/include
  3  

vcpkg_installed/x64-osx/debug/lib/pkgconfig/fluidlite.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/fluidlite_x64-osx/debug
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/debug/lib/pkgconfig/functions_framework_cpp.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/functions-framework-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/gdlib.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libgd_x64-osx/debug
  2: exec_prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libgd_x64-osx/debug
  3  libdir=${exec_prefix}/lib

vcpkg_installed/x64-osx/debug/lib/pkgconfig/geogram1.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/geogram_x64-osx/debug
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/debug/lib/pkgconfig/geographiclib.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/geographiclib_x64-osx/debug
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/debug/lib/pkgconfig/gflags.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/gflags_x64-osx/debug
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/debug/lib/pkgconfig/glut.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/freeglut_x64-osx/debug
  2: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/freeglut_x64-osx/debug/lib
  3  includedir=${prefix}/include

vcpkg_installed/x64-osx/debug/lib/pkgconfig/gmock.pc:
  1: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/gtest_x64-osx/debug/lib
  2: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/gtest_x64-osx/debug/include
  3  

vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_api_annotations_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_api_auth_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_api_client_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_api_distribution_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_api_field_behavior_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_api_http_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_api_label_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_api_launch_stage_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_api_metric_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_api_monitored_resource_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_api_resource_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_bigquery.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_bigtable_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_bigtable.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_cloud_bigquery_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_cloud_dialogflow_v2_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_cloud_dialogflow_v2beta1_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_cloud_speech_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_cloud_texttospeech_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_common.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_devtools_cloudtrace_v2_trace_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_devtools_cloudtrace_v2_tracing_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_grpc_utils.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_iam_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_iam_v1_iam_policy_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_iam_v1_options_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_iam_v1_policy_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_iam.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_logging_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_logging_type_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_longrunning_operations_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_monitoring_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_pubsub_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_pubsub.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_rpc_error_details_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_rpc_status_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_spanner_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_spanner.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_storage_grpc.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_storage_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_storage.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_type_calendar_period_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_type_color_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_type_date_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_type_datetime_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_type_dayofweek_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_type_expr_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_type_fraction_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_type_interval_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_type_latlng_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_type_localized_text_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_type_money_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_type_month_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_type_phone_number_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_type_postal_address_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_type_quaternion_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_type_timeofday_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/googleapis_cpp_api_annotations_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/googleapis_cpp_api_auth_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/googleapis_cpp_api_client_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/googleapis_cpp_api_distribution_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/googleapis_cpp_api_field_behavior_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/googleapis_cpp_api_http_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/googleapis_cpp_api_label_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/googleapis_cpp_api_launch_stage_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/googleapis_cpp_api_metric_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/googleapis_cpp_api_monitored_resource_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/googleapis_cpp_api_resource_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/googleapis_cpp_bigtable_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/googleapis_cpp_cloud_bigquery_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/googleapis_cpp_cloud_dialogflow_v2_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/googleapis_cpp_cloud_dialogflow_v2beta1_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/googleapis_cpp_cloud_speech_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/googleapis_cpp_cloud_texttospeech_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/googleapis_cpp_devtools_cloudtrace_v2_trace_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/googleapis_cpp_devtools_cloudtrace_v2_tracing_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/googleapis_cpp_iam_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/googleapis_cpp_iam_v1_iam_policy_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/googleapis_cpp_iam_v1_options_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/googleapis_cpp_iam_v1_policy_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/googleapis_cpp_logging_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/googleapis_cpp_logging_type_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/googleapis_cpp_longrunning_operations_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/googleapis_cpp_monitoring_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/googleapis_cpp_pubsub_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/googleapis_cpp_rpc_error_details_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/googleapis_cpp_rpc_status_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/googleapis_cpp_spanner_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/googleapis_cpp_storage_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/googleapis_cpp_type_calendar_period_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/googleapis_cpp_type_color_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/googleapis_cpp_type_date_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/googleapis_cpp_type_datetime_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/googleapis_cpp_type_dayofweek_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/googleapis_cpp_type_expr_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/googleapis_cpp_type_fraction_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/googleapis_cpp_type_interval_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/googleapis_cpp_type_latlng_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/googleapis_cpp_type_localized_text_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/googleapis_cpp_type_money_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/googleapis_cpp_type_month_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/googleapis_cpp_type_phone_number_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/googleapis_cpp_type_postal_address_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/googleapis_cpp_type_quaternion_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/googleapis_cpp_type_timeofday_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/googleapis.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/gpr.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/grpc_x64-osx/debug
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/debug/lib/pkgconfig/graphite2.pc:
  4  Version: 3.0.1
  5: Libs: -L/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/graphite2_x64-osx/debug/lib -lgraphite2
  6: Cflags: -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/graphite2_x64-osx/debug/include
  7  

vcpkg_installed/x64-osx/debug/lib/pkgconfig/grpc_unsecure.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/grpc_x64-osx/debug
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/debug/lib/pkgconfig/grpc.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/grpc_x64-osx/debug
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/debug/lib/pkgconfig/grpc++_unsecure.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/grpc_x64-osx/debug
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/debug/lib/pkgconfig/grpc++.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/grpc_x64-osx/debug
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/debug/lib/pkgconfig/gta.pc:
  7  
  8: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libgta_x64-osx/debug
  9  exec_prefix=${prefix}

vcpkg_installed/x64-osx/debug/lib/pkgconfig/gtest.pc:
  1: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/gtest_x64-osx/debug/lib
  2: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/gtest_x64-osx/debug/include
  3  

vcpkg_installed/x64-osx/debug/lib/pkgconfig/haisrt.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libsrt_x64-osx/debug
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/debug/lib/pkgconfig/hiredis.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/hiredis_x64-osx/debug
  2  install_libdir=lib

vcpkg_installed/x64-osx/debug/lib/pkgconfig/igraph.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/igraph_x64-osx/debug
  2: exec_prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/igraph_x64-osx/debug
  3  libdir=${exec_prefix}/lib

vcpkg_installed/x64-osx/debug/lib/pkgconfig/iniparser.pc:
   3  libdir=/lib
   4: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/iniparser_x64-osx/debug/include
   5: datarootdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/iniparser_x64-osx/debug/share
   6: datadir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/iniparser_x64-osx/debug/share
   7  

  10  Version: 4.1
  11: Libs: -L/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/iniparser_x64-osx/debug/lib -liniparser
  12: Cflags: -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/iniparser_x64-osx/debug/include

vcpkg_installed/x64-osx/debug/lib/pkgconfig/inteldfp.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/bde_x64-osx/debug
  2  libdir=${prefix}/lib

vcpkg_installed/x64-osx/debug/lib/pkgconfig/itpp.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/itpp_x64-osx/debug
  2: exec_prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/itpp_x64-osx/debug
  3: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/itpp_x64-osx/debug/lib
  4: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/itpp_x64-osx/debug/include
  5  

vcpkg_installed/x64-osx/debug/lib/pkgconfig/jansson.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/jansson_x64-osx/debug
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/debug/lib/pkgconfig/json11.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/json11_x64-osx/debug
  2  libdir=${prefix}/lib/

vcpkg_installed/x64-osx/debug/lib/pkgconfig/kj-async.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/capnproto_x64-osx/debug
  2  exec_prefix=
  3: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/capnproto_x64-osx/debug/lib
  4: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/capnproto_x64-osx/debug/include
  5  

vcpkg_installed/x64-osx/debug/lib/pkgconfig/kj-http.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/capnproto_x64-osx/debug
  2  exec_prefix=
  3: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/capnproto_x64-osx/debug/lib
  4: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/capnproto_x64-osx/debug/include
  5  

vcpkg_installed/x64-osx/debug/lib/pkgconfig/kj-test.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/capnproto_x64-osx/debug
  2  exec_prefix=
  3: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/capnproto_x64-osx/debug/lib
  4: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/capnproto_x64-osx/debug/include
  5  

vcpkg_installed/x64-osx/debug/lib/pkgconfig/kj.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/capnproto_x64-osx/debug
  2  exec_prefix=
  3: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/capnproto_x64-osx/debug/lib
  4: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/capnproto_x64-osx/debug/include
  5  

vcpkg_installed/x64-osx/debug/lib/pkgconfig/lemon.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/liblemon_x64-osx/debug
  2: exec_prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/liblemon_x64-osx/debug/bin
  3: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/liblemon_x64-osx/debug/lib
  4: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/liblemon_x64-osx/debug/include
  5  

vcpkg_installed/x64-osx/debug/lib/pkgconfig/libbcg729.pc:
  1  # This is a comment
  2: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/bcg729_x64-osx/debug
  3  exec_prefix=${prefix}

vcpkg_installed/x64-osx/debug/lib/pkgconfig/libbddisasm.pc:
  1: dir_prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/bddisasm_x64-osx/debug
  2  lib=lib

vcpkg_installed/x64-osx/debug/lib/pkgconfig/libbson-1.0.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libbson_x64-osx/debug
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/debug/lib/pkgconfig/libbsoncxx-static.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/mongo-cxx-driver_x64-osx/debug
  16  includedir=${prefix}/include

vcpkg_installed/x64-osx/debug/lib/pkgconfig/libcerf.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libcerf_x64-osx/debug
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/debug/lib/pkgconfig/libcjson.pc:
  1: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/cjson_x64-osx/debug/lib
  2: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/cjson_x64-osx/debug/include
  3  

vcpkg_installed/x64-osx/debug/lib/pkgconfig/libconfuse.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libconfuse_x64-osx/debug
  2: exec_prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libconfuse_x64-osx/debug
  3: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libconfuse_x64-osx/debug/lib
  4: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libconfuse_x64-osx/debug/include
  5  

vcpkg_installed/x64-osx/debug/lib/pkgconfig/libcpuid.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/cpuid_x64-osx/debug
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/debug/lib/pkgconfig/libevent_core.pc:
  2  
  3: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libevent_x64-osx/debug
  4: exec_prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libevent_x64-osx/debug
  5: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libevent_x64-osx/debug/lib
  6: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libevent_x64-osx/debug/include
  7  

vcpkg_installed/x64-osx/debug/lib/pkgconfig/libevent_extra.pc:
  2  
  3: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libevent_x64-osx/debug
  4: exec_prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libevent_x64-osx/debug
  5: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libevent_x64-osx/debug/lib
  6: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libevent_x64-osx/debug/include
  7  

vcpkg_installed/x64-osx/debug/lib/pkgconfig/libevent_openssl.pc:
  2  
  3: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libevent_x64-osx/debug
  4: exec_prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libevent_x64-osx/debug
  5: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libevent_x64-osx/debug/lib
  6: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libevent_x64-osx/debug/include
  7  

vcpkg_installed/x64-osx/debug/lib/pkgconfig/libevent_pthreads.pc:
  2  
  3: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libevent_x64-osx/debug
  4: exec_prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libevent_x64-osx/debug
  5: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libevent_x64-osx/debug/lib
  6: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libevent_x64-osx/debug/include
  7  

vcpkg_installed/x64-osx/debug/lib/pkgconfig/libevent.pc:
  2  
  3: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libevent_x64-osx/debug
  4: exec_prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libevent_x64-osx/debug
  5: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libevent_x64-osx/debug/lib
  6: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libevent_x64-osx/debug/include
  7  

vcpkg_installed/x64-osx/debug/lib/pkgconfig/libfolly.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/folly_x64-osx/debug
  2: exec_prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/folly_x64-osx/debug
  3  libdir=${exec_prefix}/lib

vcpkg_installed/x64-osx/debug/lib/pkgconfig/libfort.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libfort_x64-osx/debug
  2: exec_prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libfort_x64-osx/debug
  3  includedir=${prefix}/include

vcpkg_installed/x64-osx/debug/lib/pkgconfig/libftdi1.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libftdi1_x64-osx/debug
  2: exec_prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libftdi1_x64-osx/debug/bin
  3: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libftdi1_x64-osx/debug/lib
  4: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libftdi1_x64-osx/debug/include/libftdi1
  5  

vcpkg_installed/x64-osx/debug/lib/pkgconfig/libftdipp1.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libftdi1_x64-osx/debug
  2: exec_prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libftdi1_x64-osx/debug/bin
  3: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libftdi1_x64-osx/debug/lib
  4: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libftdi1_x64-osx/debug/include/libftdi1
  5  

vcpkg_installed/x64-osx/debug/lib/pkgconfig/libgit2.pc:
  1: prefix="/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libgit2_x64-osx/debug"
  2: libdir="/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libgit2_x64-osx/debug/lib"
  3: includedir="/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libgit2_x64-osx/debug/include"
  4  

vcpkg_installed/x64-osx/debug/lib/pkgconfig/libgraphqlparser.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/graphqlparser_x64-osx/debug
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/debug/lib/pkgconfig/libheif.pc:
   1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libheif_x64-osx/debug
   2: exec_prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libheif_x64-osx/debug
   3: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libheif_x64-osx/debug/lib
   4: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libheif_x64-osx/debug/include
   5  builtin_h265_decoder=yes

  15  Requires.private: libde265 x265 aom
  16: Libs: -L/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libheif_x64-osx/debug/lib -lheif
  17  Libs.private:  -lstdc++
  18: Cflags: -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libheif_x64-osx/debug/include

vcpkg_installed/x64-osx/debug/lib/pkgconfig/libhs.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/hyperscan_x64-osx/debug
  2: exec_prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/hyperscan_x64-osx/debug
  3: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/hyperscan_x64-osx/debug/lib
  4: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/hyperscan_x64-osx/debug/include
  5  

vcpkg_installed/x64-osx/debug/lib/pkgconfig/libkeyfinder.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libkeyfinder_x64-osx/debug
  2  includedir=${prefix}/include

vcpkg_installed/x64-osx/debug/lib/pkgconfig/libKF5Attica.pc:
   1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/kf5attica_x64-osx/debug
   2  exec_prefix=${prefix}
   3: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/kf5attica_x64-osx/debug/lib
   4: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/kf5attica_x64-osx/debug/include
   5  

  10  Version: 5.84.0
  11: Libs: -L/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/kf5attica_x64-osx/debug/lib -lKF5Attica
  12: Cflags: -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/kf5attica_x64-osx/debug/include/KF5/Attica

vcpkg_installed/x64-osx/debug/lib/pkgconfig/libkml.pc:
  2  
  3: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libkml_x64-osx/debug
  4  exec_prefix=${prefix}

vcpkg_installed/x64-osx/debug/lib/pkgconfig/liblo.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/liblo_x64-osx/debug
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/debug/lib/pkgconfig/libmediainfo.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libmediainfo_x64-osx/debug
  2  exec_prefix=${prefix}
  3: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libmediainfo_x64-osx/debug/lib
  4: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libmediainfo_x64-osx/debug/include
  5  Libs_Static=${libdir}/libmediainfo.a ${libdir}/libzen.a -lpthread -lz -lcurl

vcpkg_installed/x64-osx/debug/lib/pkgconfig/libmikmod.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libmikmod_x64-osx/debug
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/debug/lib/pkgconfig/libmongoc-1.0.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/mongo-c-driver_x64-osx/debug
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/debug/lib/pkgconfig/libmongoc-ssl-1.0.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/mongo-c-driver_x64-osx/debug
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/debug/lib/pkgconfig/libmongocxx-static.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/mongo-cxx-driver_x64-osx/debug
  16  includedir=${prefix}/include

vcpkg_installed/x64-osx/debug/lib/pkgconfig/libmosquitto.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/mosquitto_x64-osx/debug
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/debug/lib/pkgconfig/libmosquittopp.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/mosquitto_x64-osx/debug
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/debug/lib/pkgconfig/libopcuaclient.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/freeopcua_x64-osx/debug
  2: exec_prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/freeopcua_x64-osx/debug
  3: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/freeopcua_x64-osx/debug/lib
  4: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/freeopcua_x64-osx/debug/include
  5  

vcpkg_installed/x64-osx/debug/lib/pkgconfig/libopcuacore.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/freeopcua_x64-osx/debug
  2: exec_prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/freeopcua_x64-osx/debug
  3: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/freeopcua_x64-osx/debug/lib
  4: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/freeopcua_x64-osx/debug/include
  5  

vcpkg_installed/x64-osx/debug/lib/pkgconfig/libopcuaprotocol.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/freeopcua_x64-osx/debug
  2: exec_prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/freeopcua_x64-osx/debug
  3: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/freeopcua_x64-osx/debug/lib
  4: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/freeopcua_x64-osx/debug/include
  5  

vcpkg_installed/x64-osx/debug/lib/pkgconfig/libopcuaserver.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/freeopcua_x64-osx/debug
  2: exec_prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/freeopcua_x64-osx/debug
  3: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/freeopcua_x64-osx/debug/lib
  4: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/freeopcua_x64-osx/debug/include
  5  

vcpkg_installed/x64-osx/debug/lib/pkgconfig/libproxy-1.0.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libproxy_x64-osx/debug
  2  libdir=${prefix}/lib

vcpkg_installed/x64-osx/debug/lib/pkgconfig/libqrencode.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libqrencode_x64-osx/debug
  2: exec_prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libqrencode_x64-osx/debug/bin
  3: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libqrencode_x64-osx/debug/lib
  4: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libqrencode_x64-osx/debug/include
  5  

vcpkg_installed/x64-osx/debug/lib/pkgconfig/libraw_r.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libraw_x64-osx/debug
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/debug/lib/pkgconfig/libraw.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libraw_x64-osx/debug
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/debug/lib/pkgconfig/libsais.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/sais_x64-osx/debug
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/debug/lib/pkgconfig/libsais64.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/sais_x64-osx/debug
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/debug/lib/pkgconfig/libsbml.pc:
  5  
  6: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libsbml_x64-osx/debug
  7: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libsbml_x64-osx/debug/lib
  8: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libsbml_x64-osx/debug/include
  9  

vcpkg_installed/x64-osx/debug/lib/pkgconfig/libtins.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libtins_x64-osx/debug
  2: exec_prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libtins_x64-osx/debug
  3: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libtins_x64-osx/debug/lib
  4  includedir=${prefix}/include

vcpkg_installed/x64-osx/debug/lib/pkgconfig/libtorrent-rasterbar.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libtorrent_x64-osx/debug
  2  libdir=${prefix}/lib

  7  Libs: -L${libdir} -ltorrent-rasterbar -L/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/x64-osx/debug/lib -l-framework CoreFoundation -l-framework SystemConfiguration -lIconv::Iconv -lssl -lcrypto -ldl -lcrypto -ldl
  8: Cflags: -fexceptions -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libtorrent_x64-osx/debug/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/x64-osx/include -DTORRENT_USE_ASSERTS -DBOOST_ASIO_ENABLE_CANCELIO -DTORRENT_NO_DEPRECATE -DTORRENT_USE_ICONV -DTORRENT_USE_OPENSSL -DTORRENT_USE_LIBCRYPTO

vcpkg_installed/x64-osx/debug/lib/pkgconfig/liburiparser.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/uriparser_x64-osx/debug
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/debug/lib/pkgconfig/libwebsockets_static.pc:
  1: prefix="/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libwebsockets_x64-osx/debug"
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/debug/lib/pkgconfig/libwebsockets.pc:
  1: prefix="/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libwebsockets_x64-osx/debug"
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/debug/lib/pkgconfig/libzmq.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/zeromq_x64-osx/debug
  2: exec_prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/zeromq_x64-osx/debug
  3: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/zeromq_x64-osx/debug/lib
  4: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/zeromq_x64-osx/debug/include
  5  

vcpkg_installed/x64-osx/debug/lib/pkgconfig/msgpack.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/msgpack_x64-osx/debug
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/debug/lib/pkgconfig/msgpack11.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/msgpack11_x64-osx/debug
  2  libdir=${prefix}/lib

vcpkg_installed/x64-osx/debug/lib/pkgconfig/mt32emu.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libmt32emu_x64-osx/debug
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/debug/lib/pkgconfig/muparser.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/muparser_x64-osx/debug
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/debug/lib/pkgconfig/nanoflann.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/nanoflann_x64-osx/debug
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/debug/lib/pkgconfig/nanomsg.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/nanomsg_x64-osx/debug
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/debug/lib/pkgconfig/nlopt.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/nlopt_x64-osx/debug
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/debug/lib/pkgconfig/octomap.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/octomap_x64-osx/debug
  2: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/octomap_x64-osx/debug/lib
  3: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/octomap_x64-osx/debug/include
  4  

vcpkg_installed/x64-osx/debug/lib/pkgconfig/ode.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/ode_x64-osx/debug
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/debug/lib/pkgconfig/OIS.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/ois_x64-osx/debug
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/debug/lib/pkgconfig/oniguruma.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/oniguruma_x64-osx/debug
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/debug/lib/pkgconfig/opencc.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/opencc_x64-osx/debug
  2  exec_prefix=${prefix}
  3: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/opencc_x64-osx/debug/lib/
  4: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/opencc_x64-osx/debug/include/
  5  

vcpkg_installed/x64-osx/debug/lib/pkgconfig/OpenImageIO.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/openimageio_x64-osx/debug
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/debug/lib/pkgconfig/orocos_kdl.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/orocos-kdl_x64-osx/debug
  2  libdir=${prefix}/lib

vcpkg_installed/x64-osx/debug/lib/pkgconfig/orocos-kdl.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/orocos-kdl_x64-osx/debug
  2  libdir=${prefix}/lib

vcpkg_installed/x64-osx/debug/lib/pkgconfig/parquet.pc:
  17  
  18: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/arrow_x64-osx/debug
  19  libdir=${prefix}/lib

vcpkg_installed/x64-osx/debug/lib/pkgconfig/pcl_2d-1.11.pc:
  1  # This file was generated by CMake for PCL library pcl_2d
  2: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/pcl_x64-osx/debug
  3  exec_prefix=${prefix}

vcpkg_installed/x64-osx/debug/lib/pkgconfig/pcl_common-1.11.pc:
  1  # This file was generated by CMake for PCL library pcl_common
  2: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/pcl_x64-osx/debug
  3  exec_prefix=${prefix}

vcpkg_installed/x64-osx/debug/lib/pkgconfig/pcl_features-1.11.pc:
  1  # This file was generated by CMake for PCL library pcl_features
  2: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/pcl_x64-osx/debug
  3  exec_prefix=${prefix}

vcpkg_installed/x64-osx/debug/lib/pkgconfig/pcl_filters-1.11.pc:
  1  # This file was generated by CMake for PCL library pcl_filters
  2: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/pcl_x64-osx/debug
  3  exec_prefix=${prefix}

vcpkg_installed/x64-osx/debug/lib/pkgconfig/pcl_geometry-1.11.pc:
  1  # This file was generated by CMake for PCL library pcl_geometry
  2: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/pcl_x64-osx/debug
  3  exec_prefix=${prefix}

vcpkg_installed/x64-osx/debug/lib/pkgconfig/pcl_io-1.11.pc:
  1  # This file was generated by CMake for PCL library pcl_io
  2: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/pcl_x64-osx/debug
  3  exec_prefix=${prefix}

vcpkg_installed/x64-osx/debug/lib/pkgconfig/pcl_kdtree-1.11.pc:
  1  # This file was generated by CMake for PCL library pcl_kdtree
  2: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/pcl_x64-osx/debug
  3  exec_prefix=${prefix}

vcpkg_installed/x64-osx/debug/lib/pkgconfig/pcl_keypoints-1.11.pc:
  1  # This file was generated by CMake for PCL library pcl_keypoints
  2: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/pcl_x64-osx/debug
  3  exec_prefix=${prefix}

vcpkg_installed/x64-osx/debug/lib/pkgconfig/pcl_ml-1.11.pc:
  1  # This file was generated by CMake for PCL library pcl_ml
  2: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/pcl_x64-osx/debug
  3  exec_prefix=${prefix}

vcpkg_installed/x64-osx/debug/lib/pkgconfig/pcl_octree-1.11.pc:
  1  # This file was generated by CMake for PCL library pcl_octree
  2: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/pcl_x64-osx/debug
  3  exec_prefix=${prefix}

vcpkg_installed/x64-osx/debug/lib/pkgconfig/pcl_recognition-1.11.pc:
  1  # This file was generated by CMake for PCL library pcl_recognition
  2: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/pcl_x64-osx/debug
  3  exec_prefix=${prefix}

vcpkg_installed/x64-osx/debug/lib/pkgconfig/pcl_registration-1.11.pc:
  1  # This file was generated by CMake for PCL library pcl_registration
  2: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/pcl_x64-osx/debug
  3  exec_prefix=${prefix}

vcpkg_installed/x64-osx/debug/lib/pkgconfig/pcl_sample_consensus-1.11.pc:
  1  # This file was generated by CMake for PCL library pcl_sample_consensus
  2: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/pcl_x64-osx/debug
  3  exec_prefix=${prefix}

vcpkg_installed/x64-osx/debug/lib/pkgconfig/pcl_search-1.11.pc:
  1  # This file was generated by CMake for PCL library pcl_search
  2: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/pcl_x64-osx/debug
  3  exec_prefix=${prefix}

vcpkg_installed/x64-osx/debug/lib/pkgconfig/pcl_segmentation-1.11.pc:
  1  # This file was generated by CMake for PCL library pcl_segmentation
  2: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/pcl_x64-osx/debug
  3  exec_prefix=${prefix}

vcpkg_installed/x64-osx/debug/lib/pkgconfig/pcl_stereo-1.11.pc:
  1  # This file was generated by CMake for PCL library pcl_stereo
  2: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/pcl_x64-osx/debug
  3  exec_prefix=${prefix}

vcpkg_installed/x64-osx/debug/lib/pkgconfig/pcl_surface-1.11.pc:
  1  # This file was generated by CMake for PCL library pcl_surface
  2: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/pcl_x64-osx/debug
  3  exec_prefix=${prefix}

vcpkg_installed/x64-osx/debug/lib/pkgconfig/pcl_tracking-1.11.pc:
  1  # This file was generated by CMake for PCL library pcl_tracking
  2: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/pcl_x64-osx/debug
  3  exec_prefix=${prefix}

vcpkg_installed/x64-osx/debug/lib/pkgconfig/pcre2.pc:
  2  
  3: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/bde_x64-osx/debug
  4  exec_prefix=

vcpkg_installed/x64-osx/debug/lib/pkgconfig/plplot-c++.pc:
  1  libdir=lib
  2: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/plplot_x64-osx/debug/include/plplot
  3  drvdir=lib/plplot5.13.0/drivers

vcpkg_installed/x64-osx/debug/lib/pkgconfig/plplot-fortran.pc:
  1  libdir=lib
  2: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/plplot_x64-osx/debug/include/plplot
  3  drvdir=lib/plplot5.13.0/drivers

vcpkg_installed/x64-osx/debug/lib/pkgconfig/plplot.pc:
  1  libdir=lib
  2: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/plplot_x64-osx/debug/include/plplot
  3  drvdir=lib/plplot5.13.0/drivers

vcpkg_installed/x64-osx/debug/lib/pkgconfig/poppler-cpp.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/poppler_x64-osx/debug
  2: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/poppler_x64-osx/debug/lib
  3: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/poppler_x64-osx/debug/include
  4  

vcpkg_installed/x64-osx/debug/lib/pkgconfig/poppler.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/poppler_x64-osx/debug
  2: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/poppler_x64-osx/debug/lib
  3: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/poppler_x64-osx/debug/include
  4  

vcpkg_installed/x64-osx/debug/lib/pkgconfig/portaudio-2.0.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/portaudio_x64-osx/debug
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/debug/lib/pkgconfig/ppconsul.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/ppconsul_x64-osx/debug
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/debug/lib/pkgconfig/pubsub_client.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/qca2.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/qca_x64-osx/debug
  2: exec_prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/qca_x64-osx/debug
  3: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/qca_x64-osx/debug/lib
  4: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/qca_x64-osx/debug/include/QtCrypto
  5  

vcpkg_installed/x64-osx/debug/lib/pkgconfig/raylib.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/raylib_x64-osx/debug
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/debug/lib/pkgconfig/rbdl.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/rbdl_x64-osx/debug
  2: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/rbdl_x64-osx/debug/lib
  3: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/rbdl_x64-osx/debug/include
  4  

vcpkg_installed/x64-osx/debug/lib/pkgconfig/rdkafka-static.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/librdkafka_x64-osx/debug
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/debug/lib/pkgconfig/rdkafka++-static.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/librdkafka_x64-osx/debug
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/debug/lib/pkgconfig/realsense2.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/realsense2_x64-osx/debug
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/debug/lib/pkgconfig/redis++.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/redis-plus-plus_x64-osx/debug
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/debug/lib/pkgconfig/rpclib.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/rpclib_x64-osx/debug
  2  exec_prefix=
  3: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/rpclib_x64-osx/debug/lib
  4: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/rpclib_x64-osx/debug/include
  5  

vcpkg_installed/x64-osx/debug/lib/pkgconfig/rtaudio.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/rtaudio_x64-osx/debug
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/debug/lib/pkgconfig/sdformat.pc:
  1: prefix="/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/sdformat6_x64-osx/debug"
  2  libdir=${prefix}/lib

vcpkg_installed/x64-osx/debug/lib/pkgconfig/sentencepiece.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/sentencepiece_x64-osx/debug
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/debug/lib/pkgconfig/simage.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/simage_x64-osx/debug
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/debug/lib/pkgconfig/soxr.pc:
  3  Version: 0.1.3
  4: Libs: -L/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/soxr_x64-osx/debug/lib -lsoxr
  5: Cflags: -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/soxr_x64-osx/debug/include

vcpkg_installed/x64-osx/debug/lib/pkgconfig/spanner_client.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/srt.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libsrt_x64-osx/debug
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/debug/lib/pkgconfig/storage_client.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx/debug
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/debug/lib/pkgconfig/stxxl_debug.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/stxxl_x64-osx/debug
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/debug/lib/pkgconfig/superlu.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/superlu_x64-osx/debug
  2: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/superlu_x64-osx/debug/lib
  3: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/superlu_x64-osx/debug/include
  4  

vcpkg_installed/x64-osx/debug/lib/pkgconfig/thrift-nb.pc:
  19  
  20: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/thrift_x64-osx/debug
  21: exec_prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/thrift_x64-osx/debug/bin
  22: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/thrift_x64-osx/debug/lib
  23: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/thrift_x64-osx/debug/include
  24  

vcpkg_installed/x64-osx/debug/lib/pkgconfig/thrift-z.pc:
  19  
  20: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/thrift_x64-osx/debug
  21: exec_prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/thrift_x64-osx/debug/bin
  22: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/thrift_x64-osx/debug/lib
  23: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/thrift_x64-osx/debug/include
  24  

vcpkg_installed/x64-osx/debug/lib/pkgconfig/thrift.pc:
  19  
  20: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/thrift_x64-osx/debug
  21: exec_prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/thrift_x64-osx/debug/bin
  22: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/thrift_x64-osx/debug/lib
  23: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/thrift_x64-osx/debug/include
  24  

vcpkg_installed/x64-osx/debug/lib/pkgconfig/usrsctp.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/usrsctp_x64-osx/debug
  2: exec_prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/usrsctp_x64-osx/debug
  3: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/usrsctp_x64-osx/debug/lib
  4: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/usrsctp_x64-osx/debug/include/usrsctp
  5  

vcpkg_installed/x64-osx/debug/lib/pkgconfig/xerces-c.pc:
   1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/xerces-c_x64-osx/debug
   2: exec_prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/xerces-c_x64-osx/debug
   3: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/xerces-c_x64-osx/debug/lib
   4: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/xerces-c_x64-osx/debug/include
   5  

   8  Version: 3.2.3
   9: Libs: -L/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/xerces-c_x64-osx/debug/lib -lxerces-c
  10  Libs.private: 
  11: Cflags: -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/xerces-c_x64-osx/debug/include

vcpkg_installed/x64-osx/debug/lib/pkgconfig/xlsxwriter.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libxlsxwriter_x64-osx/debug
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/debug/lib/pkgconfig/zzipfseeko.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/zzipfseeko_x64-osx/debug
  2  libdir=${prefix}/lib

vcpkg_installed/x64-osx/debug/lib/pkgconfig/zziplib.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/zziplib_x64-osx/debug
  2  libdir=${prefix}/lib

vcpkg_installed/x64-osx/debug/lib/pkgconfig/zzipmmapped.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/zzipmmapped_x64-osx/debug
  2  libdir=${prefix}/lib

vcpkg_installed/x64-osx/debug/lib/systemd/user/plasma-kglobalaccel.service:
  5  [Service]
  6: ExecStart=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/kf5globalaccel_x64-osx/debug/bin/kglobalaccel5
  7  BusName=org.kde.kglobalaccel

vcpkg_installed/x64-osx/debug/libdata/pkgconfig/openmesh.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/openmesh_x64-osx/debug
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/debug/pkgconfig/quill.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/quill_x64-osx/debug
  2: exec_prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/quill_x64-osx/debug
  3  

vcpkg_installed/x64-osx/debug/tools/libpng/libpng-config:
  14  version="1.6.37"
  15: prefix="/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libpng_x64-osx/debug"
  16: exec_prefix="/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libpng_x64-osx/debug"
  17: libdir="/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libpng_x64-osx/debug/lib"
  18: includedir="/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libpng_x64-osx/debug/include/libpng16"
  19  libs="-lpng16"

vcpkg_installed/x64-osx/debug/tools/libpng/libpng16-config:
  14  version="1.6.37"
  15: prefix="/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libpng_x64-osx/debug"
  16: exec_prefix="/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libpng_x64-osx/debug"
  17: libdir="/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libpng_x64-osx/debug/lib"
  18: includedir="/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libpng_x64-osx/debug/include/libpng16"
  19  libs="-lpng16"

vcpkg_installed/x64-osx/etc/fonts/fonts.conf:
  91  
  92: 	<cachedir>/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/fontconfig_x64-osx/../var/cache/fontconfig</cachedir>
  93  	<cachedir prefix="xdg">fontconfig</cachedir>

vcpkg_installed/x64-osx/etc/fonts/conf.d/README:
  4  scans this directory, loading all files of the form [0-9][0-9]*.conf.
  5: These files are normally installed in /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/fontconfig_x64-osx/share/fontconfig/conf.avail
  6  and then symlinked here, allowing them to be easily installed and then

vcpkg_installed/x64-osx/gdk-pixbuf-2.0/2.10.0/loaders.cache:
  4  #
  5: # LoaderDir = /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/gdk-pixbuf_x64-osx/lib/gdk-pixbuf-2.0/2.10.0/loaders
  6  #

vcpkg_installed/x64-osx/include/gdcmConfigure.h:
  90  
  91: #define GDCM_CMAKE_INSTALL_PREFIX "/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/gdcm_x64-osx"
  92  #define GDCM_INSTALL_INCLUDE_DIR "include"

vcpkg_installed/x64-osx/include/lws_config.h:
   8  
   9: #define LWS_INSTALL_DATADIR "/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libwebsockets_x64-osx/share"
  10: #define LWS_INSTALL_LIBDIR "/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libwebsockets_x64-osx/lib"
  11  #define LWS_LIBRARY_VERSION_MAJOR 4

vcpkg_installed/x64-osx/include/botan/build.h:
   7  * Automatically generated from
   8: * 'configure.py --cpu=x86_64 --disable-shared-library --enable-static-library --distribution-info=vcpkg x64-osx --prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/botan_x64-osx --with-pkg-config --link-method=copy'
   9  *

  31  
  32: #define BOTAN_INSTALL_PREFIX R"(/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/botan_x64-osx)"
  33  #define BOTAN_INSTALL_HEADER_DIR R"(include/botan-2)"
  34: #define BOTAN_INSTALL_LIB_DIR R"(/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/botan_x64-osx/lib)"
  35  #define BOTAN_LIB_LINK "-ldl -framework CoreFoundation -framework Security"

vcpkg_installed/x64-osx/include/dart/config.hpp:
  67  #define DART_DATA_LOCAL_PATH "/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dartsim/src/v6.9.4-5c5f2063c8.clean/data/"
  68: #define DART_DATA_GLOBAL_PATH "/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/dartsim_x64-osx/share/doc/dart/data/"
  69  

vcpkg_installed/x64-osx/include/dcmtk/config/osconfig.h:
   50  /* Define the DCMTK default path */
   51: #define DCMTK_PREFIX "/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/dcmtk_x64-osx"
   52  
   53  /* Define the default data dictionary path for the dcmdata library package */
   54: #define DCM_DICT_DEFAULT_PATH "/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/dcmtk_x64-osx/share/dcmtk/dicom.dic:/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/dcmtk_x64-osx/share/dcmtk/private.dic"
   55  

  842  /* Define to the default configuration directory (used by some applications) */
  843: #define DEFAULT_CONFIGURATION_DIR "/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/dcmtk_x64-osx/etc/dcmtk/"
  844  
  845  /* Define to the default support data directory (used by some applications) */
  846: #define DEFAULT_SUPPORT_DATA_DIR "/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/dcmtk_x64-osx/share/dcmtk/"
  847  

vcpkg_installed/x64-osx/include/KF5/KWindowSystem/config-kwindowsystem.h:
  12  /* Path to xcb plugin */
  13: #define XCB_PLUGIN_PATH "/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/kf5windowsystem_x64-osx/plugins/kf5/kwindowsystem/KF5WindowSystemX11Plugin.so"

vcpkg_installed/x64-osx/include/lager/resources_path.hpp:
  22    return env_resources_path ? env_resources_path
  23:                             : "/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/lager_x64-osx" "/share/lager";
  24  }

vcpkg_installed/x64-osx/include/proton/version.h:
  28  
  29: #define PN_INSTALL_PREFIX "/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/qpid-proton_x64-osx"
  30  

vcpkg_installed/x64-osx/include/sdformat-6.2/sdf/sdf_config.h:
  29  
  30: #define SDF_SHARE_PATH "/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/sdformat6_x64-osx/share/"
  31: #define SDF_VERSION_PATH "/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/sdformat6_x64-osx/share/sdformat/"

vcpkg_installed/x64-osx/include/symengine/symengine_config_cling.h:
  3  
  4: #pragma cling add_library_path("/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/symengine_x64-osx/lib")
  5  #pragma cling load("libsymengine")

vcpkg_installed/x64-osx/lib/botan-2.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/botan_x64-osx
  2  exec_prefix=${prefix}
  3: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/botan_x64-osx/lib
  4  includedir=${prefix}/include/botan-2

vcpkg_installed/x64-osx/lib/libgraphite2.la:
  37  # Directory that this library needs to be installed in:
  38: libdir='/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/graphite2_x64-osx/lib'

vcpkg_installed/x64-osx/lib/libhdf5.settings:
  11                         Byte sex: little-endian
  12:              Installation point: /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/hdf5_x64-osx
  13  

vcpkg_installed/x64-osx/lib/libnetcdf.settings:
  10  Build Directory: 	/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/netcdf-c/x64-osx-rel
  11: Install Prefix:         /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/netcdf-c_x64-osx
  12  

vcpkg_installed/x64-osx/lib/glibmm-2.68/proc/gmmproc:
  42    # get prefix info from configure
  43:   my $prefix = "/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/glibmm_x64-osx";
  44    my $exec_prefix = "${prefix}";

vcpkg_installed/x64-osx/lib/manual-link/pkgconfig/gmock_main.pc:
  1: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/gtest_x64-osx/lib
  2: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/gtest_x64-osx/include
  3  

vcpkg_installed/x64-osx/lib/manual-link/pkgconfig/gtest_main.pc:
  1: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/gtest_x64-osx/lib
  2: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/gtest_x64-osx/include
  3  

vcpkg_installed/x64-osx/lib/pkgconfig/ACE_ETCL_Parser.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/ace_x64-osx
  2: exec_prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/ace_x64-osx
  3: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/ace_x64-osx/lib
  4: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/ace_x64-osx/include
  5  

vcpkg_installed/x64-osx/lib/pkgconfig/ACE_ETCL.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/ace_x64-osx
  2: exec_prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/ace_x64-osx
  3: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/ace_x64-osx/lib
  4: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/ace_x64-osx/include
  5  

vcpkg_installed/x64-osx/lib/pkgconfig/ACE.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/ace_x64-osx
  2: exec_prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/ace_x64-osx
  3: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/ace_x64-osx/lib
  4: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/ace_x64-osx/include
  5  

vcpkg_installed/x64-osx/lib/pkgconfig/amqpcpp.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/amqpcpp_x64-osx
  2  libdir=${prefix}/lib

vcpkg_installed/x64-osx/lib/pkgconfig/aom.pc:
  1  # libaom pkg-config.
  2: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/aom_x64-osx
  3  exec_prefix=${prefix}

vcpkg_installed/x64-osx/lib/pkgconfig/armadillo.pc:
  1: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/armadillo_x64-osx/lib
  2: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/armadillo_x64-osx/include
  3  

vcpkg_installed/x64-osx/lib/pkgconfig/arrow-csv.pc:
  17  
  18: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/arrow_x64-osx/lib
  19: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/arrow_x64-osx/include
  20  

vcpkg_installed/x64-osx/lib/pkgconfig/arrow-filesystem.pc:
  17  
  18: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/arrow_x64-osx/lib
  19: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/arrow_x64-osx/include
  20  

vcpkg_installed/x64-osx/lib/pkgconfig/arrow-json.pc:
  17  
  18: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/arrow_x64-osx/lib
  19: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/arrow_x64-osx/include
  20  

vcpkg_installed/x64-osx/lib/pkgconfig/arrow.pc:
  17  
  18: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/arrow_x64-osx/lib
  19: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/arrow_x64-osx/include
  20  

vcpkg_installed/x64-osx/lib/pkgconfig/bal.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/bde_x64-osx
  2  libdir=${prefix}/lib

vcpkg_installed/x64-osx/lib/pkgconfig/bbl.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/bde_x64-osx
  2  libdir=${prefix}/lib

vcpkg_installed/x64-osx/lib/pkgconfig/bdl.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/bde_x64-osx
  2  libdir=${prefix}/lib

vcpkg_installed/x64-osx/lib/pkgconfig/bigtable_client.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/blitz-static.pc:
  1  # generated by cmake. remove this line to disable regeneration
  2: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/blitz_x64-osx
  3  exec_prefix=${prefix}

vcpkg_installed/x64-osx/lib/pkgconfig/blitz.pc:
  1  # generated by cmake. remove this line to disable regeneration
  2: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/blitz_x64-osx
  3  exec_prefix=${prefix}

vcpkg_installed/x64-osx/lib/pkgconfig/blosc.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/blosc_x64-osx
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/lib/pkgconfig/brpc.pc:
  15  
  16: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/brpc_x64-osx
  17  exec_prefix=${prefix}

vcpkg_installed/x64-osx/lib/pkgconfig/bsl.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/bde_x64-osx
  2  libdir=${prefix}/lib

vcpkg_installed/x64-osx/lib/pkgconfig/btl.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/bde_x64-osx
  2  libdir=${prefix}/lib

vcpkg_installed/x64-osx/lib/pkgconfig/bullet.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/bullet3_x64-osx
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/lib/pkgconfig/capnp-json.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/capnproto_x64-osx
  2  exec_prefix=
  3: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/capnproto_x64-osx/lib
  4: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/capnproto_x64-osx/include
  5  

vcpkg_installed/x64-osx/lib/pkgconfig/capnp-rpc.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/capnproto_x64-osx
  2  exec_prefix=
  3: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/capnproto_x64-osx/lib
  4: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/capnproto_x64-osx/include
  5  

vcpkg_installed/x64-osx/lib/pkgconfig/capnp.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/capnproto_x64-osx
  2  exec_prefix=
  3: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/capnproto_x64-osx/lib
  4: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/capnproto_x64-osx/include
  5  

vcpkg_installed/x64-osx/lib/pkgconfig/ccd.pc:
  2  
  3: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/ccd_x64-osx
  4  exec_prefix=${prefix}

vcpkg_installed/x64-osx/lib/pkgconfig/charls.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/charls_x64-osx
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/lib/pkgconfig/check.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/check_x64-osx
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/lib/pkgconfig/clblast.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/clblast_x64-osx
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/lib/pkgconfig/cmocka.pc:
  3  Version: 1.1.5
  4: Libs: -L/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/cmocka_x64-osx/lib -lcmocka
  5: Cflags: -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/cmocka_x64-osx/include

vcpkg_installed/x64-osx/lib/pkgconfig/Coin.pc:
   1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/coin_x64-osx
   2  exec_prefix=${prefix}
   3: bindir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/coin_x64-osx/bin
   4: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/coin_x64-osx/lib
   5: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/coin_x64-osx/include
   6: datarootdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/coin_x64-osx/share
   7: datadir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/coin_x64-osx/share
   8: docdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/coin_x64-osx/share/doc/Coin
   9: infodir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/coin_x64-osx/share/info
  10: mandir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/coin_x64-osx/share/man
  11  htmldir=${docdir}/html

vcpkg_installed/x64-osx/lib/pkgconfig/collada-dom-141.pc:
  2  
  3: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/collada-dom_x64-osx
  4: exec_prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/collada-dom_x64-osx/bin
  5: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/collada-dom_x64-osx/lib
  6: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/collada-dom_x64-osx/include/collada-dom2.5
  7  

vcpkg_installed/x64-osx/lib/pkgconfig/collada-dom-150.pc:
  2  
  3: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/collada-dom_x64-osx
  4: exec_prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/collada-dom_x64-osx/bin
  5: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/collada-dom_x64-osx/lib
  6: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/collada-dom_x64-osx/include/collada-dom2.5
  7  

vcpkg_installed/x64-osx/lib/pkgconfig/collada-dom.pc:
  2  
  3: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/collada-dom_x64-osx
  4: exec_prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/collada-dom_x64-osx/bin
  5: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/collada-dom_x64-osx/lib
  6: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/collada-dom_x64-osx/include/collada-dom2.5
  7  

vcpkg_installed/x64-osx/lib/pkgconfig/cpp_redis.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/cpp-redis_x64-osx
  2  libdir=${prefix}/lib

vcpkg_installed/x64-osx/lib/pkgconfig/csv2.pc:
  1: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/p-ranav-csv2_x64-osx/include
  2  

vcpkg_installed/x64-osx/lib/pkgconfig/curlpp.pc:
  1  # This is a comment
  2: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/curlpp_x64-osx
  3: exec_prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/curlpp_x64-osx
  4  includedir=include

vcpkg_installed/x64-osx/lib/pkgconfig/Cutelyst2Qt5Authentication.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/cutelyst2_x64-osx
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/lib/pkgconfig/Cutelyst2Qt5Core.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/cutelyst2_x64-osx
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/lib/pkgconfig/Cutelyst2Qt5Session.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/cutelyst2_x64-osx
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/lib/pkgconfig/Cutelyst2Qt5StaticSimple.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/cutelyst2_x64-osx
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/lib/pkgconfig/Cutelyst2Qt5StatusMessage.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/cutelyst2_x64-osx
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/lib/pkgconfig/Cutelyst2Qt5UserAgent.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/cutelyst2_x64-osx
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/lib/pkgconfig/Cutelyst2Qt5UtilsLangSelect.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/cutelyst2_x64-osx
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/lib/pkgconfig/Cutelyst2Qt5UtilsPagination.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/cutelyst2_x64-osx
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/lib/pkgconfig/Cutelyst2Qt5UtilsSql.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/cutelyst2_x64-osx
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/lib/pkgconfig/Cutelyst2Qt5UtilsValidator.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/cutelyst2_x64-osx
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/lib/pkgconfig/Cutelyst2Qt5ViewJSON.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/cutelyst2_x64-osx
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/lib/pkgconfig/Cutelyst2Qt5WSGI.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/cutelyst2_x64-osx
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/lib/pkgconfig/decnumber.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/bde_x64-osx
  2  libdir=${prefix}/lib

vcpkg_installed/x64-osx/lib/pkgconfig/dlib-1.pc:
  1: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/dlib_x64-osx/lib
  2: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/dlib_x64-osx/include
  3  

vcpkg_installed/x64-osx/lib/pkgconfig/edlib-1.pc:
  1: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/edlib_x64-osx/lib
  2: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/edlib_x64-osx/include
  3  

vcpkg_installed/x64-osx/lib/pkgconfig/evhtp.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libevhtp_x64-osx
  2  libdir=

vcpkg_installed/x64-osx/lib/pkgconfig/fcl.pc:
  1  # This file was generated by CMake for fcl
  2: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/fcl_x64-osx
  3  exec_prefix=${prefix}
  4: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/fcl_x64-osx/lib
  5: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/fcl_x64-osx/include
  6  

vcpkg_installed/x64-osx/lib/pkgconfig/fftw3.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/fftw3_x64-osx
  2: exec_prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/fftw3_x64-osx
  3: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/fftw3_x64-osx/lib
  4: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/fftw3_x64-osx/include
  5  

vcpkg_installed/x64-osx/lib/pkgconfig/fftw3f.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/fftw3_x64-osx
  2: exec_prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/fftw3_x64-osx
  3: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/fftw3_x64-osx/lib
  4: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/fftw3_x64-osx/include
  5  

vcpkg_installed/x64-osx/lib/pkgconfig/fftw3l.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/fftw3_x64-osx
  2: exec_prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/fftw3_x64-osx
  3: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/fftw3_x64-osx/lib
  4: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/fftw3_x64-osx/include
  5  

vcpkg_installed/x64-osx/lib/pkgconfig/flann.pc:
  1  # This file was generated by CMake for flann
  2: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/flann_x64-osx
  3  exec_prefix=${prefix}

vcpkg_installed/x64-osx/lib/pkgconfig/flatbuffers.pc:
  1: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/flatbuffers_x64-osx/lib
  2: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/flatbuffers_x64-osx/include
  3  

vcpkg_installed/x64-osx/lib/pkgconfig/fluidlite.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/fluidlite_x64-osx
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/lib/pkgconfig/functions_framework_cpp.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/functions-framework-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/gdlib.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libgd_x64-osx
  2: exec_prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libgd_x64-osx
  3  libdir=${exec_prefix}/lib

vcpkg_installed/x64-osx/lib/pkgconfig/geogram1.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/geogram_x64-osx
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/lib/pkgconfig/geographiclib.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/geographiclib_x64-osx
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/lib/pkgconfig/gflags.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/gflags_x64-osx
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/lib/pkgconfig/glut.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/freeglut_x64-osx
  2: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/freeglut_x64-osx/lib
  3  includedir=${prefix}/include

vcpkg_installed/x64-osx/lib/pkgconfig/gmock.pc:
  1: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/gtest_x64-osx/lib
  2: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/gtest_x64-osx/include
  3  

vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_api_annotations_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_api_auth_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_api_client_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_api_distribution_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_api_field_behavior_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_api_http_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_api_label_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_api_launch_stage_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_api_metric_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_api_monitored_resource_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_api_resource_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_bigquery.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_bigtable_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_bigtable.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_cloud_bigquery_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_cloud_dialogflow_v2_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_cloud_dialogflow_v2beta1_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_cloud_speech_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_cloud_texttospeech_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_common.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_devtools_cloudtrace_v2_trace_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_devtools_cloudtrace_v2_tracing_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_grpc_utils.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_iam_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_iam_v1_iam_policy_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_iam_v1_options_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_iam_v1_policy_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_iam.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_logging_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_logging_type_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_longrunning_operations_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_monitoring_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_pubsub_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_pubsub.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_rpc_error_details_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_rpc_status_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_spanner_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_spanner.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_storage_grpc.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_storage_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_storage.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_type_calendar_period_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_type_color_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_type_date_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_type_datetime_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_type_dayofweek_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_type_expr_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_type_fraction_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_type_interval_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_type_latlng_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_type_localized_text_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_type_money_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_type_month_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_type_phone_number_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_type_postal_address_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_type_quaternion_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_type_timeofday_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/googleapis_cpp_api_annotations_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/googleapis_cpp_api_auth_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/googleapis_cpp_api_client_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/googleapis_cpp_api_distribution_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/googleapis_cpp_api_field_behavior_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/googleapis_cpp_api_http_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/googleapis_cpp_api_label_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/googleapis_cpp_api_launch_stage_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/googleapis_cpp_api_metric_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/googleapis_cpp_api_monitored_resource_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/googleapis_cpp_api_resource_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/googleapis_cpp_bigtable_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/googleapis_cpp_cloud_bigquery_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/googleapis_cpp_cloud_dialogflow_v2_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/googleapis_cpp_cloud_dialogflow_v2beta1_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/googleapis_cpp_cloud_speech_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/googleapis_cpp_cloud_texttospeech_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/googleapis_cpp_devtools_cloudtrace_v2_trace_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/googleapis_cpp_devtools_cloudtrace_v2_tracing_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/googleapis_cpp_iam_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/googleapis_cpp_iam_v1_iam_policy_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/googleapis_cpp_iam_v1_options_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/googleapis_cpp_iam_v1_policy_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/googleapis_cpp_logging_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/googleapis_cpp_logging_type_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/googleapis_cpp_longrunning_operations_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/googleapis_cpp_monitoring_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/googleapis_cpp_pubsub_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/googleapis_cpp_rpc_error_details_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/googleapis_cpp_rpc_status_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/googleapis_cpp_spanner_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/googleapis_cpp_storage_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/googleapis_cpp_type_calendar_period_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/googleapis_cpp_type_color_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/googleapis_cpp_type_date_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/googleapis_cpp_type_datetime_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/googleapis_cpp_type_dayofweek_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/googleapis_cpp_type_expr_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/googleapis_cpp_type_fraction_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/googleapis_cpp_type_interval_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/googleapis_cpp_type_latlng_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/googleapis_cpp_type_localized_text_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/googleapis_cpp_type_money_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/googleapis_cpp_type_month_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/googleapis_cpp_type_phone_number_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/googleapis_cpp_type_postal_address_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/googleapis_cpp_type_quaternion_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/googleapis_cpp_type_timeofday_protos.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/googleapis.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/gpr.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/grpc_x64-osx
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/lib/pkgconfig/graphite2.pc:
  4  Version: 3.0.1
  5: Libs: -L/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/graphite2_x64-osx/lib -lgraphite2
  6: Cflags: -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/graphite2_x64-osx/include
  7  

vcpkg_installed/x64-osx/lib/pkgconfig/grpc_unsecure.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/grpc_x64-osx
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/lib/pkgconfig/grpc.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/grpc_x64-osx
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/lib/pkgconfig/grpc++_unsecure.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/grpc_x64-osx
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/lib/pkgconfig/grpc++.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/grpc_x64-osx
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/lib/pkgconfig/gta.pc:
  7  
  8: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libgta_x64-osx
  9  exec_prefix=${prefix}

vcpkg_installed/x64-osx/lib/pkgconfig/gtest.pc:
  1: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/gtest_x64-osx/lib
  2: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/gtest_x64-osx/include
  3  

vcpkg_installed/x64-osx/lib/pkgconfig/haisrt.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libsrt_x64-osx
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/lib/pkgconfig/hiredis.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/hiredis_x64-osx
  2  install_libdir=lib

vcpkg_installed/x64-osx/lib/pkgconfig/igraph.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/igraph_x64-osx
  2: exec_prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/igraph_x64-osx
  3  libdir=${exec_prefix}/lib

vcpkg_installed/x64-osx/lib/pkgconfig/iniparser.pc:
   3  libdir=/lib
   4: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/iniparser_x64-osx/include
   5: datarootdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/iniparser_x64-osx/share
   6: datadir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/iniparser_x64-osx/share
   7  

  10  Version: 4.1
  11: Libs: -L/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/iniparser_x64-osx/lib -liniparser
  12: Cflags: -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/iniparser_x64-osx/include

vcpkg_installed/x64-osx/lib/pkgconfig/inteldfp.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/bde_x64-osx
  2  libdir=${prefix}/lib

vcpkg_installed/x64-osx/lib/pkgconfig/itpp.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/itpp_x64-osx
  2: exec_prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/itpp_x64-osx
  3: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/itpp_x64-osx/lib
  4: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/itpp_x64-osx/include
  5  

vcpkg_installed/x64-osx/lib/pkgconfig/jansson.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/jansson_x64-osx
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/lib/pkgconfig/json11.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/json11_x64-osx
  2  libdir=${prefix}/lib/

vcpkg_installed/x64-osx/lib/pkgconfig/kj-async.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/capnproto_x64-osx
  2  exec_prefix=
  3: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/capnproto_x64-osx/lib
  4: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/capnproto_x64-osx/include
  5  

vcpkg_installed/x64-osx/lib/pkgconfig/kj-http.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/capnproto_x64-osx
  2  exec_prefix=
  3: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/capnproto_x64-osx/lib
  4: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/capnproto_x64-osx/include
  5  

vcpkg_installed/x64-osx/lib/pkgconfig/kj-test.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/capnproto_x64-osx
  2  exec_prefix=
  3: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/capnproto_x64-osx/lib
  4: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/capnproto_x64-osx/include
  5  

vcpkg_installed/x64-osx/lib/pkgconfig/kj.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/capnproto_x64-osx
  2  exec_prefix=
  3: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/capnproto_x64-osx/lib
  4: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/capnproto_x64-osx/include
  5  

vcpkg_installed/x64-osx/lib/pkgconfig/lemon.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/liblemon_x64-osx
  2: exec_prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/liblemon_x64-osx/bin
  3: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/liblemon_x64-osx/lib
  4: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/liblemon_x64-osx/include
  5  

vcpkg_installed/x64-osx/lib/pkgconfig/libbcg729.pc:
  1  # This is a comment
  2: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/bcg729_x64-osx
  3  exec_prefix=${prefix}

vcpkg_installed/x64-osx/lib/pkgconfig/libbddisasm.pc:
  1: dir_prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/bddisasm_x64-osx
  2  lib=lib

vcpkg_installed/x64-osx/lib/pkgconfig/libbson-1.0.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libbson_x64-osx
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/lib/pkgconfig/libbsoncxx-static.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/mongo-cxx-driver_x64-osx
  16  includedir=${prefix}/include

vcpkg_installed/x64-osx/lib/pkgconfig/libcerf.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libcerf_x64-osx
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/lib/pkgconfig/libcjson.pc:
  1: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/cjson_x64-osx/lib
  2: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/cjson_x64-osx/include
  3  

vcpkg_installed/x64-osx/lib/pkgconfig/libconfuse.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libconfuse_x64-osx
  2: exec_prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libconfuse_x64-osx
  3: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libconfuse_x64-osx/lib
  4: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libconfuse_x64-osx/include
  5  

vcpkg_installed/x64-osx/lib/pkgconfig/libcpuid.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/cpuid_x64-osx
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/lib/pkgconfig/libevent_core.pc:
  2  
  3: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libevent_x64-osx
  4: exec_prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libevent_x64-osx
  5: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libevent_x64-osx/lib
  6: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libevent_x64-osx/include
  7  

vcpkg_installed/x64-osx/lib/pkgconfig/libevent_extra.pc:
  2  
  3: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libevent_x64-osx
  4: exec_prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libevent_x64-osx
  5: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libevent_x64-osx/lib
  6: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libevent_x64-osx/include
  7  

vcpkg_installed/x64-osx/lib/pkgconfig/libevent_openssl.pc:
  2  
  3: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libevent_x64-osx
  4: exec_prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libevent_x64-osx
  5: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libevent_x64-osx/lib
  6: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libevent_x64-osx/include
  7  

vcpkg_installed/x64-osx/lib/pkgconfig/libevent_pthreads.pc:
  2  
  3: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libevent_x64-osx
  4: exec_prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libevent_x64-osx
  5: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libevent_x64-osx/lib
  6: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libevent_x64-osx/include
  7  

vcpkg_installed/x64-osx/lib/pkgconfig/libevent.pc:
  2  
  3: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libevent_x64-osx
  4: exec_prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libevent_x64-osx
  5: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libevent_x64-osx/lib
  6: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libevent_x64-osx/include
  7  

vcpkg_installed/x64-osx/lib/pkgconfig/libfolly.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/folly_x64-osx
  2: exec_prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/folly_x64-osx
  3  libdir=${exec_prefix}/lib

vcpkg_installed/x64-osx/lib/pkgconfig/libfort.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libfort_x64-osx
  2: exec_prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libfort_x64-osx
  3  includedir=${prefix}/include

vcpkg_installed/x64-osx/lib/pkgconfig/libftdi1.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libftdi1_x64-osx
  2: exec_prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libftdi1_x64-osx/bin
  3: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libftdi1_x64-osx/lib
  4: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libftdi1_x64-osx/include/libftdi1
  5  

vcpkg_installed/x64-osx/lib/pkgconfig/libftdipp1.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libftdi1_x64-osx
  2: exec_prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libftdi1_x64-osx/bin
  3: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libftdi1_x64-osx/lib
  4: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libftdi1_x64-osx/include/libftdi1
  5  

vcpkg_installed/x64-osx/lib/pkgconfig/libgit2.pc:
  1: prefix="/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libgit2_x64-osx"
  2: libdir="/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libgit2_x64-osx/lib"
  3: includedir="/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libgit2_x64-osx/include"
  4  

vcpkg_installed/x64-osx/lib/pkgconfig/libgraphqlparser.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/graphqlparser_x64-osx
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/lib/pkgconfig/libheif.pc:
   1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libheif_x64-osx
   2: exec_prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libheif_x64-osx
   3: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libheif_x64-osx/lib
   4: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libheif_x64-osx/include
   5  builtin_h265_decoder=yes

  15  Requires.private: libde265 x265 aom
  16: Libs: -L/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libheif_x64-osx/lib -lheif
  17  Libs.private:  -lstdc++
  18: Cflags: -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libheif_x64-osx/include

vcpkg_installed/x64-osx/lib/pkgconfig/libhs.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/hyperscan_x64-osx
  2: exec_prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/hyperscan_x64-osx
  3: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/hyperscan_x64-osx/lib
  4: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/hyperscan_x64-osx/include
  5  

vcpkg_installed/x64-osx/lib/pkgconfig/libkeyfinder.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libkeyfinder_x64-osx
  2  includedir=${prefix}/include

vcpkg_installed/x64-osx/lib/pkgconfig/libKF5Attica.pc:
   1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/kf5attica_x64-osx
   2  exec_prefix=${prefix}
   3: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/kf5attica_x64-osx/lib
   4: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/kf5attica_x64-osx/include
   5  

  10  Version: 5.84.0
  11: Libs: -L/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/kf5attica_x64-osx/lib -lKF5Attica
  12: Cflags: -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/kf5attica_x64-osx/include/KF5/Attica

vcpkg_installed/x64-osx/lib/pkgconfig/libkml.pc:
  2  
  3: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libkml_x64-osx
  4  exec_prefix=${prefix}

vcpkg_installed/x64-osx/lib/pkgconfig/liblo.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/liblo_x64-osx
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/lib/pkgconfig/libmediainfo.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libmediainfo_x64-osx
  2  exec_prefix=${prefix}
  3: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libmediainfo_x64-osx/lib
  4: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libmediainfo_x64-osx/include
  5  Libs_Static=${libdir}/libmediainfo.a ${libdir}/libzen.a -lpthread -lz -lcurl

vcpkg_installed/x64-osx/lib/pkgconfig/libmikmod.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libmikmod_x64-osx
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/lib/pkgconfig/libmongoc-1.0.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/mongo-c-driver_x64-osx
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/lib/pkgconfig/libmongoc-ssl-1.0.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/mongo-c-driver_x64-osx
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/lib/pkgconfig/libmongocxx-static.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/mongo-cxx-driver_x64-osx
  16  includedir=${prefix}/include

vcpkg_installed/x64-osx/lib/pkgconfig/libmosquitto.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/mosquitto_x64-osx
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/lib/pkgconfig/libmosquittopp.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/mosquitto_x64-osx
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/lib/pkgconfig/libopcuaclient.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/freeopcua_x64-osx
  2: exec_prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/freeopcua_x64-osx
  3: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/freeopcua_x64-osx/lib
  4: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/freeopcua_x64-osx/include
  5  

vcpkg_installed/x64-osx/lib/pkgconfig/libopcuacore.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/freeopcua_x64-osx
  2: exec_prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/freeopcua_x64-osx
  3: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/freeopcua_x64-osx/lib
  4: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/freeopcua_x64-osx/include
  5  

vcpkg_installed/x64-osx/lib/pkgconfig/libopcuaprotocol.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/freeopcua_x64-osx
  2: exec_prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/freeopcua_x64-osx
  3: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/freeopcua_x64-osx/lib
  4: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/freeopcua_x64-osx/include
  5  

vcpkg_installed/x64-osx/lib/pkgconfig/libopcuaserver.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/freeopcua_x64-osx
  2: exec_prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/freeopcua_x64-osx
  3: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/freeopcua_x64-osx/lib
  4: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/freeopcua_x64-osx/include
  5  

vcpkg_installed/x64-osx/lib/pkgconfig/libpmemobj++.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libpmemobj-cpp_x64-osx
  2  includedir=${prefix}/include

vcpkg_installed/x64-osx/lib/pkgconfig/libproxy-1.0.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libproxy_x64-osx
  2  libdir=${prefix}/lib

vcpkg_installed/x64-osx/lib/pkgconfig/libqrencode.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libqrencode_x64-osx
  2: exec_prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libqrencode_x64-osx/bin
  3: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libqrencode_x64-osx/lib
  4: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libqrencode_x64-osx/include
  5  

vcpkg_installed/x64-osx/lib/pkgconfig/libraw_r.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libraw_x64-osx
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/lib/pkgconfig/libraw.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libraw_x64-osx
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/lib/pkgconfig/libsais.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/sais_x64-osx
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/lib/pkgconfig/libsais64.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/sais_x64-osx
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/lib/pkgconfig/libsbml.pc:
  5  
  6: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libsbml_x64-osx
  7: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libsbml_x64-osx/lib
  8: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libsbml_x64-osx/include
  9  

vcpkg_installed/x64-osx/lib/pkgconfig/libsparsehash.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/sparsehash_x64-osx
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/lib/pkgconfig/libtins.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libtins_x64-osx
  2: exec_prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libtins_x64-osx
  3: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libtins_x64-osx/lib
  4  includedir=${prefix}/include

vcpkg_installed/x64-osx/lib/pkgconfig/libtorrent-rasterbar.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libtorrent_x64-osx
  2  libdir=${prefix}/lib

  7  Libs: -L${libdir} -ltorrent-rasterbar -L/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/x64-osx/lib -l-framework CoreFoundation -l-framework SystemConfiguration -lIconv::Iconv -lssl -lcrypto -ldl -lcrypto -ldl
  8: Cflags: -fexceptions -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libtorrent_x64-osx/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/x64-osx/include -DBOOST_ASIO_ENABLE_CANCELIO -DTORRENT_NO_DEPRECATE -DTORRENT_USE_ICONV -DTORRENT_USE_OPENSSL -DTORRENT_USE_LIBCRYPTO

vcpkg_installed/x64-osx/lib/pkgconfig/liburiparser.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/uriparser_x64-osx
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/lib/pkgconfig/libwebsockets_static.pc:
  1: prefix="/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libwebsockets_x64-osx"
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/lib/pkgconfig/libwebsockets.pc:
  1: prefix="/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libwebsockets_x64-osx"
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/lib/pkgconfig/libzmq.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/zeromq_x64-osx
  2: exec_prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/zeromq_x64-osx
  3: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/zeromq_x64-osx/lib
  4: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/zeromq_x64-osx/include
  5  

vcpkg_installed/x64-osx/lib/pkgconfig/msgpack.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/msgpack_x64-osx
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/lib/pkgconfig/msgpack11.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/msgpack11_x64-osx
  2  libdir=${prefix}/lib

vcpkg_installed/x64-osx/lib/pkgconfig/mt32emu.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libmt32emu_x64-osx
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/lib/pkgconfig/muparser.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/muparser_x64-osx
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/lib/pkgconfig/nanoflann.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/nanoflann_x64-osx
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/lib/pkgconfig/nanomsg.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/nanomsg_x64-osx
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/lib/pkgconfig/nlopt.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/nlopt_x64-osx
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/lib/pkgconfig/octomap.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/octomap_x64-osx
  2: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/octomap_x64-osx/lib
  3: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/octomap_x64-osx/include
  4  

vcpkg_installed/x64-osx/lib/pkgconfig/ode.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/ode_x64-osx
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/lib/pkgconfig/OIS.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/ois_x64-osx
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/lib/pkgconfig/oniguruma.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/oniguruma_x64-osx
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/lib/pkgconfig/opencc.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/opencc_x64-osx
  2  exec_prefix=${prefix}
  3: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/opencc_x64-osx/lib/
  4: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/opencc_x64-osx/include/
  5  

vcpkg_installed/x64-osx/lib/pkgconfig/OpenImageIO.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/openimageio_x64-osx
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/lib/pkgconfig/orocos_kdl.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/orocos-kdl_x64-osx
  2  libdir=${prefix}/lib

vcpkg_installed/x64-osx/lib/pkgconfig/orocos-kdl.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/orocos-kdl_x64-osx
  2  libdir=${prefix}/lib

vcpkg_installed/x64-osx/lib/pkgconfig/parquet.pc:
  17  
  18: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/arrow_x64-osx
  19  libdir=${prefix}/lib

vcpkg_installed/x64-osx/lib/pkgconfig/pcl_2d-1.11.pc:
  1  # This file was generated by CMake for PCL library pcl_2d
  2: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/pcl_x64-osx
  3  exec_prefix=${prefix}

vcpkg_installed/x64-osx/lib/pkgconfig/pcl_common-1.11.pc:
  1  # This file was generated by CMake for PCL library pcl_common
  2: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/pcl_x64-osx
  3  exec_prefix=${prefix}

vcpkg_installed/x64-osx/lib/pkgconfig/pcl_features-1.11.pc:
  1  # This file was generated by CMake for PCL library pcl_features
  2: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/pcl_x64-osx
  3  exec_prefix=${prefix}

vcpkg_installed/x64-osx/lib/pkgconfig/pcl_filters-1.11.pc:
  1  # This file was generated by CMake for PCL library pcl_filters
  2: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/pcl_x64-osx
  3  exec_prefix=${prefix}

vcpkg_installed/x64-osx/lib/pkgconfig/pcl_geometry-1.11.pc:
  1  # This file was generated by CMake for PCL library pcl_geometry
  2: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/pcl_x64-osx
  3  exec_prefix=${prefix}

vcpkg_installed/x64-osx/lib/pkgconfig/pcl_io-1.11.pc:
  1  # This file was generated by CMake for PCL library pcl_io
  2: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/pcl_x64-osx
  3  exec_prefix=${prefix}

vcpkg_installed/x64-osx/lib/pkgconfig/pcl_kdtree-1.11.pc:
  1  # This file was generated by CMake for PCL library pcl_kdtree
  2: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/pcl_x64-osx
  3  exec_prefix=${prefix}

vcpkg_installed/x64-osx/lib/pkgconfig/pcl_keypoints-1.11.pc:
  1  # This file was generated by CMake for PCL library pcl_keypoints
  2: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/pcl_x64-osx
  3  exec_prefix=${prefix}

vcpkg_installed/x64-osx/lib/pkgconfig/pcl_ml-1.11.pc:
  1  # This file was generated by CMake for PCL library pcl_ml
  2: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/pcl_x64-osx
  3  exec_prefix=${prefix}

vcpkg_installed/x64-osx/lib/pkgconfig/pcl_octree-1.11.pc:
  1  # This file was generated by CMake for PCL library pcl_octree
  2: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/pcl_x64-osx
  3  exec_prefix=${prefix}

vcpkg_installed/x64-osx/lib/pkgconfig/pcl_recognition-1.11.pc:
  1  # This file was generated by CMake for PCL library pcl_recognition
  2: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/pcl_x64-osx
  3  exec_prefix=${prefix}

vcpkg_installed/x64-osx/lib/pkgconfig/pcl_registration-1.11.pc:
  1  # This file was generated by CMake for PCL library pcl_registration
  2: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/pcl_x64-osx
  3  exec_prefix=${prefix}

vcpkg_installed/x64-osx/lib/pkgconfig/pcl_sample_consensus-1.11.pc:
  1  # This file was generated by CMake for PCL library pcl_sample_consensus
  2: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/pcl_x64-osx
  3  exec_prefix=${prefix}

vcpkg_installed/x64-osx/lib/pkgconfig/pcl_search-1.11.pc:
  1  # This file was generated by CMake for PCL library pcl_search
  2: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/pcl_x64-osx
  3  exec_prefix=${prefix}

vcpkg_installed/x64-osx/lib/pkgconfig/pcl_segmentation-1.11.pc:
  1  # This file was generated by CMake for PCL library pcl_segmentation
  2: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/pcl_x64-osx
  3  exec_prefix=${prefix}

vcpkg_installed/x64-osx/lib/pkgconfig/pcl_stereo-1.11.pc:
  1  # This file was generated by CMake for PCL library pcl_stereo
  2: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/pcl_x64-osx
  3  exec_prefix=${prefix}

vcpkg_installed/x64-osx/lib/pkgconfig/pcl_surface-1.11.pc:
  1  # This file was generated by CMake for PCL library pcl_surface
  2: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/pcl_x64-osx
  3  exec_prefix=${prefix}

vcpkg_installed/x64-osx/lib/pkgconfig/pcl_tracking-1.11.pc:
  1  # This file was generated by CMake for PCL library pcl_tracking
  2: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/pcl_x64-osx
  3  exec_prefix=${prefix}

vcpkg_installed/x64-osx/lib/pkgconfig/pcre2.pc:
  2  
  3: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/bde_x64-osx
  4  exec_prefix=

vcpkg_installed/x64-osx/lib/pkgconfig/plplot-c++.pc:
  1  libdir=lib
  2: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/plplot_x64-osx/include/plplot
  3  drvdir=lib/plplot5.13.0/drivers

vcpkg_installed/x64-osx/lib/pkgconfig/plplot-fortran.pc:
  1  libdir=lib
  2: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/plplot_x64-osx/include/plplot
  3  drvdir=lib/plplot5.13.0/drivers

vcpkg_installed/x64-osx/lib/pkgconfig/plplot.pc:
  1  libdir=lib
  2: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/plplot_x64-osx/include/plplot
  3  drvdir=lib/plplot5.13.0/drivers

vcpkg_installed/x64-osx/lib/pkgconfig/poppler-cpp.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/poppler_x64-osx
  2: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/poppler_x64-osx/lib
  3: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/poppler_x64-osx/include
  4  

vcpkg_installed/x64-osx/lib/pkgconfig/poppler.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/poppler_x64-osx
  2: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/poppler_x64-osx/lib
  3: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/poppler_x64-osx/include
  4  

vcpkg_installed/x64-osx/lib/pkgconfig/portaudio-2.0.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/portaudio_x64-osx
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/lib/pkgconfig/ppconsul.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/ppconsul_x64-osx
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/lib/pkgconfig/pubsub_client.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/qca2.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/qca_x64-osx
  2: exec_prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/qca_x64-osx
  3: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/qca_x64-osx/lib
  4: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/qca_x64-osx/include/QtCrypto
  5  

vcpkg_installed/x64-osx/lib/pkgconfig/raylib.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/raylib_x64-osx
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/lib/pkgconfig/rbdl.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/rbdl_x64-osx
  2: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/rbdl_x64-osx/lib
  3: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/rbdl_x64-osx/include
  4  

vcpkg_installed/x64-osx/lib/pkgconfig/rdkafka-static.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/librdkafka_x64-osx
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/lib/pkgconfig/rdkafka++-static.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/librdkafka_x64-osx
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/lib/pkgconfig/realsense2.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/realsense2_x64-osx
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/lib/pkgconfig/redis++.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/redis-plus-plus_x64-osx
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/lib/pkgconfig/rpclib.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/rpclib_x64-osx
  2  exec_prefix=
  3: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/rpclib_x64-osx/lib
  4: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/rpclib_x64-osx/include
  5  

vcpkg_installed/x64-osx/lib/pkgconfig/rtaudio.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/rtaudio_x64-osx
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/lib/pkgconfig/sdformat.pc:
  1: prefix="/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/sdformat6_x64-osx"
  2  libdir=${prefix}/lib

vcpkg_installed/x64-osx/lib/pkgconfig/sentencepiece.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/sentencepiece_x64-osx
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/lib/pkgconfig/simage.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/simage_x64-osx
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/lib/pkgconfig/soxr.pc:
  3  Version: 0.1.3
  4: Libs: -L/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/soxr_x64-osx/lib -lsoxr
  5: Cflags: -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/soxr_x64-osx/include

vcpkg_installed/x64-osx/lib/pkgconfig/spanner_client.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/srt.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libsrt_x64-osx
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/lib/pkgconfig/storage_client.pc:
  14  
  15: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/google-cloud-cpp_x64-osx
  16  exec_prefix=${prefix}/bin

vcpkg_installed/x64-osx/lib/pkgconfig/stxxl.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/stxxl_x64-osx
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/lib/pkgconfig/superlu.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/superlu_x64-osx
  2: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/superlu_x64-osx/lib
  3: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/superlu_x64-osx/include
  4  

vcpkg_installed/x64-osx/lib/pkgconfig/thrift-nb.pc:
  19  
  20: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/thrift_x64-osx
  21: exec_prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/thrift_x64-osx/bin
  22: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/thrift_x64-osx/lib
  23: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/thrift_x64-osx/include
  24  

vcpkg_installed/x64-osx/lib/pkgconfig/thrift-z.pc:
  19  
  20: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/thrift_x64-osx
  21: exec_prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/thrift_x64-osx/bin
  22: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/thrift_x64-osx/lib
  23: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/thrift_x64-osx/include
  24  

vcpkg_installed/x64-osx/lib/pkgconfig/thrift.pc:
  19  
  20: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/thrift_x64-osx
  21: exec_prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/thrift_x64-osx/bin
  22: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/thrift_x64-osx/lib
  23: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/thrift_x64-osx/include
  24  

vcpkg_installed/x64-osx/lib/pkgconfig/usrsctp.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/usrsctp_x64-osx
  2: exec_prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/usrsctp_x64-osx
  3: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/usrsctp_x64-osx/lib
  4: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/usrsctp_x64-osx/include/usrsctp
  5  

vcpkg_installed/x64-osx/lib/pkgconfig/xerces-c.pc:
   1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/xerces-c_x64-osx
   2: exec_prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/xerces-c_x64-osx
   3: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/xerces-c_x64-osx/lib
   4: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/xerces-c_x64-osx/include
   5  

   8  Version: 3.2.3
   9: Libs: -L/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/xerces-c_x64-osx/lib -lxerces-c
  10  Libs.private: 
  11: Cflags: -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/xerces-c_x64-osx/include

vcpkg_installed/x64-osx/lib/pkgconfig/xlsxwriter.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libxlsxwriter_x64-osx
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/lib/pkgconfig/zzipfseeko.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/zzipfseeko_x64-osx
  2  libdir=${prefix}/lib

vcpkg_installed/x64-osx/lib/pkgconfig/zziplib.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/zziplib_x64-osx
  2  libdir=${prefix}/lib

vcpkg_installed/x64-osx/lib/pkgconfig/zzipmmapped.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/zzipmmapped_x64-osx
  2  libdir=${prefix}/lib

vcpkg_installed/x64-osx/lib/systemd/user/plasma-kglobalaccel.service:
  5  [Service]
  6: ExecStart=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/kf5globalaccel_x64-osx/bin/kglobalaccel5
  7  BusName=org.kde.kglobalaccel

vcpkg_installed/x64-osx/libdata/pkgconfig/openmesh.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/openmesh_x64-osx
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/pkgconfig/quill.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/quill_x64-osx
  2: exec_prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/quill_x64-osx
  3  

vcpkg_installed/x64-osx/share/ace/ace-devel.sh:
  1: export ACE_ROOT=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/ace_x64-osx/share/ace

vcpkg_installed/x64-osx/share/ace/bin/MakeProjectCreator/config/MPC.cfg:
  1: command_line = -value_project libpaths+=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/ace_x64-osx/lib
  2  dynamic_types = $ACE_ROOT/bin/MakeProjectCreator, $?DDS_ROOT/MPC, $?TAO_ROOT/MPC

vcpkg_installed/x64-osx/share/ace/include/makeinclude/wrapper_macros.GNU:
  200  ifeq (cmd,$(findstring cmd,$(SHELL)))
  201:   ACE_MAJOR_VERSION := $(shell awk "/ACE_MAJOR_VERSION/ { print $$3}" /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/ace_x64-osx/include/ace/Version.h)
  202:   ACE_MINOR_VERSION := $(shell awk "/ACE_MINOR_VERSION/ { print $$3}" /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/ace_x64-osx/include/ace/Version.h)
  203:   ACE_MICRO_VERSION := $(shell awk "/ACE_MICRO_VERSION/ { print $$3}" /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/ace_x64-osx/include/ace/Version.h)
  204:   ACE_BETA_VERSION := $(shell awk "/ACE_MICRO_VERSION/ { print $$3}" /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/ace_x64-osx/include/ace/Version.h)
  205  else
  206:   ACE_MAJOR_VERSION := $(shell awk '/ACE_MAJOR_VERSION/ { print $$3}' /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/ace_x64-osx/include/ace/Version.h)
  207:   ACE_MINOR_VERSION := $(shell awk '/ACE_MINOR_VERSION/ { print $$3}' /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/ace_x64-osx/include/ace/Version.h)
  208:   ACE_MICRO_VERSION := $(shell awk '/ACE_MICRO_VERSION/ { print $$3}' /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/ace_x64-osx/include/ace/Version.h)
  209:   ACE_BETA_VERSION := $(shell awk '/ACE_MICRO_VERSION/ { print $$3}' /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/ace_x64-osx/include/ace/Version.h)
  210  endif

  287  
  288: ifeq (,$(findstring -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/ace_x64-osx/include,$(INCLDIRS)))
  289:   INCLDIRS        += -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/ace_x64-osx/include
  290  endif

  403  # on components built in another makefile and written a file (that might no
  404: # longer be available) and accessed by /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/ace_x64-osx/include/bin/ace_components.
  405  #

vcpkg_installed/x64-osx/share/Bcg729/cmake/Bcg729Config.cmake:
  73  
  74: list(INSERT BCG729_INCLUDE_DIRS 0 "/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/bcg729_x64-osx/include")
  75  

vcpkg_installed/x64-osx/share/cgns/cgnsBuild.defs:
  30  
  31: CGNS_LIBDIR     = /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/cgns_x64-osx/lib
  32: CGNS_INCLUDEDIR = /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/cgns_x64-osx/include
  33  

vcpkg_installed/x64-osx/share/Coin/conf/simage-default.cfg:
  1  # prefix is determined in simage-config script, making the package "movable"
  2: # prefix="/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/simage_x64-osx"
  3  exec_prefix="${prefix}"

vcpkg_installed/x64-osx/share/dbus-1/services/org.kde.kglobalaccel.service:
  2  Name=org.kde.kglobalaccel
  3: Exec=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/kf5globalaccel_x64-osx/bin/kglobalaccel5
  4  SystemdService=plasma-kglobalaccel.service

vcpkg_installed/x64-osx/share/etcdcpp/etcdcppConfig.cmake:
  55  set_target_properties(etcdcpp PROPERTIES
  56:   INTERFACE_INCLUDE_DIRECTORIES "/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/offscale-libetcd-cpp_x64-osx/include"
  57    INTERFACE_LINK_LIBRARIES "\$<LINK_ONLY:gRPC::gpr>;\$<LINK_ONLY:gRPC::grpc>;\$<LINK_ONLY:gRPC::grpc++>;\$<LINK_ONLY:gRPC::grpc++_alts>;\$<LINK_ONLY:protobuf::libprotoc>;\$<LINK_ONLY:protobuf::libprotobuf>"

vcpkg_installed/x64-osx/share/ftgl/FTGL/FTGLConfig.cmake:
  2  
  3: set(FTGL_INCLUDE_DIR "/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/ftgl_x64-osx/include")
  4  set(FTGL_LIBRARY     ftgl)

vcpkg_installed/x64-osx/share/gdb/auto-load/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/glib_x64-osx/lib/libglib-2.0.so.0.6600.4-gdb.py:
  4  # Update module path.
  5: dir_ = '/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/glib_x64-osx/share/glib-2.0/gdb'
  6  if not dir_ in sys.path:

vcpkg_installed/x64-osx/share/gdb/auto-load/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/glib_x64-osx/lib/libgobject-2.0.so.0.6600.4-gdb.py:
  4  # Update module path.
  5: dir_ = '/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/glib_x64-osx/share/glib-2.0/gdb'
  6  if not dir_ in sys.path:

vcpkg_installed/x64-osx/share/ignition/msgs1.yaml:
  4  library_version: 1.0.0
  5: library_path: /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/ignition-msgs1_x64-osx/lib/ruby/ignition/cmdmsgs1
  6  commands:

vcpkg_installed/x64-osx/share/ignition/sdformat6.yaml:
  4  library_version: 6.2.0
  5: library_path: /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/sdformat6_x64-osx/lib/ruby/ignition/cmdsdformat6
  6  commands:

vcpkg_installed/x64-osx/share/ignition/transport4.yaml:
  4  library_version: 4.0.0
  5: library_path: /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/ignition-transport4_x64-osx/lib/ruby/ignition/cmdtransport4
  6  commands:

vcpkg_installed/x64-osx/share/ismrmrd/cmake/ISMRMRDConfig.cmake:
  38  #   ISMRMRD_SCHEMA_DIR   - where to find ismrmrd.xsd 
  39: set(ISMRMRD_SCHEMA_DIR   "/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/ismrmrd_x64-osx/debug/share/ismrmrd/schema")
  40  #   ISMRMRD_INCLUDE_DIR  - where to find ismrmrd.h, etc.
  41: set(ISMRMRD_INCLUDE_DIRS "/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/ismrmrd_x64-osx/debug/include")
  42  #   ISMRMRD_LIBRARY_DIRS - where to search for libraries
  43: set(ISMRMRD_LIBRARY_DIRS "/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/ismrmrd_x64-osx/debug/lib")
  44  #   ISMRMRD_LIBRARIES    - i.e. ismrmrd

  51  ## to conform with naming schemes in many other cmake packages
  52: set(ISMRMRD_INCLUDE_DIR  "/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/ismrmrd_x64-osx/debug/include")
  53: set(ISMRMRD_LIB_DIR "/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/ismrmrd_x64-osx/debug/lib")
  54  

vcpkg_installed/x64-osx/share/libavif/libavif-config.cmake:
  56    INTERFACE_INCLUDE_DIRECTORIES "${_IMPORT_PREFIX}/include"
  57:   INTERFACE_LINK_LIBRARIES "${_IMPORT_PREFIX}/lib/libaom.a;-L/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/aom_x64-osx/lib;\$<LINK_ONLY:-laom>;\$<LINK_ONLY:m>;\$<LINK_ONLY:Threads::Threads>;${_IMPORT_PREFIX}/lib/libyuv.a"
  58  )

vcpkg_installed/x64-osx/share/log4cpp/orocos-log4cpp.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/log4cpp_x64-osx
  2  exec_prefix=${prefix}

vcpkg_installed/x64-osx/share/minitrace/minitraceConfig-release.cmake:
  11    IMPORTED_LINK_INTERFACE_LANGUAGES_RELEASE "C"
  12:   IMPORTED_LOCATION_RELEASE "/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/minitrace_x64-osx/lib/libminitrace.a"
  13    )

  15  list(APPEND _IMPORT_CHECK_TARGETS minitrace::minitrace )
  16: list(APPEND _IMPORT_CHECK_FILES_FOR_minitrace::minitrace "/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/minitrace_x64-osx/lib/libminitrace.a" )
  17  

vcpkg_installed/x64-osx/share/minitrace/minitraceConfig.cmake:
  44  # The installation prefix configured by this project.
  45: set(_IMPORT_PREFIX "/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/minitrace_x64-osx")
  46  

vcpkg_installed/x64-osx/share/mongo-c-driver/uninstall.sh:
   17  save_pwd=$(pwd)
   18: cd /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/mongo-c-driver_x64-osx/
   19  

  176  cd ..
  177: printf "Removing top-level installation directory: "/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/mongo-c-driver_x64-osx/""
  178: (rmdir "/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/mongo-c-driver_x64-osx/" 2>/dev/null && printf "\n") || printf " ... not removed (probably not empty)\n"
  179  

vcpkg_installed/x64-osx/share/pkgconfig/aricpp.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/aricpp_x64-osx
  2  includedir=${prefix}/include

vcpkg_installed/x64-osx/share/pkgconfig/cli.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/cli_x64-osx
  2  includedir=${prefix}/include

vcpkg_installed/x64-osx/share/pkgconfig/cppcodec-1.pc:
  1: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/cppcodec_x64-osx/include
  2  

vcpkg_installed/x64-osx/share/pkgconfig/easyloggingpp.pc:
  3  Version: 9.96.7
  4: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/easyloggingpp_x64-osx
  5: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/easyloggingpp_x64-osx/include
  6  Cflags: -I${includedir}

vcpkg_installed/x64-osx/share/pkgconfig/polyclipping.pc:
  1: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/polyclipping_x64-osx
  2: exec_prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/polyclipping_x64-osx
  3: libdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/polyclipping_x64-osx/lib
  4: sharedlibdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/polyclipping_x64-osx/lib
  5: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/polyclipping_x64-osx/include/polyclipping
  6  

vcpkg_installed/x64-osx/share/pkgconfig/sol2.pc:
  1: includedir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/sol2_x64-osx/include
  2  

vcpkg_installed/x64-osx/share/plplot/examples/plplot-test-interactive.sh:
  334  	tclsh <<EOF
  335: lappend auto_path "/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/plplot_x64-osx/share/plplot"
  336  package require Pltcl

  455  	wish <<EOF
  456: lappend auto_path "/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/plplot_x64-osx/share/plplot"
  457  package require Pltk

  526  	wish <<EOF
  527: lappend auto_path "/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/plplot_x64-osx/share/plplot"
  528  package require Plplotter

vcpkg_installed/x64-osx/share/plplot/examples/cmake/modules/plplot_configure.cmake:
   20  
   21: set(INCLUDE_DIR "/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/plplot_x64-osx/include/plplot")
   22: set(CMAKE_PREFIX_PATH "/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/plplot_x64-osx")
   23  set(MATH_LIB /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.3.sdk/usr/lib/libm.tbd)

  139    # Needed to configure tclsh_standard_examples and wish_standard_examples
  140:   set(DATA_DIR "/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/plplot_x64-osx/share/plplot")
  141  endif(ENABLE_tcl OR ENABLE_tk)

vcpkg_installed/x64-osx/share/plplot/examples/fortran/Makefile:
  80  .f90$(EXEEXT):
  81: 	$(Fortran)  $< -o $@ $(FORTRAN_RPATHCMD) $(shell $(PKG_CONFIG_ENV) pkg-config --static --cflags --libs plplot-fortran) -lplfortrandemolib -I"/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/plplot_x64-osx/share/plplot"/examples/fortran
  82  

vcpkg_installed/x64-osx/share/protopuf/protopufConfig.cmake:
  44  # The installation prefix configured by this project.
  45: set(_IMPORT_PREFIX "/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/protopuf_x64-osx")
  46  

vcpkg_installed/x64-osx/share/qca/cmake/QcaTargets-debug.cmake:
  11    IMPORTED_LINK_INTERFACE_LANGUAGES_DEBUG "CXX"
  12:   IMPORTED_LOCATION_DEBUG "/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/qca_x64-osx/debug/lib/libqca_debug.a"
  13    )

  15  list(APPEND _IMPORT_CHECK_TARGETS qca )
  16: list(APPEND _IMPORT_CHECK_FILES_FOR_qca "/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/qca_x64-osx/debug/lib/libqca_debug.a" )
  17  

vcpkg_installed/x64-osx/share/shp/export_shp.cmake:
  44  # The installation prefix configured by this project.
  45: set(_IMPORT_PREFIX "/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/shapelib_x64-osx")
  46  

vcpkg_installed/x64-osx/share/thumbnailers/gdk-pixbuf-thumbnailer.thumbnailer:
  1  [Thumbnailer Entry]
  2: TryExec=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/gdk-pixbuf_x64-osx/bin/gdk-pixbuf-thumbnailer
  3: Exec=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/gdk-pixbuf_x64-osx/bin/gdk-pixbuf-thumbnailer -s %s %u %o
  4  MimeType=application/x-navi-animation;image/png;image/bmp;image/x-bmp;image/x-MS-bmp;image/gif;image/x-icon;image/x-ico;image/x-win-bitmap;image/vnd.microsoft.icon;application/ico;image/ico;image/icon;text/ico;image/jpeg;image/x-portable-anymap;image/x-portable-bitmap;image/x-portable-graymap;image/x-portable-pixmap;image/tiff;image/x-xpixmap;image/x-xbitmap;image/x-tga;image/x-icns;image/x-quicktime;image/qtif;

vcpkg_installed/x64-osx/tools/glib/glib-gettextize:
  43  # Directory where the sources are stored.
  44: prefix=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/glib_x64-osx
  45  case `uname` in

  50  
  51: datarootdir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/glib_x64-osx/share
  52: datadir=/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/glib_x64-osx/share
  53  

vcpkg_installed/x64-osx/tools/libpng/libpng-config:
  14  version="1.6.37"
  15: prefix="/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libpng_x64-osx"
  16: exec_prefix="/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libpng_x64-osx"
  17: libdir="/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libpng_x64-osx/lib"
  18: includedir="/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libpng_x64-osx/include/libpng16"
  19  libs="-lpng16"

vcpkg_installed/x64-osx/tools/libpng/libpng16-config:
  14  version="1.6.37"
  15: prefix="/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libpng_x64-osx"
  16: exec_prefix="/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libpng_x64-osx"
  17: libdir="/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libpng_x64-osx/lib"
  18: includedir="/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/libpng_x64-osx/include/libpng16"
  19  libs="-lpng16"

```
</details>

Reply:
Or `grep -r -l -I "/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages" vcpkg_installed`
<details><summary>Result</summary>

```
vcpkg_installed/x64-osx/tools/libpng/libpng16-config
vcpkg_installed/x64-osx/tools/libpng/libpng-config
vcpkg_installed/x64-osx/tools/glib/glib-gettextize
vcpkg_installed/x64-osx/pkgconfig/quill.pc
vcpkg_installed/x64-osx/libdata/pkgconfig/openmesh.pc
vcpkg_installed/x64-osx/bin/hpxcxx
vcpkg_installed/x64-osx/include/botan/build.h
vcpkg_installed/x64-osx/include/symengine/symengine_config_cling.h
vcpkg_installed/x64-osx/include/lager/resources_path.hpp
vcpkg_installed/x64-osx/include/proton/version.h
vcpkg_installed/x64-osx/include/dart/config.hpp
vcpkg_installed/x64-osx/include/sdformat-6.2/sdf/sdf_config.h
vcpkg_installed/x64-osx/include/KF5/KWindowSystem/config-kwindowsystem.h
vcpkg_installed/x64-osx/include/dcmtk/config/osconfig.h
vcpkg_installed/x64-osx/include/gdcmConfigure.h
vcpkg_installed/x64-osx/include/lws_config.h
vcpkg_installed/x64-osx/etc/fonts/fonts.conf
vcpkg_installed/x64-osx/etc/fonts/conf.d/README
vcpkg_installed/x64-osx/gdk-pixbuf-2.0/2.10.0/loaders.cache
vcpkg_installed/x64-osx/lib/glibmm-2.68/proc/gmmproc
vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_grpc_utils.pc
vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_longrunning_operations_protos.pc
vcpkg_installed/x64-osx/lib/pkgconfig/googleapis_cpp_cloud_dialogflow_v2_protos.pc
vcpkg_installed/x64-osx/lib/pkgconfig/poppler.pc
vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_type_dayofweek_protos.pc
vcpkg_installed/x64-osx/lib/pkgconfig/libtins.pc
vcpkg_installed/x64-osx/lib/pkgconfig/googleapis_cpp_api_http_protos.pc
vcpkg_installed/x64-osx/lib/pkgconfig/pcl_2d-1.11.pc
vcpkg_installed/x64-osx/lib/pkgconfig/ACE_ETCL.pc
vcpkg_installed/x64-osx/lib/pkgconfig/msgpack.pc
vcpkg_installed/x64-osx/lib/pkgconfig/libqrencode.pc
vcpkg_installed/x64-osx/lib/pkgconfig/pubsub_client.pc
vcpkg_installed/x64-osx/lib/pkgconfig/googleapis_cpp_api_annotations_protos.pc
vcpkg_installed/x64-osx/lib/pkgconfig/zzipmmapped.pc
vcpkg_installed/x64-osx/lib/pkgconfig/libbsoncxx-static.pc
vcpkg_installed/x64-osx/lib/pkgconfig/googleapis_cpp_type_dayofweek_protos.pc
vcpkg_installed/x64-osx/lib/pkgconfig/redis++.pc
vcpkg_installed/x64-osx/lib/pkgconfig/googleapis_cpp_api_auth_protos.pc
vcpkg_installed/x64-osx/lib/pkgconfig/fluidlite.pc
vcpkg_installed/x64-osx/lib/pkgconfig/pcl_features-1.11.pc
vcpkg_installed/x64-osx/lib/pkgconfig/libsais64.pc
vcpkg_installed/x64-osx/lib/pkgconfig/inteldfp.pc
vcpkg_installed/x64-osx/lib/pkgconfig/orocos_kdl.pc
vcpkg_installed/x64-osx/lib/pkgconfig/libbson-1.0.pc
vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_rpc_status_protos.pc
vcpkg_installed/x64-osx/lib/pkgconfig/libopcuaserver.pc
vcpkg_installed/x64-osx/lib/pkgconfig/libmongoc-1.0.pc
vcpkg_installed/x64-osx/lib/pkgconfig/googleapis_cpp_api_metric_protos.pc
vcpkg_installed/x64-osx/lib/pkgconfig/googleapis_cpp_api_label_protos.pc
vcpkg_installed/x64-osx/lib/pkgconfig/Cutelyst2Qt5UserAgent.pc
vcpkg_installed/x64-osx/lib/pkgconfig/googleapis_cpp_type_fraction_protos.pc
vcpkg_installed/x64-osx/lib/pkgconfig/libpmemobj++.pc
vcpkg_installed/x64-osx/lib/pkgconfig/googleapis_cpp_api_resource_protos.pc
vcpkg_installed/x64-osx/lib/pkgconfig/pcl_surface-1.11.pc
vcpkg_installed/x64-osx/lib/pkgconfig/grpc.pc
vcpkg_installed/x64-osx/lib/pkgconfig/nlopt.pc
vcpkg_installed/x64-osx/lib/pkgconfig/libcpuid.pc
vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_bigtable_protos.pc
vcpkg_installed/x64-osx/lib/pkgconfig/Cutelyst2Qt5WSGI.pc
vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_api_launch_stage_protos.pc
vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_pubsub_protos.pc
vcpkg_installed/x64-osx/lib/pkgconfig/libproxy-1.0.pc
vcpkg_installed/x64-osx/lib/pkgconfig/gflags.pc
vcpkg_installed/x64-osx/lib/pkgconfig/liburiparser.pc
vcpkg_installed/x64-osx/lib/pkgconfig/googleapis_cpp_type_calendar_period_protos.pc
vcpkg_installed/x64-osx/lib/pkgconfig/geogram1.pc
vcpkg_installed/x64-osx/lib/pkgconfig/arrow-csv.pc
vcpkg_installed/x64-osx/lib/pkgconfig/googleapis_cpp_type_expr_protos.pc
vcpkg_installed/x64-osx/lib/pkgconfig/plplot-c++.pc
vcpkg_installed/x64-osx/lib/pkgconfig/realsense2.pc
vcpkg_installed/x64-osx/lib/pkgconfig/blitz.pc
vcpkg_installed/x64-osx/lib/pkgconfig/amqpcpp.pc
vcpkg_installed/x64-osx/lib/pkgconfig/cmocka.pc
vcpkg_installed/x64-osx/lib/pkgconfig/gpr.pc
vcpkg_installed/x64-osx/lib/pkgconfig/OpenImageIO.pc
vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_type_phone_number_protos.pc
vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_api_annotations_protos.pc
vcpkg_installed/x64-osx/lib/pkgconfig/rbdl.pc
vcpkg_installed/x64-osx/lib/pkgconfig/ACE.pc
vcpkg_installed/x64-osx/lib/pkgconfig/storage_client.pc
vcpkg_installed/x64-osx/lib/pkgconfig/pcl_ml-1.11.pc
vcpkg_installed/x64-osx/lib/pkgconfig/kj-async.pc
vcpkg_installed/x64-osx/lib/pkgconfig/check.pc
vcpkg_installed/x64-osx/lib/pkgconfig/bigtable_client.pc
vcpkg_installed/x64-osx/lib/pkgconfig/parquet.pc
vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_spanner_protos.pc
vcpkg_installed/x64-osx/lib/pkgconfig/libmosquittopp.pc
vcpkg_installed/x64-osx/lib/pkgconfig/portaudio-2.0.pc
vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_api_resource_protos.pc
vcpkg_installed/x64-osx/lib/pkgconfig/grpc++_unsecure.pc
vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_logging_protos.pc
vcpkg_installed/x64-osx/lib/pkgconfig/capnp-rpc.pc
vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_iam_v1_iam_policy_protos.pc
vcpkg_installed/x64-osx/lib/pkgconfig/fftw3f.pc
vcpkg_installed/x64-osx/lib/pkgconfig/googleapis_cpp_type_interval_protos.pc
vcpkg_installed/x64-osx/lib/pkgconfig/kj-http.pc
vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_cloud_dialogflow_v2_protos.pc
vcpkg_installed/x64-osx/lib/pkgconfig/grpc++.pc
vcpkg_installed/x64-osx/lib/pkgconfig/libKF5Attica.pc
vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_api_client_protos.pc
vcpkg_installed/x64-osx/lib/pkgconfig/functions_framework_cpp.pc
vcpkg_installed/x64-osx/lib/pkgconfig/glut.pc
vcpkg_installed/x64-osx/lib/pkgconfig/googleapis_cpp_cloud_dialogflow_v2beta1_protos.pc
vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_bigquery.pc
vcpkg_installed/x64-osx/lib/pkgconfig/googleapis.pc
vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_api_http_protos.pc
vcpkg_installed/x64-osx/lib/pkgconfig/googleapis_cpp_rpc_error_details_protos.pc
vcpkg_installed/x64-osx/lib/pkgconfig/plplot.pc
vcpkg_installed/x64-osx/lib/pkgconfig/soxr.pc
vcpkg_installed/x64-osx/lib/pkgconfig/geographiclib.pc
vcpkg_installed/x64-osx/lib/pkgconfig/spanner_client.pc
vcpkg_installed/x64-osx/lib/pkgconfig/armadillo.pc
vcpkg_installed/x64-osx/lib/pkgconfig/mt32emu.pc
vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_type_interval_protos.pc
vcpkg_installed/x64-osx/lib/pkgconfig/aom.pc
vcpkg_installed/x64-osx/lib/pkgconfig/googleapis_cpp_monitoring_protos.pc
vcpkg_installed/x64-osx/lib/pkgconfig/json11.pc
vcpkg_installed/x64-osx/lib/pkgconfig/libkml.pc
vcpkg_installed/x64-osx/lib/pkgconfig/libraw.pc
vcpkg_installed/x64-osx/lib/pkgconfig/nanomsg.pc
vcpkg_installed/x64-osx/lib/pkgconfig/liblo.pc
vcpkg_installed/x64-osx/lib/pkgconfig/thrift-nb.pc
vcpkg_installed/x64-osx/lib/pkgconfig/googleapis_cpp_type_localized_text_protos.pc
vcpkg_installed/x64-osx/lib/pkgconfig/googleapis_cpp_api_monitored_resource_protos.pc
vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_api_auth_protos.pc
vcpkg_installed/x64-osx/lib/pkgconfig/bdl.pc
vcpkg_installed/x64-osx/lib/pkgconfig/nanoflann.pc
vcpkg_installed/x64-osx/lib/pkgconfig/octomap.pc
vcpkg_installed/x64-osx/lib/pkgconfig/bsl.pc
vcpkg_installed/x64-osx/lib/pkgconfig/googleapis_cpp_bigtable_protos.pc
vcpkg_installed/x64-osx/lib/pkgconfig/raylib.pc
vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_storage_protos.pc
vcpkg_installed/x64-osx/lib/pkgconfig/Cutelyst2Qt5UtilsValidator.pc
vcpkg_installed/x64-osx/lib/pkgconfig/pcl_tracking-1.11.pc
vcpkg_installed/x64-osx/lib/pkgconfig/msgpack11.pc
vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_api_distribution_protos.pc
vcpkg_installed/x64-osx/lib/pkgconfig/googleapis_cpp_cloud_speech_protos.pc
vcpkg_installed/x64-osx/lib/pkgconfig/poppler-cpp.pc
vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_iam.pc
vcpkg_installed/x64-osx/lib/pkgconfig/libopcuaclient.pc
vcpkg_installed/x64-osx/lib/pkgconfig/flatbuffers.pc
vcpkg_installed/x64-osx/lib/pkgconfig/googleapis_cpp_iam_protos.pc
vcpkg_installed/x64-osx/lib/pkgconfig/googleapis_cpp_type_date_protos.pc
vcpkg_installed/x64-osx/lib/pkgconfig/pcl_search-1.11.pc
vcpkg_installed/x64-osx/lib/pkgconfig/bal.pc
vcpkg_installed/x64-osx/lib/pkgconfig/gdlib.pc
vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_type_fraction_protos.pc
vcpkg_installed/x64-osx/lib/pkgconfig/pcl_recognition-1.11.pc
vcpkg_installed/x64-osx/lib/pkgconfig/libconfuse.pc
vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_type_month_protos.pc
vcpkg_installed/x64-osx/lib/pkgconfig/libsparsehash.pc
vcpkg_installed/x64-osx/lib/pkgconfig/libmediainfo.pc
vcpkg_installed/x64-osx/lib/pkgconfig/hiredis.pc
vcpkg_installed/x64-osx/lib/pkgconfig/blosc.pc
vcpkg_installed/x64-osx/lib/pkgconfig/arrow-json.pc
vcpkg_installed/x64-osx/lib/pkgconfig/lemon.pc
vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_type_localized_text_protos.pc
vcpkg_installed/x64-osx/lib/pkgconfig/gta.pc
vcpkg_installed/x64-osx/lib/pkgconfig/libgit2.pc
vcpkg_installed/x64-osx/lib/pkgconfig/libevent.pc
vcpkg_installed/x64-osx/lib/pkgconfig/libhs.pc
vcpkg_installed/x64-osx/lib/pkgconfig/gmock.pc
vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_type_color_protos.pc
vcpkg_installed/x64-osx/lib/pkgconfig/bbl.pc
vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_type_latlng_protos.pc
vcpkg_installed/x64-osx/lib/pkgconfig/libcjson.pc
vcpkg_installed/x64-osx/lib/pkgconfig/libopcuaprotocol.pc
vcpkg_installed/x64-osx/lib/pkgconfig/decnumber.pc
vcpkg_installed/x64-osx/lib/pkgconfig/kj.pc
vcpkg_installed/x64-osx/lib/pkgconfig/Cutelyst2Qt5StaticSimple.pc
vcpkg_installed/x64-osx/lib/pkgconfig/libevent_extra.pc
vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_cloud_speech_protos.pc
vcpkg_installed/x64-osx/lib/pkgconfig/libmikmod.pc
vcpkg_installed/x64-osx/lib/pkgconfig/OIS.pc
vcpkg_installed/x64-osx/lib/pkgconfig/btl.pc
vcpkg_installed/x64-osx/lib/pkgconfig/brpc.pc
vcpkg_installed/x64-osx/lib/pkgconfig/fftw3l.pc
vcpkg_installed/x64-osx/lib/pkgconfig/arrow.pc
vcpkg_installed/x64-osx/lib/pkgconfig/googleapis_cpp_type_money_protos.pc
vcpkg_installed/x64-osx/lib/pkgconfig/libbddisasm.pc
vcpkg_installed/x64-osx/lib/pkgconfig/graphite2.pc
vcpkg_installed/x64-osx/lib/pkgconfig/haisrt.pc
vcpkg_installed/x64-osx/lib/pkgconfig/googleapis_cpp_type_month_protos.pc
vcpkg_installed/x64-osx/lib/pkgconfig/googleapis_cpp_devtools_cloudtrace_v2_tracing_protos.pc
vcpkg_installed/x64-osx/lib/pkgconfig/libftdipp1.pc
vcpkg_installed/x64-osx/lib/pkgconfig/libmosquitto.pc
vcpkg_installed/x64-osx/lib/pkgconfig/Cutelyst2Qt5Core.pc
vcpkg_installed/x64-osx/lib/pkgconfig/simage.pc
vcpkg_installed/x64-osx/lib/pkgconfig/libmongoc-ssl-1.0.pc
vcpkg_installed/x64-osx/lib/pkgconfig/rdkafka-static.pc
vcpkg_installed/x64-osx/lib/pkgconfig/igraph.pc
vcpkg_installed/x64-osx/lib/pkgconfig/libcerf.pc
vcpkg_installed/x64-osx/lib/pkgconfig/pcl_octree-1.11.pc
vcpkg_installed/x64-osx/lib/pkgconfig/libtorrent-rasterbar.pc
vcpkg_installed/x64-osx/lib/pkgconfig/libftdi1.pc
vcpkg_installed/x64-osx/lib/pkgconfig/pcl_sample_consensus-1.11.pc
vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_type_money_protos.pc
vcpkg_installed/x64-osx/lib/pkgconfig/muparser.pc
vcpkg_installed/x64-osx/lib/pkgconfig/xlsxwriter.pc
vcpkg_installed/x64-osx/lib/pkgconfig/iniparser.pc
vcpkg_installed/x64-osx/lib/pkgconfig/googleapis_cpp_type_color_protos.pc
vcpkg_installed/x64-osx/lib/pkgconfig/pcl_common-1.11.pc
vcpkg_installed/x64-osx/lib/pkgconfig/Cutelyst2Qt5StatusMessage.pc
vcpkg_installed/x64-osx/lib/pkgconfig/thrift.pc
vcpkg_installed/x64-osx/lib/pkgconfig/collada-dom-150.pc
vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_cloud_texttospeech_protos.pc
vcpkg_installed/x64-osx/lib/pkgconfig/xerces-c.pc
vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_monitoring_protos.pc
vcpkg_installed/x64-osx/lib/pkgconfig/collada-dom-141.pc
vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_devtools_cloudtrace_v2_tracing_protos.pc
vcpkg_installed/x64-osx/lib/pkgconfig/orocos-kdl.pc
vcpkg_installed/x64-osx/lib/pkgconfig/libevent_pthreads.pc
vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_rpc_error_details_protos.pc
vcpkg_installed/x64-osx/lib/pkgconfig/stxxl.pc
vcpkg_installed/x64-osx/lib/pkgconfig/Cutelyst2Qt5Session.pc
vcpkg_installed/x64-osx/lib/pkgconfig/capnp-json.pc
vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_devtools_cloudtrace_v2_trace_protos.pc
vcpkg_installed/x64-osx/lib/pkgconfig/Cutelyst2Qt5Authentication.pc
vcpkg_installed/x64-osx/lib/pkgconfig/googleapis_cpp_spanner_protos.pc
vcpkg_installed/x64-osx/lib/pkgconfig/superlu.pc
vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_type_date_protos.pc
vcpkg_installed/x64-osx/lib/pkgconfig/usrsctp.pc
vcpkg_installed/x64-osx/lib/pkgconfig/googleapis_cpp_logging_protos.pc
vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_storage.pc
vcpkg_installed/x64-osx/lib/pkgconfig/rdkafka++-static.pc
vcpkg_installed/x64-osx/lib/pkgconfig/googleapis_cpp_type_latlng_protos.pc
vcpkg_installed/x64-osx/lib/pkgconfig/libevent_core.pc
vcpkg_installed/x64-osx/lib/pkgconfig/pcl_kdtree-1.11.pc
vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_spanner.pc
vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_bigtable.pc
vcpkg_installed/x64-osx/lib/pkgconfig/zzipfseeko.pc
vcpkg_installed/x64-osx/lib/pkgconfig/pcl_registration-1.11.pc
vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_type_calendar_period_protos.pc
vcpkg_installed/x64-osx/lib/pkgconfig/sdformat.pc
vcpkg_installed/x64-osx/lib/pkgconfig/googleapis_cpp_type_datetime_protos.pc
vcpkg_installed/x64-osx/lib/pkgconfig/libwebsockets_static.pc
vcpkg_installed/x64-osx/lib/pkgconfig/googleapis_cpp_devtools_cloudtrace_v2_trace_protos.pc
vcpkg_installed/x64-osx/lib/pkgconfig/ppconsul.pc
vcpkg_installed/x64-osx/lib/pkgconfig/pcl_io-1.11.pc
vcpkg_installed/x64-osx/lib/pkgconfig/plplot-fortran.pc
vcpkg_installed/x64-osx/lib/pkgconfig/googleapis_cpp_api_distribution_protos.pc
vcpkg_installed/x64-osx/lib/pkgconfig/libwebsockets.pc
vcpkg_installed/x64-osx/lib/pkgconfig/edlib-1.pc
vcpkg_installed/x64-osx/lib/pkgconfig/googleapis_cpp_cloud_texttospeech_protos.pc
vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_api_monitored_resource_protos.pc
vcpkg_installed/x64-osx/lib/pkgconfig/curlpp.pc
vcpkg_installed/x64-osx/lib/pkgconfig/googleapis_cpp_iam_v1_policy_protos.pc
vcpkg_installed/x64-osx/lib/pkgconfig/flann.pc
vcpkg_installed/x64-osx/lib/pkgconfig/grpc_unsecure.pc
vcpkg_installed/x64-osx/lib/pkgconfig/capnp.pc
vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_cloud_dialogflow_v2beta1_protos.pc
vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_type_datetime_protos.pc
vcpkg_installed/x64-osx/lib/pkgconfig/Cutelyst2Qt5UtilsSql.pc
vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_cloud_bigquery_protos.pc
vcpkg_installed/x64-osx/lib/pkgconfig/srt.pc
vcpkg_installed/x64-osx/lib/pkgconfig/Cutelyst2Qt5ViewJSON.pc
vcpkg_installed/x64-osx/lib/pkgconfig/thrift-z.pc
vcpkg_installed/x64-osx/lib/pkgconfig/qca2.pc
vcpkg_installed/x64-osx/lib/pkgconfig/evhtp.pc
vcpkg_installed/x64-osx/lib/pkgconfig/charls.pc
vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_iam_v1_policy_protos.pc
vcpkg_installed/x64-osx/lib/pkgconfig/pcre2.pc
vcpkg_installed/x64-osx/lib/pkgconfig/fftw3.pc
vcpkg_installed/x64-osx/lib/pkgconfig/libbcg729.pc
vcpkg_installed/x64-osx/lib/pkgconfig/oniguruma.pc
vcpkg_installed/x64-osx/lib/pkgconfig/blitz-static.pc
vcpkg_installed/x64-osx/lib/pkgconfig/libgraphqlparser.pc
vcpkg_installed/x64-osx/lib/pkgconfig/googleapis_cpp_storage_protos.pc
vcpkg_installed/x64-osx/lib/pkgconfig/googleapis_cpp_cloud_bigquery_protos.pc
vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_type_quaternion_protos.pc
vcpkg_installed/x64-osx/lib/pkgconfig/googleapis_cpp_type_phone_number_protos.pc
vcpkg_installed/x64-osx/lib/pkgconfig/libmongocxx-static.pc
vcpkg_installed/x64-osx/lib/pkgconfig/googleapis_cpp_type_postal_address_protos.pc
vcpkg_installed/x64-osx/lib/pkgconfig/ccd.pc
vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_type_expr_protos.pc
vcpkg_installed/x64-osx/lib/pkgconfig/Cutelyst2Qt5UtilsLangSelect.pc
vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_api_field_behavior_protos.pc
vcpkg_installed/x64-osx/lib/pkgconfig/googleapis_cpp_pubsub_protos.pc
vcpkg_installed/x64-osx/lib/pkgconfig/googleapis_cpp_api_client_protos.pc
vcpkg_installed/x64-osx/lib/pkgconfig/googleapis_cpp_type_timeofday_protos.pc
vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_iam_protos.pc
vcpkg_installed/x64-osx/lib/pkgconfig/googleapis_cpp_iam_v1_iam_policy_protos.pc
vcpkg_installed/x64-osx/lib/pkgconfig/pcl_filters-1.11.pc
vcpkg_installed/x64-osx/lib/pkgconfig/opencc.pc
vcpkg_installed/x64-osx/lib/pkgconfig/libsais.pc
vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_common.pc
vcpkg_installed/x64-osx/lib/pkgconfig/googleapis_cpp_logging_type_protos.pc
vcpkg_installed/x64-osx/lib/pkgconfig/libfolly.pc
vcpkg_installed/x64-osx/lib/pkgconfig/pcl_geometry-1.11.pc
vcpkg_installed/x64-osx/lib/pkgconfig/clblast.pc
vcpkg_installed/x64-osx/lib/pkgconfig/itpp.pc
vcpkg_installed/x64-osx/lib/pkgconfig/ode.pc
vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_type_timeofday_protos.pc
vcpkg_installed/x64-osx/lib/pkgconfig/dlib-1.pc
vcpkg_installed/x64-osx/lib/pkgconfig/pcl_segmentation-1.11.pc
vcpkg_installed/x64-osx/lib/pkgconfig/sentencepiece.pc
vcpkg_installed/x64-osx/lib/pkgconfig/kj-test.pc
vcpkg_installed/x64-osx/lib/pkgconfig/libevent_openssl.pc
vcpkg_installed/x64-osx/lib/pkgconfig/fcl.pc
vcpkg_installed/x64-osx/lib/pkgconfig/pcl_stereo-1.11.pc
vcpkg_installed/x64-osx/lib/pkgconfig/libkeyfinder.pc
vcpkg_installed/x64-osx/lib/pkgconfig/collada-dom.pc
vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_api_label_protos.pc
vcpkg_installed/x64-osx/lib/pkgconfig/libsbml.pc
vcpkg_installed/x64-osx/lib/pkgconfig/arrow-filesystem.pc
vcpkg_installed/x64-osx/lib/pkgconfig/libfort.pc
vcpkg_installed/x64-osx/lib/pkgconfig/zziplib.pc
vcpkg_installed/x64-osx/lib/pkgconfig/googleapis_cpp_longrunning_operations_protos.pc
vcpkg_installed/x64-osx/lib/pkgconfig/Cutelyst2Qt5UtilsPagination.pc
vcpkg_installed/x64-osx/lib/pkgconfig/libopcuacore.pc
vcpkg_installed/x64-osx/lib/pkgconfig/bullet.pc
vcpkg_installed/x64-osx/lib/pkgconfig/cpp_redis.pc
vcpkg_installed/x64-osx/lib/pkgconfig/googleapis_cpp_type_quaternion_protos.pc
vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_type_postal_address_protos.pc
vcpkg_installed/x64-osx/lib/pkgconfig/libheif.pc
vcpkg_installed/x64-osx/lib/pkgconfig/libzmq.pc
vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_iam_v1_options_protos.pc
vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_pubsub.pc
vcpkg_installed/x64-osx/lib/pkgconfig/googleapis_cpp_api_field_behavior_protos.pc
vcpkg_installed/x64-osx/lib/pkgconfig/googleapis_cpp_api_launch_stage_protos.pc
vcpkg_installed/x64-osx/lib/pkgconfig/libraw_r.pc
vcpkg_installed/x64-osx/lib/pkgconfig/Coin.pc
vcpkg_installed/x64-osx/lib/pkgconfig/rtaudio.pc
vcpkg_installed/x64-osx/lib/pkgconfig/csv2.pc
vcpkg_installed/x64-osx/lib/pkgconfig/pcl_keypoints-1.11.pc
vcpkg_installed/x64-osx/lib/pkgconfig/jansson.pc
vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_logging_type_protos.pc
vcpkg_installed/x64-osx/lib/pkgconfig/ACE_ETCL_Parser.pc
vcpkg_installed/x64-osx/lib/pkgconfig/googleapis_cpp_rpc_status_protos.pc
vcpkg_installed/x64-osx/lib/pkgconfig/rpclib.pc
vcpkg_installed/x64-osx/lib/pkgconfig/googleapis_cpp_iam_v1_options_protos.pc
vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_api_metric_protos.pc
vcpkg_installed/x64-osx/lib/pkgconfig/gtest.pc
vcpkg_installed/x64-osx/lib/pkgconfig/google_cloud_cpp_storage_grpc.pc
vcpkg_installed/x64-osx/lib/libgraphite2.la
vcpkg_installed/x64-osx/lib/manual-link/pkgconfig/gmock_main.pc
vcpkg_installed/x64-osx/lib/manual-link/pkgconfig/gtest_main.pc
vcpkg_installed/x64-osx/lib/libhdf5.settings
vcpkg_installed/x64-osx/lib/systemd/user/plasma-kglobalaccel.service
vcpkg_installed/x64-osx/lib/libnetcdf.settings
vcpkg_installed/x64-osx/lib/botan-2.pc
vcpkg_installed/x64-osx/debug/tools/libpng/libpng16-config
vcpkg_installed/x64-osx/debug/tools/libpng/libpng-config
vcpkg_installed/x64-osx/debug/pkgconfig/quill.pc
vcpkg_installed/x64-osx/debug/libdata/pkgconfig/openmesh.pc
vcpkg_installed/x64-osx/debug/bin/hpxcxx
vcpkg_installed/x64-osx/debug/gdk-pixbuf-2.0/2.10.0/loaders.cache
vcpkg_installed/x64-osx/debug/lib/glibmm-2.68/proc/gmmproc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_grpc_utils.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_longrunning_operations_protos.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/googleapis_cpp_cloud_dialogflow_v2_protos.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/poppler.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_type_dayofweek_protos.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/libtins.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/googleapis_cpp_api_http_protos.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/pcl_2d-1.11.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/ACE_ETCL.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/msgpack.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/libqrencode.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/pubsub_client.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/googleapis_cpp_api_annotations_protos.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/zzipmmapped.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/libbsoncxx-static.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/googleapis_cpp_type_dayofweek_protos.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/redis++.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/googleapis_cpp_api_auth_protos.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/fluidlite.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/pcl_features-1.11.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/libsais64.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/inteldfp.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/orocos_kdl.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/libbson-1.0.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_rpc_status_protos.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/libopcuaserver.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/libmongoc-1.0.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/googleapis_cpp_api_metric_protos.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/googleapis_cpp_api_label_protos.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/Cutelyst2Qt5UserAgent.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/googleapis_cpp_type_fraction_protos.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/googleapis_cpp_api_resource_protos.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/pcl_surface-1.11.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/grpc.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/nlopt.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/libcpuid.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_bigtable_protos.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/Cutelyst2Qt5WSGI.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_api_launch_stage_protos.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_pubsub_protos.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/libproxy-1.0.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/gflags.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/liburiparser.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/googleapis_cpp_type_calendar_period_protos.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/geogram1.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/arrow-csv.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/googleapis_cpp_type_expr_protos.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/stxxl_debug.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/plplot-c++.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/realsense2.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/blitz.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/amqpcpp.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/cmocka.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/gpr.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/OpenImageIO.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_type_phone_number_protos.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_api_annotations_protos.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/rbdl.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/ACE.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/storage_client.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/pcl_ml-1.11.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/kj-async.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/check.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/bigtable_client.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/parquet.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_spanner_protos.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/libmosquittopp.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/portaudio-2.0.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_api_resource_protos.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/grpc++_unsecure.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_logging_protos.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/capnp-rpc.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_iam_v1_iam_policy_protos.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/fftw3f.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/googleapis_cpp_type_interval_protos.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/kj-http.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_cloud_dialogflow_v2_protos.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/grpc++.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/libKF5Attica.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_api_client_protos.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/functions_framework_cpp.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/glut.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/googleapis_cpp_cloud_dialogflow_v2beta1_protos.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_bigquery.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/googleapis.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_api_http_protos.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/googleapis_cpp_rpc_error_details_protos.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/plplot.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/soxr.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/geographiclib.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/spanner_client.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/armadillo.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/mt32emu.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_type_interval_protos.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/aom.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/googleapis_cpp_monitoring_protos.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/json11.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/libkml.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/libraw.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/nanomsg.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/liblo.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/thrift-nb.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/googleapis_cpp_type_localized_text_protos.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/googleapis_cpp_api_monitored_resource_protos.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_api_auth_protos.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/bdl.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/nanoflann.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/octomap.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/bsl.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/googleapis_cpp_bigtable_protos.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/raylib.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_storage_protos.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/Cutelyst2Qt5UtilsValidator.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/pcl_tracking-1.11.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/msgpack11.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_api_distribution_protos.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/googleapis_cpp_cloud_speech_protos.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/poppler-cpp.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_iam.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/libopcuaclient.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/flatbuffers.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/googleapis_cpp_iam_protos.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/googleapis_cpp_type_date_protos.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/pcl_search-1.11.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/bal.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/gdlib.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_type_fraction_protos.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/pcl_recognition-1.11.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/libconfuse.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_type_month_protos.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/libmediainfo.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/hiredis.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/blosc.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/arrow-json.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/lemon.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_type_localized_text_protos.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/gta.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/libgit2.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/libevent.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/libhs.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/gmock.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_type_color_protos.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/bbl.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_type_latlng_protos.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/libcjson.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/libopcuaprotocol.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/decnumber.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/kj.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/Cutelyst2Qt5StaticSimple.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/libevent_extra.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_cloud_speech_protos.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/libmikmod.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/OIS.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/btl.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/brpc.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/fftw3l.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/arrow.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/googleapis_cpp_type_money_protos.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/libbddisasm.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/graphite2.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/haisrt.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/googleapis_cpp_type_month_protos.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/googleapis_cpp_devtools_cloudtrace_v2_tracing_protos.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/libftdipp1.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/libmosquitto.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/Cutelyst2Qt5Core.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/simage.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/libmongoc-ssl-1.0.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/rdkafka-static.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/igraph.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/libcerf.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/pcl_octree-1.11.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/libtorrent-rasterbar.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/libftdi1.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/pcl_sample_consensus-1.11.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_type_money_protos.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/muparser.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/xlsxwriter.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/iniparser.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/googleapis_cpp_type_color_protos.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/pcl_common-1.11.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/Cutelyst2Qt5StatusMessage.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/thrift.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/collada-dom-150.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_cloud_texttospeech_protos.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/xerces-c.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_monitoring_protos.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/collada-dom-141.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_devtools_cloudtrace_v2_tracing_protos.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/orocos-kdl.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/libevent_pthreads.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_rpc_error_details_protos.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/Cutelyst2Qt5Session.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/capnp-json.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_devtools_cloudtrace_v2_trace_protos.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/Cutelyst2Qt5Authentication.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/googleapis_cpp_spanner_protos.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/superlu.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_type_date_protos.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/usrsctp.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/googleapis_cpp_logging_protos.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_storage.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/rdkafka++-static.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/googleapis_cpp_type_latlng_protos.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/libevent_core.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/pcl_kdtree-1.11.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_spanner.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_bigtable.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/zzipfseeko.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/pcl_registration-1.11.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_type_calendar_period_protos.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/sdformat.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/googleapis_cpp_type_datetime_protos.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/libwebsockets_static.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/googleapis_cpp_devtools_cloudtrace_v2_trace_protos.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/ppconsul.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/pcl_io-1.11.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/plplot-fortran.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/googleapis_cpp_api_distribution_protos.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/libwebsockets.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/edlib-1.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/googleapis_cpp_cloud_texttospeech_protos.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_api_monitored_resource_protos.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/curlpp.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/googleapis_cpp_iam_v1_policy_protos.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/flann.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/grpc_unsecure.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/capnp.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_cloud_dialogflow_v2beta1_protos.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_type_datetime_protos.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/Cutelyst2Qt5UtilsSql.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_cloud_bigquery_protos.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/srt.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/Cutelyst2Qt5ViewJSON.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/thrift-z.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/qca2.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/evhtp.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/charls.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_iam_v1_policy_protos.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/pcre2.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/fftw3.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/libbcg729.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/oniguruma.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/blitz-static.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/libgraphqlparser.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/googleapis_cpp_storage_protos.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/googleapis_cpp_cloud_bigquery_protos.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_type_quaternion_protos.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/googleapis_cpp_type_phone_number_protos.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/libmongocxx-static.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/googleapis_cpp_type_postal_address_protos.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/ccd.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_type_expr_protos.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/Cutelyst2Qt5UtilsLangSelect.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_api_field_behavior_protos.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/googleapis_cpp_pubsub_protos.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/googleapis_cpp_api_client_protos.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/googleapis_cpp_type_timeofday_protos.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_iam_protos.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/googleapis_cpp_iam_v1_iam_policy_protos.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/pcl_filters-1.11.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/opencc.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/libsais.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_common.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/googleapis_cpp_logging_type_protos.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/libfolly.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/pcl_geometry-1.11.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/clblast.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/itpp.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/ode.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_type_timeofday_protos.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/dlib-1.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/pcl_segmentation-1.11.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/sentencepiece.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/kj-test.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/libevent_openssl.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/fcl.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/pcl_stereo-1.11.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/libkeyfinder.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/collada-dom.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_api_label_protos.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/libsbml.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/arrow-filesystem.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/libfort.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/zziplib.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/googleapis_cpp_longrunning_operations_protos.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/Cutelyst2Qt5UtilsPagination.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/libopcuacore.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/bullet.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/cpp_redis.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/googleapis_cpp_type_quaternion_protos.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_type_postal_address_protos.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/libheif.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/libzmq.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_iam_v1_options_protos.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_pubsub.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/googleapis_cpp_api_field_behavior_protos.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/googleapis_cpp_api_launch_stage_protos.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/libraw_r.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/Coin.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/rtaudio.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/csv2.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/pcl_keypoints-1.11.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/jansson.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_logging_type_protos.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/ACE_ETCL_Parser.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/googleapis_cpp_rpc_status_protos.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/rpclib.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/googleapis_cpp_iam_v1_options_protos.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_api_metric_protos.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/gtest.pc
vcpkg_installed/x64-osx/debug/lib/pkgconfig/google_cloud_cpp_storage_grpc.pc
vcpkg_installed/x64-osx/debug/lib/libgraphite2.la
vcpkg_installed/x64-osx/debug/lib/manual-link/pkgconfig/gmock_main.pc
vcpkg_installed/x64-osx/debug/lib/manual-link/pkgconfig/gtest_main.pc
vcpkg_installed/x64-osx/debug/lib/libhdf5.settings
vcpkg_installed/x64-osx/debug/lib/systemd/user/plasma-kglobalaccel.service
vcpkg_installed/x64-osx/debug/lib/libnetcdf.settings
vcpkg_installed/x64-osx/debug/lib/botan-2.pc
vcpkg_installed/x64-osx/share/pkgconfig/sol2.pc
vcpkg_installed/x64-osx/share/pkgconfig/cli.pc
vcpkg_installed/x64-osx/share/pkgconfig/polyclipping.pc
vcpkg_installed/x64-osx/share/pkgconfig/easyloggingpp.pc
vcpkg_installed/x64-osx/share/pkgconfig/cppcodec-1.pc
vcpkg_installed/x64-osx/share/pkgconfig/aricpp.pc
vcpkg_installed/x64-osx/share/gdb/auto-load/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/glib_x64-osx/lib/libgobject-2.0.so.0.6600.4-gdb.py
vcpkg_installed/x64-osx/share/gdb/auto-load/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/glib_x64-osx/lib/libglib-2.0.so.0.6600.4-gdb.py
vcpkg_installed/x64-osx/share/shp/export_shp.cmake
vcpkg_installed/x64-osx/share/plplot/examples/cmake/modules/plplot_configure.cmake
vcpkg_installed/x64-osx/share/plplot/examples/plplot-test-interactive.sh
vcpkg_installed/x64-osx/share/plplot/examples/fortran/Makefile
vcpkg_installed/x64-osx/share/ftgl/FTGL/FTGLConfig.cmake
vcpkg_installed/x64-osx/share/libavif/libavif-config.cmake
vcpkg_installed/x64-osx/share/etcdcpp/etcdcppConfig.cmake
vcpkg_installed/x64-osx/share/thumbnailers/gdk-pixbuf-thumbnailer.thumbnailer
vcpkg_installed/x64-osx/share/dbus-1/services/org.kde.kglobalaccel.service
vcpkg_installed/x64-osx/share/cgns/cgnsBuild.defs
vcpkg_installed/x64-osx/share/Bcg729/cmake/Bcg729Config.cmake
vcpkg_installed/x64-osx/share/ismrmrd/cmake/ISMRMRDConfig.cmake
vcpkg_installed/x64-osx/share/Coin/conf/simage-default.cfg
vcpkg_installed/x64-osx/share/mongo-c-driver/uninstall.sh
vcpkg_installed/x64-osx/share/ignition/msgs1.yaml
vcpkg_installed/x64-osx/share/ignition/transport4.yaml
vcpkg_installed/x64-osx/share/ignition/sdformat6.yaml
vcpkg_installed/x64-osx/share/log4cpp/orocos-log4cpp.pc
vcpkg_installed/x64-osx/share/qca/cmake/QcaTargets-debug.cmake
vcpkg_installed/x64-osx/share/protopuf/protopufConfig.cmake
vcpkg_installed/x64-osx/share/ace/bin/MakeProjectCreator/config/MPC.cfg
vcpkg_installed/x64-osx/share/ace/include/makeinclude/wrapper_macros.GNU
vcpkg_installed/x64-osx/share/ace/ace-devel.sh
vcpkg_installed/x64-osx/share/minitrace/minitraceConfig.cmake
vcpkg_installed/x64-osx/share/minitrace/minitraceConfig-release.cmake
vcpkg_installed/vcpkg/info/glib_2.66.4_x64-osx.list
```
</details>

And `... | sed 's/^.*\.//' | sort | uniq -c | sort`:
<details><summary>Result</summary>

```

   1 GNU
   1 conf
   1 d/README
   1 defs
   1 list
   1 thumbnailer
   1 vcpkg_installed/x64-osx/bin/hpxcxx
   1 vcpkg_installed/x64-osx/debug/bin/hpxcxx
   1 vcpkg_installed/x64-osx/debug/tools/libpng/libpng-config
   1 vcpkg_installed/x64-osx/debug/tools/libpng/libpng16-config
   1 vcpkg_installed/x64-osx/share/plplot/examples/fortran/Makefile
   1 vcpkg_installed/x64-osx/tools/glib/glib-gettextize
   1 vcpkg_installed/x64-osx/tools/libpng/libpng-config
   1 vcpkg_installed/x64-osx/tools/libpng/libpng16-config
   2 68/proc/gmmproc
   2 cache
   2 cfg
   2 hpp
   2 la
   2 py
   3 service
   3 sh
   3 yaml
   4 settings
   8 h
  11 cmake
 643 pc
```
</details>

Reply:
If you use the file list from ci you could figure out which ports those files belong to.

Reply:
Or I simply use `vcpkg owns`

Reply:
It results in 
<details><summary>Result</summary>

```
libpng:x64-osx: x64-osx/debug/tools/libpng/libpng16-config
libpng:x64-osx: x64-osx/tools/libpng/libpng16-config
libpng:x64-osx: x64-osx/debug/tools/libpng/libpng-config
libpng:x64-osx: x64-osx/tools/libpng/libpng-config
glib:x64-osx: x64-osx/tools/glib/glib-gettextize
quill:x64-osx: x64-osx/debug/pkgconfig/quill.pc
quill:x64-osx: x64-osx/pkgconfig/quill.pc
openmesh:x64-osx: x64-osx/debug/libdata/pkgconfig/openmesh.pc
openmesh:x64-osx: x64-osx/libdata/pkgconfig/openmesh.pc
hpx:x64-osx: x64-osx/bin/hpxcxx
hpx:x64-osx: x64-osx/debug/bin/hpxcxx
botan:x64-osx: x64-osx/include/botan/build.h
symengine:x64-osx: x64-osx/include/symengine/symengine_config_cling.h
lager:x64-osx: x64-osx/include/lager/resources_path.hpp
lager:x64-osx: x64-osx/include/lager/resources_path.hpp.in
qpid-proton:x64-osx: x64-osx/include/proton/version.h
dartsim:x64-osx: x64-osx/include/dart/config.hpp
sdformat6:x64-osx: x64-osx/include/sdformat-6.2/sdf/sdf_config.h
kf5windowsystem:x64-osx: x64-osx/include/KF5/KWindowSystem/config-kwindowsystem.h
dcmtk:x64-osx: x64-osx/include/dcmtk/config/osconfig.h
gdcm:x64-osx: x64-osx/include/gdcmConfigure.h
libwebsockets:x64-osx: x64-osx/include/lws_config.h
fontconfig:x64-osx: x64-osx/etc/fonts/fonts.conf
fontconfig:x64-osx: x64-osx/etc/fonts/conf.d/README
gdk-pixbuf:x64-osx: x64-osx/debug/gdk-pixbuf-2.0/2.10.0/loaders.cache
gdk-pixbuf:x64-osx: x64-osx/gdk-pixbuf-2.0/2.10.0/loaders.cache
glibmm:x64-osx: x64-osx/debug/lib/glibmm-2.68/proc/gmmproc
glibmm:x64-osx: x64-osx/lib/glibmm-2.68/proc/gmmproc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_grpc_utils.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/google_cloud_cpp_grpc_utils.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_longrunning_operations_protos.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/google_cloud_cpp_longrunning_operations_protos.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/googleapis_cpp_cloud_dialogflow_v2_protos.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/googleapis_cpp_cloud_dialogflow_v2_protos.pc
poppler:x64-osx: x64-osx/debug/lib/pkgconfig/poppler.pc
poppler:x64-osx: x64-osx/lib/pkgconfig/poppler.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_type_dayofweek_protos.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/google_cloud_cpp_type_dayofweek_protos.pc
libtins:x64-osx: x64-osx/debug/lib/pkgconfig/libtins.pc
libtins:x64-osx: x64-osx/lib/pkgconfig/libtins.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/googleapis_cpp_api_http_protos.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/googleapis_cpp_api_http_protos.pc
pcl:x64-osx: x64-osx/debug/lib/pkgconfig/pcl_2d-1.11.pc
pcl:x64-osx: x64-osx/lib/pkgconfig/pcl_2d-1.11.pc
ace:x64-osx: x64-osx/debug/lib/pkgconfig/ACE_ETCL.pc
ace:x64-osx: x64-osx/lib/pkgconfig/ACE_ETCL.pc
msgpack:x64-osx: x64-osx/debug/lib/pkgconfig/msgpack.pc
msgpack:x64-osx: x64-osx/lib/pkgconfig/msgpack.pc
libqrencode:x64-osx: x64-osx/debug/lib/pkgconfig/libqrencode.pc
libqrencode:x64-osx: x64-osx/lib/pkgconfig/libqrencode.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/pubsub_client.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/pubsub_client.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/googleapis_cpp_api_annotations_protos.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/googleapis_cpp_api_annotations_protos.pc
zziplib:x64-osx: x64-osx/debug/lib/pkgconfig/zzipmmapped.pc
zziplib:x64-osx: x64-osx/lib/pkgconfig/zzipmmapped.pc
mongo-cxx-driver:x64-osx: x64-osx/debug/lib/pkgconfig/libbsoncxx-static.pc
mongo-cxx-driver:x64-osx: x64-osx/lib/pkgconfig/libbsoncxx-static.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/googleapis_cpp_type_dayofweek_protos.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/googleapis_cpp_type_dayofweek_protos.pc
redis-plus-plus:x64-osx: x64-osx/debug/lib/pkgconfig/redis++.pc
redis-plus-plus:x64-osx: x64-osx/lib/pkgconfig/redis++.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/googleapis_cpp_api_auth_protos.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/googleapis_cpp_api_auth_protos.pc
fluidlite:x64-osx: x64-osx/debug/lib/pkgconfig/fluidlite.pc
fluidlite:x64-osx: x64-osx/lib/pkgconfig/fluidlite.pc
pcl:x64-osx: x64-osx/debug/lib/pkgconfig/pcl_features-1.11.pc
pcl:x64-osx: x64-osx/lib/pkgconfig/pcl_features-1.11.pc
sais:x64-osx: x64-osx/debug/lib/pkgconfig/libsais64.pc
sais:x64-osx: x64-osx/lib/pkgconfig/libsais64.pc
bde:x64-osx: x64-osx/debug/lib/pkgconfig/inteldfp.pc
bde:x64-osx: x64-osx/lib/pkgconfig/inteldfp.pc
orocos-kdl:x64-osx: x64-osx/debug/lib/pkgconfig/orocos_kdl.pc
orocos-kdl:x64-osx: x64-osx/lib/pkgconfig/orocos_kdl.pc
libbson:x64-osx: x64-osx/debug/lib/pkgconfig/libbson-1.0.pc
libbson:x64-osx: x64-osx/lib/pkgconfig/libbson-1.0.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_rpc_status_protos.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/google_cloud_cpp_rpc_status_protos.pc
freeopcua:x64-osx: x64-osx/debug/lib/pkgconfig/libopcuaserver.pc
freeopcua:x64-osx: x64-osx/lib/pkgconfig/libopcuaserver.pc
mongo-c-driver:x64-osx: x64-osx/debug/lib/pkgconfig/libmongoc-1.0.pc
mongo-c-driver:x64-osx: x64-osx/lib/pkgconfig/libmongoc-1.0.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/googleapis_cpp_api_metric_protos.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/googleapis_cpp_api_metric_protos.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/googleapis_cpp_api_label_protos.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/googleapis_cpp_api_label_protos.pc
cutelyst2:x64-osx: x64-osx/debug/lib/pkgconfig/Cutelyst2Qt5UserAgent.pc
cutelyst2:x64-osx: x64-osx/lib/pkgconfig/Cutelyst2Qt5UserAgent.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/googleapis_cpp_type_fraction_protos.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/googleapis_cpp_type_fraction_protos.pc
libpmemobj-cpp:x64-osx: x64-osx/lib/pkgconfig/libpmemobj++.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/googleapis_cpp_api_resource_protos.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/googleapis_cpp_api_resource_protos.pc
pcl:x64-osx: x64-osx/debug/lib/pkgconfig/pcl_surface-1.11.pc
pcl:x64-osx: x64-osx/lib/pkgconfig/pcl_surface-1.11.pc
grpc:x64-osx: x64-osx/debug/lib/pkgconfig/grpc.pc
grpc:x64-osx: x64-osx/lib/pkgconfig/grpc.pc
nlopt:x64-osx: x64-osx/debug/lib/pkgconfig/nlopt.pc
nlopt:x64-osx: x64-osx/lib/pkgconfig/nlopt.pc
cpuid:x64-osx: x64-osx/debug/lib/pkgconfig/libcpuid.pc
cpuid:x64-osx: x64-osx/lib/pkgconfig/libcpuid.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_bigtable_protos.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/google_cloud_cpp_bigtable_protos.pc
cutelyst2:x64-osx: x64-osx/debug/lib/pkgconfig/Cutelyst2Qt5WSGI.pc
cutelyst2:x64-osx: x64-osx/lib/pkgconfig/Cutelyst2Qt5WSGI.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_api_launch_stage_protos.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/google_cloud_cpp_api_launch_stage_protos.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_pubsub_protos.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/google_cloud_cpp_pubsub_protos.pc
libproxy:x64-osx: x64-osx/debug/lib/pkgconfig/libproxy-1.0.pc
libproxy:x64-osx: x64-osx/lib/pkgconfig/libproxy-1.0.pc
gflags:x64-osx: x64-osx/debug/lib/pkgconfig/gflags.pc
gflags:x64-osx: x64-osx/lib/pkgconfig/gflags.pc
uriparser:x64-osx: x64-osx/debug/lib/pkgconfig/liburiparser.pc
uriparser:x64-osx: x64-osx/lib/pkgconfig/liburiparser.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/googleapis_cpp_type_calendar_period_protos.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/googleapis_cpp_type_calendar_period_protos.pc
geogram:x64-osx: x64-osx/debug/lib/pkgconfig/geogram1.pc
geogram:x64-osx: x64-osx/lib/pkgconfig/geogram1.pc
arrow:x64-osx: x64-osx/debug/lib/pkgconfig/arrow-csv.pc
arrow:x64-osx: x64-osx/lib/pkgconfig/arrow-csv.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/googleapis_cpp_type_expr_protos.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/googleapis_cpp_type_expr_protos.pc
plplot:x64-osx: x64-osx/debug/lib/pkgconfig/plplot-c++.pc
plplot:x64-osx: x64-osx/lib/pkgconfig/plplot-c++.pc
realsense2:x64-osx: x64-osx/debug/lib/pkgconfig/realsense2.pc
realsense2:x64-osx: x64-osx/lib/pkgconfig/realsense2.pc
blitz:x64-osx: x64-osx/debug/lib/pkgconfig/blitz.pc
blitz:x64-osx: x64-osx/lib/pkgconfig/blitz.pc
amqpcpp:x64-osx: x64-osx/debug/lib/pkgconfig/amqpcpp.pc
amqpcpp:x64-osx: x64-osx/lib/pkgconfig/amqpcpp.pc
cmocka:x64-osx: x64-osx/debug/lib/pkgconfig/cmocka.pc
cmocka:x64-osx: x64-osx/lib/pkgconfig/cmocka.pc
grpc:x64-osx: x64-osx/debug/lib/pkgconfig/gpr.pc
grpc:x64-osx: x64-osx/lib/pkgconfig/gpr.pc
openimageio:x64-osx: x64-osx/debug/lib/pkgconfig/OpenImageIO.pc
openimageio:x64-osx: x64-osx/lib/pkgconfig/OpenImageIO.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_type_phone_number_protos.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/google_cloud_cpp_type_phone_number_protos.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_api_annotations_protos.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/google_cloud_cpp_api_annotations_protos.pc
rbdl:x64-osx: x64-osx/debug/lib/pkgconfig/rbdl.pc
rbdl:x64-osx: x64-osx/lib/pkgconfig/rbdl.pc
ace:x64-osx: x64-osx/debug/lib/pkgconfig/ACE.pc
ace:x64-osx: x64-osx/lib/pkgconfig/ACE.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/storage_client.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/storage_client.pc
pcl:x64-osx: x64-osx/debug/lib/pkgconfig/pcl_ml-1.11.pc
pcl:x64-osx: x64-osx/lib/pkgconfig/pcl_ml-1.11.pc
capnproto:x64-osx: x64-osx/debug/lib/pkgconfig/kj-async.pc
capnproto:x64-osx: x64-osx/lib/pkgconfig/kj-async.pc
check:x64-osx: x64-osx/debug/lib/pkgconfig/check.pc
check:x64-osx: x64-osx/lib/pkgconfig/check.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/bigtable_client.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/bigtable_client.pc
arrow:x64-osx: x64-osx/debug/lib/pkgconfig/parquet.pc
arrow:x64-osx: x64-osx/lib/pkgconfig/parquet.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_spanner_protos.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/google_cloud_cpp_spanner_protos.pc
mosquitto:x64-osx: x64-osx/debug/lib/pkgconfig/libmosquittopp.pc
mosquitto:x64-osx: x64-osx/lib/pkgconfig/libmosquittopp.pc
portaudio:x64-osx: x64-osx/debug/lib/pkgconfig/portaudio-2.0.pc
portaudio:x64-osx: x64-osx/lib/pkgconfig/portaudio-2.0.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_api_resource_protos.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/google_cloud_cpp_api_resource_protos.pc
grpc:x64-osx: x64-osx/debug/lib/pkgconfig/grpc++_unsecure.pc
grpc:x64-osx: x64-osx/lib/pkgconfig/grpc++_unsecure.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_logging_protos.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/google_cloud_cpp_logging_protos.pc
capnproto:x64-osx: x64-osx/debug/lib/pkgconfig/capnp-rpc.pc
capnproto:x64-osx: x64-osx/lib/pkgconfig/capnp-rpc.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_iam_v1_iam_policy_protos.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/google_cloud_cpp_iam_v1_iam_policy_protos.pc
fftw3:x64-osx: x64-osx/debug/lib/pkgconfig/fftw3f.pc
fftw3:x64-osx: x64-osx/lib/pkgconfig/fftw3f.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/googleapis_cpp_type_interval_protos.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/googleapis_cpp_type_interval_protos.pc
capnproto:x64-osx: x64-osx/debug/lib/pkgconfig/kj-http.pc
capnproto:x64-osx: x64-osx/lib/pkgconfig/kj-http.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_cloud_dialogflow_v2_protos.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/google_cloud_cpp_cloud_dialogflow_v2_protos.pc
grpc:x64-osx: x64-osx/debug/lib/pkgconfig/grpc++.pc
grpc:x64-osx: x64-osx/lib/pkgconfig/grpc++.pc
kf5attica:x64-osx: x64-osx/debug/lib/pkgconfig/libKF5Attica.pc
kf5attica:x64-osx: x64-osx/lib/pkgconfig/libKF5Attica.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_api_client_protos.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/google_cloud_cpp_api_client_protos.pc
functions-framework-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/functions_framework_cpp.pc
functions-framework-cpp:x64-osx: x64-osx/lib/pkgconfig/functions_framework_cpp.pc
freeglut:x64-osx: x64-osx/debug/lib/pkgconfig/glut.pc
freeglut:x64-osx: x64-osx/lib/pkgconfig/glut.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/googleapis_cpp_cloud_dialogflow_v2beta1_protos.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/googleapis_cpp_cloud_dialogflow_v2beta1_protos.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_bigquery.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/google_cloud_cpp_bigquery.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/googleapis.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/googleapis.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_api_http_protos.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/google_cloud_cpp_api_http_protos.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/googleapis_cpp_rpc_error_details_protos.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/googleapis_cpp_rpc_error_details_protos.pc
plplot:x64-osx: x64-osx/debug/lib/pkgconfig/plplot.pc
plplot:x64-osx: x64-osx/lib/pkgconfig/plplot.pc
soxr:x64-osx: x64-osx/debug/lib/pkgconfig/soxr.pc
soxr:x64-osx: x64-osx/lib/pkgconfig/soxr.pc
geographiclib:x64-osx: x64-osx/debug/lib/pkgconfig/geographiclib.pc
geographiclib:x64-osx: x64-osx/lib/pkgconfig/geographiclib.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/spanner_client.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/spanner_client.pc
armadillo:x64-osx: x64-osx/debug/lib/pkgconfig/armadillo.pc
armadillo:x64-osx: x64-osx/lib/pkgconfig/armadillo.pc
libmt32emu:x64-osx: x64-osx/debug/lib/pkgconfig/mt32emu.pc
libmt32emu:x64-osx: x64-osx/lib/pkgconfig/mt32emu.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_type_interval_protos.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/google_cloud_cpp_type_interval_protos.pc
aom:x64-osx: x64-osx/debug/lib/pkgconfig/aom.pc
aom:x64-osx: x64-osx/lib/pkgconfig/aom.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/googleapis_cpp_monitoring_protos.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/googleapis_cpp_monitoring_protos.pc
json11:x64-osx: x64-osx/debug/lib/pkgconfig/json11.pc
json11:x64-osx: x64-osx/lib/pkgconfig/json11.pc
libkml:x64-osx: x64-osx/debug/lib/pkgconfig/libkml.pc
libkml:x64-osx: x64-osx/lib/pkgconfig/libkml.pc
libraw:x64-osx: x64-osx/debug/lib/pkgconfig/libraw.pc
libraw:x64-osx: x64-osx/lib/pkgconfig/libraw.pc
nanomsg:x64-osx: x64-osx/debug/lib/pkgconfig/nanomsg.pc
nanomsg:x64-osx: x64-osx/lib/pkgconfig/nanomsg.pc
liblo:x64-osx: x64-osx/debug/lib/pkgconfig/liblo.pc
liblo:x64-osx: x64-osx/lib/pkgconfig/liblo.pc
thrift:x64-osx: x64-osx/debug/lib/pkgconfig/thrift-nb.pc
thrift:x64-osx: x64-osx/lib/pkgconfig/thrift-nb.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/googleapis_cpp_type_localized_text_protos.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/googleapis_cpp_type_localized_text_protos.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/googleapis_cpp_api_monitored_resource_protos.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/googleapis_cpp_api_monitored_resource_protos.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_api_auth_protos.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/google_cloud_cpp_api_auth_protos.pc
bde:x64-osx: x64-osx/debug/lib/pkgconfig/bdl.pc
bde:x64-osx: x64-osx/lib/pkgconfig/bdl.pc
nanoflann:x64-osx: x64-osx/debug/lib/pkgconfig/nanoflann.pc
nanoflann:x64-osx: x64-osx/lib/pkgconfig/nanoflann.pc
octomap:x64-osx: x64-osx/debug/lib/pkgconfig/octomap.pc
octomap:x64-osx: x64-osx/lib/pkgconfig/octomap.pc
bde:x64-osx: x64-osx/debug/lib/pkgconfig/bsl.pc
bde:x64-osx: x64-osx/lib/pkgconfig/bsl.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/googleapis_cpp_bigtable_protos.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/googleapis_cpp_bigtable_protos.pc
raylib:x64-osx: x64-osx/debug/lib/pkgconfig/raylib.pc
raylib:x64-osx: x64-osx/lib/pkgconfig/raylib.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_storage_protos.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/google_cloud_cpp_storage_protos.pc
cutelyst2:x64-osx: x64-osx/debug/lib/pkgconfig/Cutelyst2Qt5UtilsValidator.pc
cutelyst2:x64-osx: x64-osx/lib/pkgconfig/Cutelyst2Qt5UtilsValidator.pc
pcl:x64-osx: x64-osx/debug/lib/pkgconfig/pcl_tracking-1.11.pc
pcl:x64-osx: x64-osx/lib/pkgconfig/pcl_tracking-1.11.pc
msgpack11:x64-osx: x64-osx/debug/lib/pkgconfig/msgpack11.pc
msgpack11:x64-osx: x64-osx/lib/pkgconfig/msgpack11.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_api_distribution_protos.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/google_cloud_cpp_api_distribution_protos.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/googleapis_cpp_cloud_speech_protos.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/googleapis_cpp_cloud_speech_protos.pc
poppler:x64-osx: x64-osx/debug/lib/pkgconfig/poppler-cpp.pc
poppler:x64-osx: x64-osx/lib/pkgconfig/poppler-cpp.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_iam.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/google_cloud_cpp_iam.pc
freeopcua:x64-osx: x64-osx/debug/lib/pkgconfig/libopcuaclient.pc
freeopcua:x64-osx: x64-osx/lib/pkgconfig/libopcuaclient.pc
flatbuffers:x64-osx: x64-osx/debug/lib/pkgconfig/flatbuffers.pc
flatbuffers:x64-osx: x64-osx/lib/pkgconfig/flatbuffers.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/googleapis_cpp_iam_protos.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/googleapis_cpp_iam_protos.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/googleapis_cpp_type_date_protos.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/googleapis_cpp_type_date_protos.pc
pcl:x64-osx: x64-osx/debug/lib/pkgconfig/pcl_search-1.11.pc
pcl:x64-osx: x64-osx/lib/pkgconfig/pcl_search-1.11.pc
bde:x64-osx: x64-osx/debug/lib/pkgconfig/bal.pc
bde:x64-osx: x64-osx/lib/pkgconfig/bal.pc
libgd:x64-osx: x64-osx/debug/lib/pkgconfig/gdlib.pc
libgd:x64-osx: x64-osx/lib/pkgconfig/gdlib.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_type_fraction_protos.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/google_cloud_cpp_type_fraction_protos.pc
pcl:x64-osx: x64-osx/debug/lib/pkgconfig/pcl_recognition-1.11.pc
pcl:x64-osx: x64-osx/lib/pkgconfig/pcl_recognition-1.11.pc
libconfuse:x64-osx: x64-osx/debug/lib/pkgconfig/libconfuse.pc
libconfuse:x64-osx: x64-osx/lib/pkgconfig/libconfuse.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_type_month_protos.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/google_cloud_cpp_type_month_protos.pc
sparsehash:x64-osx: x64-osx/lib/pkgconfig/libsparsehash.pc
libmediainfo:x64-osx: x64-osx/debug/lib/pkgconfig/libmediainfo.pc
libmediainfo:x64-osx: x64-osx/lib/pkgconfig/libmediainfo.pc
hiredis:x64-osx: x64-osx/debug/lib/pkgconfig/hiredis.pc
hiredis:x64-osx: x64-osx/lib/pkgconfig/hiredis.pc
blosc:x64-osx: x64-osx/debug/lib/pkgconfig/blosc.pc
blosc:x64-osx: x64-osx/lib/pkgconfig/blosc.pc
arrow:x64-osx: x64-osx/debug/lib/pkgconfig/arrow-json.pc
arrow:x64-osx: x64-osx/lib/pkgconfig/arrow-json.pc
liblemon:x64-osx: x64-osx/debug/lib/pkgconfig/lemon.pc
liblemon:x64-osx: x64-osx/lib/pkgconfig/lemon.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_type_localized_text_protos.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/google_cloud_cpp_type_localized_text_protos.pc
libgta:x64-osx: x64-osx/debug/lib/pkgconfig/gta.pc
libgta:x64-osx: x64-osx/lib/pkgconfig/gta.pc
libgit2:x64-osx: x64-osx/debug/lib/pkgconfig/libgit2.pc
libgit2:x64-osx: x64-osx/lib/pkgconfig/libgit2.pc
libevent:x64-osx: x64-osx/debug/lib/pkgconfig/libevent.pc
libevent:x64-osx: x64-osx/lib/pkgconfig/libevent.pc
hyperscan:x64-osx: x64-osx/debug/lib/pkgconfig/libhs.pc
hyperscan:x64-osx: x64-osx/lib/pkgconfig/libhs.pc
gtest:x64-osx: x64-osx/debug/lib/pkgconfig/gmock.pc
gtest:x64-osx: x64-osx/lib/pkgconfig/gmock.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_type_color_protos.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/google_cloud_cpp_type_color_protos.pc
bde:x64-osx: x64-osx/debug/lib/pkgconfig/bbl.pc
bde:x64-osx: x64-osx/lib/pkgconfig/bbl.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_type_latlng_protos.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/google_cloud_cpp_type_latlng_protos.pc
cjson:x64-osx: x64-osx/debug/lib/pkgconfig/libcjson.pc
cjson:x64-osx: x64-osx/lib/pkgconfig/libcjson.pc
freeopcua:x64-osx: x64-osx/debug/lib/pkgconfig/libopcuaprotocol.pc
freeopcua:x64-osx: x64-osx/lib/pkgconfig/libopcuaprotocol.pc
bde:x64-osx: x64-osx/debug/lib/pkgconfig/decnumber.pc
bde:x64-osx: x64-osx/lib/pkgconfig/decnumber.pc
capnproto:x64-osx: x64-osx/debug/lib/pkgconfig/kj.pc
capnproto:x64-osx: x64-osx/lib/pkgconfig/kj.pc
cutelyst2:x64-osx: x64-osx/debug/lib/pkgconfig/Cutelyst2Qt5StaticSimple.pc
cutelyst2:x64-osx: x64-osx/lib/pkgconfig/Cutelyst2Qt5StaticSimple.pc
libevent:x64-osx: x64-osx/debug/lib/pkgconfig/libevent_extra.pc
libevent:x64-osx: x64-osx/lib/pkgconfig/libevent_extra.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_cloud_speech_protos.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/google_cloud_cpp_cloud_speech_protos.pc
libmikmod:x64-osx: x64-osx/debug/lib/pkgconfig/libmikmod.pc
libmikmod:x64-osx: x64-osx/lib/pkgconfig/libmikmod.pc
ois:x64-osx: x64-osx/debug/lib/pkgconfig/OIS.pc
ois:x64-osx: x64-osx/lib/pkgconfig/OIS.pc
bde:x64-osx: x64-osx/debug/lib/pkgconfig/btl.pc
bde:x64-osx: x64-osx/lib/pkgconfig/btl.pc
brpc:x64-osx: x64-osx/debug/lib/pkgconfig/brpc.pc
brpc:x64-osx: x64-osx/lib/pkgconfig/brpc.pc
fftw3:x64-osx: x64-osx/debug/lib/pkgconfig/fftw3l.pc
fftw3:x64-osx: x64-osx/lib/pkgconfig/fftw3l.pc
arrow:x64-osx: x64-osx/debug/lib/pkgconfig/arrow.pc
arrow:x64-osx: x64-osx/lib/pkgconfig/arrow.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/googleapis_cpp_type_money_protos.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/googleapis_cpp_type_money_protos.pc
bddisasm:x64-osx: x64-osx/debug/lib/pkgconfig/libbddisasm.pc
bddisasm:x64-osx: x64-osx/lib/pkgconfig/libbddisasm.pc
graphite2:x64-osx: x64-osx/debug/lib/pkgconfig/graphite2.pc
graphite2:x64-osx: x64-osx/lib/pkgconfig/graphite2.pc
libsrt:x64-osx: x64-osx/debug/lib/pkgconfig/haisrt.pc
libsrt:x64-osx: x64-osx/lib/pkgconfig/haisrt.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/googleapis_cpp_type_month_protos.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/googleapis_cpp_type_month_protos.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/googleapis_cpp_devtools_cloudtrace_v2_tracing_protos.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/googleapis_cpp_devtools_cloudtrace_v2_tracing_protos.pc
libftdi1:x64-osx: x64-osx/debug/lib/pkgconfig/libftdipp1.pc
libftdi1:x64-osx: x64-osx/lib/pkgconfig/libftdipp1.pc
mosquitto:x64-osx: x64-osx/debug/lib/pkgconfig/libmosquitto.pc
mosquitto:x64-osx: x64-osx/lib/pkgconfig/libmosquitto.pc
cutelyst2:x64-osx: x64-osx/debug/lib/pkgconfig/Cutelyst2Qt5Core.pc
cutelyst2:x64-osx: x64-osx/lib/pkgconfig/Cutelyst2Qt5Core.pc
simage:x64-osx: x64-osx/debug/lib/pkgconfig/simage.pc
simage:x64-osx: x64-osx/lib/pkgconfig/simage.pc
mongo-c-driver:x64-osx: x64-osx/debug/lib/pkgconfig/libmongoc-ssl-1.0.pc
mongo-c-driver:x64-osx: x64-osx/lib/pkgconfig/libmongoc-ssl-1.0.pc
librdkafka:x64-osx: x64-osx/debug/lib/pkgconfig/rdkafka-static.pc
librdkafka:x64-osx: x64-osx/lib/pkgconfig/rdkafka-static.pc
igraph:x64-osx: x64-osx/debug/lib/pkgconfig/igraph.pc
igraph:x64-osx: x64-osx/lib/pkgconfig/igraph.pc
libcerf:x64-osx: x64-osx/debug/lib/pkgconfig/libcerf.pc
libcerf:x64-osx: x64-osx/lib/pkgconfig/libcerf.pc
pcl:x64-osx: x64-osx/debug/lib/pkgconfig/pcl_octree-1.11.pc
pcl:x64-osx: x64-osx/lib/pkgconfig/pcl_octree-1.11.pc
libtorrent:x64-osx: x64-osx/debug/lib/pkgconfig/libtorrent-rasterbar.pc
libtorrent:x64-osx: x64-osx/lib/pkgconfig/libtorrent-rasterbar.pc
libftdi1:x64-osx: x64-osx/debug/lib/pkgconfig/libftdi1.pc
libftdi1:x64-osx: x64-osx/lib/pkgconfig/libftdi1.pc
pcl:x64-osx: x64-osx/debug/lib/pkgconfig/pcl_sample_consensus-1.11.pc
pcl:x64-osx: x64-osx/lib/pkgconfig/pcl_sample_consensus-1.11.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_type_money_protos.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/google_cloud_cpp_type_money_protos.pc
muparser:x64-osx: x64-osx/debug/lib/pkgconfig/muparser.pc
muparser:x64-osx: x64-osx/lib/pkgconfig/muparser.pc
libxlsxwriter:x64-osx: x64-osx/debug/lib/pkgconfig/xlsxwriter.pc
libxlsxwriter:x64-osx: x64-osx/lib/pkgconfig/xlsxwriter.pc
iniparser:x64-osx: x64-osx/debug/lib/pkgconfig/iniparser.pc
iniparser:x64-osx: x64-osx/lib/pkgconfig/iniparser.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/googleapis_cpp_type_color_protos.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/googleapis_cpp_type_color_protos.pc
pcl:x64-osx: x64-osx/debug/lib/pkgconfig/pcl_common-1.11.pc
pcl:x64-osx: x64-osx/lib/pkgconfig/pcl_common-1.11.pc
cutelyst2:x64-osx: x64-osx/debug/lib/pkgconfig/Cutelyst2Qt5StatusMessage.pc
cutelyst2:x64-osx: x64-osx/lib/pkgconfig/Cutelyst2Qt5StatusMessage.pc
thrift:x64-osx: x64-osx/debug/lib/pkgconfig/thrift.pc
thrift:x64-osx: x64-osx/lib/pkgconfig/thrift.pc
collada-dom:x64-osx: x64-osx/debug/lib/pkgconfig/collada-dom-150.pc
collada-dom:x64-osx: x64-osx/lib/pkgconfig/collada-dom-150.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_cloud_texttospeech_protos.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/google_cloud_cpp_cloud_texttospeech_protos.pc
xerces-c:x64-osx: x64-osx/debug/lib/pkgconfig/xerces-c.pc
xerces-c:x64-osx: x64-osx/lib/pkgconfig/xerces-c.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_monitoring_protos.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/google_cloud_cpp_monitoring_protos.pc
collada-dom:x64-osx: x64-osx/debug/lib/pkgconfig/collada-dom-141.pc
collada-dom:x64-osx: x64-osx/lib/pkgconfig/collada-dom-141.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_devtools_cloudtrace_v2_tracing_protos.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/google_cloud_cpp_devtools_cloudtrace_v2_tracing_protos.pc
orocos-kdl:x64-osx: x64-osx/debug/lib/pkgconfig/orocos-kdl.pc
orocos-kdl:x64-osx: x64-osx/lib/pkgconfig/orocos-kdl.pc
libevent:x64-osx: x64-osx/debug/lib/pkgconfig/libevent_pthreads.pc
libevent:x64-osx: x64-osx/lib/pkgconfig/libevent_pthreads.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_rpc_error_details_protos.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/google_cloud_cpp_rpc_error_details_protos.pc
stxxl:x64-osx: x64-osx/lib/pkgconfig/stxxl.pc
cutelyst2:x64-osx: x64-osx/debug/lib/pkgconfig/Cutelyst2Qt5Session.pc
cutelyst2:x64-osx: x64-osx/lib/pkgconfig/Cutelyst2Qt5Session.pc
capnproto:x64-osx: x64-osx/debug/lib/pkgconfig/capnp-json.pc
capnproto:x64-osx: x64-osx/lib/pkgconfig/capnp-json.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_devtools_cloudtrace_v2_trace_protos.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/google_cloud_cpp_devtools_cloudtrace_v2_trace_protos.pc
cutelyst2:x64-osx: x64-osx/debug/lib/pkgconfig/Cutelyst2Qt5Authentication.pc
cutelyst2:x64-osx: x64-osx/lib/pkgconfig/Cutelyst2Qt5Authentication.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/googleapis_cpp_spanner_protos.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/googleapis_cpp_spanner_protos.pc
superlu:x64-osx: x64-osx/debug/lib/pkgconfig/superlu.pc
superlu:x64-osx: x64-osx/lib/pkgconfig/superlu.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_type_date_protos.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/google_cloud_cpp_type_date_protos.pc
usrsctp:x64-osx: x64-osx/debug/lib/pkgconfig/usrsctp.pc
usrsctp:x64-osx: x64-osx/lib/pkgconfig/usrsctp.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/googleapis_cpp_logging_protos.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/googleapis_cpp_logging_protos.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_storage.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/google_cloud_cpp_storage.pc
librdkafka:x64-osx: x64-osx/debug/lib/pkgconfig/rdkafka++-static.pc
librdkafka:x64-osx: x64-osx/lib/pkgconfig/rdkafka++-static.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/googleapis_cpp_type_latlng_protos.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/googleapis_cpp_type_latlng_protos.pc
libevent:x64-osx: x64-osx/debug/lib/pkgconfig/libevent_core.pc
libevent:x64-osx: x64-osx/lib/pkgconfig/libevent_core.pc
pcl:x64-osx: x64-osx/debug/lib/pkgconfig/pcl_kdtree-1.11.pc
pcl:x64-osx: x64-osx/lib/pkgconfig/pcl_kdtree-1.11.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_spanner.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/google_cloud_cpp_spanner.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_bigtable.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/google_cloud_cpp_bigtable.pc
zziplib:x64-osx: x64-osx/debug/lib/pkgconfig/zzipfseeko.pc
zziplib:x64-osx: x64-osx/lib/pkgconfig/zzipfseeko.pc
pcl:x64-osx: x64-osx/debug/lib/pkgconfig/pcl_registration-1.11.pc
pcl:x64-osx: x64-osx/lib/pkgconfig/pcl_registration-1.11.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_type_calendar_period_protos.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/google_cloud_cpp_type_calendar_period_protos.pc
sdformat6:x64-osx: x64-osx/debug/lib/pkgconfig/sdformat.pc
sdformat6:x64-osx: x64-osx/lib/pkgconfig/sdformat.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/googleapis_cpp_type_datetime_protos.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/googleapis_cpp_type_datetime_protos.pc
libwebsockets:x64-osx: x64-osx/debug/lib/pkgconfig/libwebsockets_static.pc
libwebsockets:x64-osx: x64-osx/lib/pkgconfig/libwebsockets_static.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/googleapis_cpp_devtools_cloudtrace_v2_trace_protos.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/googleapis_cpp_devtools_cloudtrace_v2_trace_protos.pc
ppconsul:x64-osx: x64-osx/debug/lib/pkgconfig/ppconsul.pc
ppconsul:x64-osx: x64-osx/lib/pkgconfig/ppconsul.pc
pcl:x64-osx: x64-osx/debug/lib/pkgconfig/pcl_io-1.11.pc
pcl:x64-osx: x64-osx/lib/pkgconfig/pcl_io-1.11.pc
plplot:x64-osx: x64-osx/debug/lib/pkgconfig/plplot-fortran.pc
plplot:x64-osx: x64-osx/lib/pkgconfig/plplot-fortran.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/googleapis_cpp_api_distribution_protos.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/googleapis_cpp_api_distribution_protos.pc
libwebsockets:x64-osx: x64-osx/debug/lib/pkgconfig/libwebsockets.pc
libwebsockets:x64-osx: x64-osx/lib/pkgconfig/libwebsockets.pc
edlib:x64-osx: x64-osx/debug/lib/pkgconfig/edlib-1.pc
edlib:x64-osx: x64-osx/lib/pkgconfig/edlib-1.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/googleapis_cpp_cloud_texttospeech_protos.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/googleapis_cpp_cloud_texttospeech_protos.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_api_monitored_resource_protos.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/google_cloud_cpp_api_monitored_resource_protos.pc
curlpp:x64-osx: x64-osx/debug/lib/pkgconfig/curlpp.pc
curlpp:x64-osx: x64-osx/lib/pkgconfig/curlpp.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/googleapis_cpp_iam_v1_policy_protos.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/googleapis_cpp_iam_v1_policy_protos.pc
flann:x64-osx: x64-osx/debug/lib/pkgconfig/flann.pc
flann:x64-osx: x64-osx/lib/pkgconfig/flann.pc
grpc:x64-osx: x64-osx/debug/lib/pkgconfig/grpc_unsecure.pc
grpc:x64-osx: x64-osx/lib/pkgconfig/grpc_unsecure.pc
capnproto:x64-osx: x64-osx/debug/lib/pkgconfig/capnp.pc
capnproto:x64-osx: x64-osx/lib/pkgconfig/capnp.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_cloud_dialogflow_v2beta1_protos.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/google_cloud_cpp_cloud_dialogflow_v2beta1_protos.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_type_datetime_protos.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/google_cloud_cpp_type_datetime_protos.pc
cutelyst2:x64-osx: x64-osx/debug/lib/pkgconfig/Cutelyst2Qt5UtilsSql.pc
cutelyst2:x64-osx: x64-osx/lib/pkgconfig/Cutelyst2Qt5UtilsSql.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_cloud_bigquery_protos.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/google_cloud_cpp_cloud_bigquery_protos.pc
libsrt:x64-osx: x64-osx/debug/lib/pkgconfig/srt.pc
libsrt:x64-osx: x64-osx/lib/pkgconfig/srt.pc
cutelyst2:x64-osx: x64-osx/debug/lib/pkgconfig/Cutelyst2Qt5ViewJSON.pc
cutelyst2:x64-osx: x64-osx/lib/pkgconfig/Cutelyst2Qt5ViewJSON.pc
thrift:x64-osx: x64-osx/debug/lib/pkgconfig/thrift-z.pc
thrift:x64-osx: x64-osx/lib/pkgconfig/thrift-z.pc
qca:x64-osx: x64-osx/debug/lib/pkgconfig/qca2.pc
qca:x64-osx: x64-osx/lib/pkgconfig/qca2.pc
libevhtp:x64-osx: x64-osx/debug/lib/pkgconfig/evhtp.pc
libevhtp:x64-osx: x64-osx/lib/pkgconfig/evhtp.pc
charls:x64-osx: x64-osx/debug/lib/pkgconfig/charls.pc
charls:x64-osx: x64-osx/lib/pkgconfig/charls.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_iam_v1_policy_protos.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/google_cloud_cpp_iam_v1_policy_protos.pc
bde:x64-osx: x64-osx/debug/lib/pkgconfig/pcre2.pc
bde:x64-osx: x64-osx/lib/pkgconfig/pcre2.pc
fftw3:x64-osx: x64-osx/debug/lib/pkgconfig/fftw3.pc
fftw3:x64-osx: x64-osx/lib/pkgconfig/fftw3.pc
bcg729:x64-osx: x64-osx/debug/lib/pkgconfig/libbcg729.pc
bcg729:x64-osx: x64-osx/lib/pkgconfig/libbcg729.pc
oniguruma:x64-osx: x64-osx/debug/lib/pkgconfig/oniguruma.pc
oniguruma:x64-osx: x64-osx/lib/pkgconfig/oniguruma.pc
blitz:x64-osx: x64-osx/debug/lib/pkgconfig/blitz-static.pc
blitz:x64-osx: x64-osx/lib/pkgconfig/blitz-static.pc
graphqlparser:x64-osx: x64-osx/debug/lib/pkgconfig/libgraphqlparser.pc
graphqlparser:x64-osx: x64-osx/lib/pkgconfig/libgraphqlparser.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/googleapis_cpp_storage_protos.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/googleapis_cpp_storage_protos.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/googleapis_cpp_cloud_bigquery_protos.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/googleapis_cpp_cloud_bigquery_protos.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_type_quaternion_protos.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/google_cloud_cpp_type_quaternion_protos.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/googleapis_cpp_type_phone_number_protos.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/googleapis_cpp_type_phone_number_protos.pc
mongo-cxx-driver:x64-osx: x64-osx/debug/lib/pkgconfig/libmongocxx-static.pc
mongo-cxx-driver:x64-osx: x64-osx/lib/pkgconfig/libmongocxx-static.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/googleapis_cpp_type_postal_address_protos.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/googleapis_cpp_type_postal_address_protos.pc
ccd:x64-osx: x64-osx/debug/lib/pkgconfig/ccd.pc
ccd:x64-osx: x64-osx/lib/pkgconfig/ccd.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_type_expr_protos.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/google_cloud_cpp_type_expr_protos.pc
cutelyst2:x64-osx: x64-osx/debug/lib/pkgconfig/Cutelyst2Qt5UtilsLangSelect.pc
cutelyst2:x64-osx: x64-osx/lib/pkgconfig/Cutelyst2Qt5UtilsLangSelect.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_api_field_behavior_protos.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/google_cloud_cpp_api_field_behavior_protos.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/googleapis_cpp_pubsub_protos.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/googleapis_cpp_pubsub_protos.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/googleapis_cpp_api_client_protos.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/googleapis_cpp_api_client_protos.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/googleapis_cpp_type_timeofday_protos.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/googleapis_cpp_type_timeofday_protos.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_iam_protos.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/google_cloud_cpp_iam_protos.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/googleapis_cpp_iam_v1_iam_policy_protos.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/googleapis_cpp_iam_v1_iam_policy_protos.pc
pcl:x64-osx: x64-osx/debug/lib/pkgconfig/pcl_filters-1.11.pc
pcl:x64-osx: x64-osx/lib/pkgconfig/pcl_filters-1.11.pc
opencc:x64-osx: x64-osx/debug/lib/pkgconfig/opencc.pc
opencc:x64-osx: x64-osx/lib/pkgconfig/opencc.pc
sais:x64-osx: x64-osx/debug/lib/pkgconfig/libsais.pc
sais:x64-osx: x64-osx/lib/pkgconfig/libsais.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_common.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/google_cloud_cpp_common.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/googleapis_cpp_logging_type_protos.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/googleapis_cpp_logging_type_protos.pc
folly:x64-osx: x64-osx/debug/lib/pkgconfig/libfolly.pc
folly:x64-osx: x64-osx/lib/pkgconfig/libfolly.pc
pcl:x64-osx: x64-osx/debug/lib/pkgconfig/pcl_geometry-1.11.pc
pcl:x64-osx: x64-osx/lib/pkgconfig/pcl_geometry-1.11.pc
clblast:x64-osx: x64-osx/debug/lib/pkgconfig/clblast.pc
clblast:x64-osx: x64-osx/lib/pkgconfig/clblast.pc
itpp:x64-osx: x64-osx/debug/lib/pkgconfig/itpp.pc
itpp:x64-osx: x64-osx/lib/pkgconfig/itpp.pc
ode:x64-osx: x64-osx/debug/lib/pkgconfig/ode.pc
ode:x64-osx: x64-osx/lib/pkgconfig/ode.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_type_timeofday_protos.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/google_cloud_cpp_type_timeofday_protos.pc
dlib:x64-osx: x64-osx/debug/lib/pkgconfig/dlib-1.pc
dlib:x64-osx: x64-osx/lib/pkgconfig/dlib-1.pc
pcl:x64-osx: x64-osx/debug/lib/pkgconfig/pcl_segmentation-1.11.pc
pcl:x64-osx: x64-osx/lib/pkgconfig/pcl_segmentation-1.11.pc
sentencepiece:x64-osx: x64-osx/debug/lib/pkgconfig/sentencepiece.pc
sentencepiece:x64-osx: x64-osx/lib/pkgconfig/sentencepiece.pc
capnproto:x64-osx: x64-osx/debug/lib/pkgconfig/kj-test.pc
capnproto:x64-osx: x64-osx/lib/pkgconfig/kj-test.pc
libevent:x64-osx: x64-osx/debug/lib/pkgconfig/libevent_openssl.pc
libevent:x64-osx: x64-osx/lib/pkgconfig/libevent_openssl.pc
fcl:x64-osx: x64-osx/debug/lib/pkgconfig/fcl.pc
fcl:x64-osx: x64-osx/lib/pkgconfig/fcl.pc
pcl:x64-osx: x64-osx/debug/lib/pkgconfig/pcl_stereo-1.11.pc
pcl:x64-osx: x64-osx/lib/pkgconfig/pcl_stereo-1.11.pc
libkeyfinder:x64-osx: x64-osx/debug/lib/pkgconfig/libkeyfinder.pc
libkeyfinder:x64-osx: x64-osx/lib/pkgconfig/libkeyfinder.pc
collada-dom:x64-osx: x64-osx/debug/lib/pkgconfig/collada-dom.pc
collada-dom:x64-osx: x64-osx/lib/pkgconfig/collada-dom.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_api_label_protos.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/google_cloud_cpp_api_label_protos.pc
libsbml:x64-osx: x64-osx/debug/lib/pkgconfig/libsbml.pc
libsbml:x64-osx: x64-osx/lib/pkgconfig/libsbml.pc
arrow:x64-osx: x64-osx/debug/lib/pkgconfig/arrow-filesystem.pc
arrow:x64-osx: x64-osx/lib/pkgconfig/arrow-filesystem.pc
libfort:x64-osx: x64-osx/debug/lib/pkgconfig/libfort.pc
libfort:x64-osx: x64-osx/lib/pkgconfig/libfort.pc
zziplib:x64-osx: x64-osx/debug/lib/pkgconfig/zziplib.pc
zziplib:x64-osx: x64-osx/lib/pkgconfig/zziplib.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/googleapis_cpp_longrunning_operations_protos.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/googleapis_cpp_longrunning_operations_protos.pc
cutelyst2:x64-osx: x64-osx/debug/lib/pkgconfig/Cutelyst2Qt5UtilsPagination.pc
cutelyst2:x64-osx: x64-osx/lib/pkgconfig/Cutelyst2Qt5UtilsPagination.pc
freeopcua:x64-osx: x64-osx/debug/lib/pkgconfig/libopcuacore.pc
freeopcua:x64-osx: x64-osx/lib/pkgconfig/libopcuacore.pc
bullet3:x64-osx: x64-osx/debug/lib/pkgconfig/bullet.pc
bullet3:x64-osx: x64-osx/lib/pkgconfig/bullet.pc
cpp-redis:x64-osx: x64-osx/debug/lib/pkgconfig/cpp_redis.pc
cpp-redis:x64-osx: x64-osx/lib/pkgconfig/cpp_redis.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/googleapis_cpp_type_quaternion_protos.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/googleapis_cpp_type_quaternion_protos.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_type_postal_address_protos.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/google_cloud_cpp_type_postal_address_protos.pc
libheif:x64-osx: x64-osx/debug/lib/pkgconfig/libheif.pc
libheif:x64-osx: x64-osx/lib/pkgconfig/libheif.pc
zeromq:x64-osx: x64-osx/debug/lib/pkgconfig/libzmq.pc
zeromq:x64-osx: x64-osx/lib/pkgconfig/libzmq.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_iam_v1_options_protos.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/google_cloud_cpp_iam_v1_options_protos.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_pubsub.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/google_cloud_cpp_pubsub.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/googleapis_cpp_api_field_behavior_protos.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/googleapis_cpp_api_field_behavior_protos.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/googleapis_cpp_api_launch_stage_protos.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/googleapis_cpp_api_launch_stage_protos.pc
libraw:x64-osx: x64-osx/debug/lib/pkgconfig/libraw_r.pc
libraw:x64-osx: x64-osx/lib/pkgconfig/libraw_r.pc
coin:x64-osx: x64-osx/debug/lib/pkgconfig/Coin.pc
coin:x64-osx: x64-osx/lib/pkgconfig/Coin.pc
rtaudio:x64-osx: x64-osx/debug/lib/pkgconfig/rtaudio.pc
rtaudio:x64-osx: x64-osx/lib/pkgconfig/rtaudio.pc
p-ranav-csv2:x64-osx: x64-osx/debug/lib/pkgconfig/csv2.pc
p-ranav-csv2:x64-osx: x64-osx/lib/pkgconfig/csv2.pc
pcl:x64-osx: x64-osx/debug/lib/pkgconfig/pcl_keypoints-1.11.pc
pcl:x64-osx: x64-osx/lib/pkgconfig/pcl_keypoints-1.11.pc
jansson:x64-osx: x64-osx/debug/lib/pkgconfig/jansson.pc
jansson:x64-osx: x64-osx/lib/pkgconfig/jansson.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_logging_type_protos.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/google_cloud_cpp_logging_type_protos.pc
ace:x64-osx: x64-osx/debug/lib/pkgconfig/ACE_ETCL_Parser.pc
ace:x64-osx: x64-osx/lib/pkgconfig/ACE_ETCL_Parser.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/googleapis_cpp_rpc_status_protos.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/googleapis_cpp_rpc_status_protos.pc
rpclib:x64-osx: x64-osx/debug/lib/pkgconfig/rpclib.pc
rpclib:x64-osx: x64-osx/lib/pkgconfig/rpclib.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/googleapis_cpp_iam_v1_options_protos.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/googleapis_cpp_iam_v1_options_protos.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_api_metric_protos.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/google_cloud_cpp_api_metric_protos.pc
gtest:x64-osx: x64-osx/debug/lib/pkgconfig/gtest.pc
gtest:x64-osx: x64-osx/lib/pkgconfig/gtest.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_storage_grpc.pc
google-cloud-cpp:x64-osx: x64-osx/lib/pkgconfig/google_cloud_cpp_storage_grpc.pc
graphite2:x64-osx: x64-osx/debug/lib/libgraphite2.la
graphite2:x64-osx: x64-osx/lib/libgraphite2.la
gtest:x64-osx: x64-osx/debug/lib/manual-link/pkgconfig/gmock_main.pc
gtest:x64-osx: x64-osx/lib/manual-link/pkgconfig/gmock_main.pc
gtest:x64-osx: x64-osx/debug/lib/manual-link/pkgconfig/gtest_main.pc
gtest:x64-osx: x64-osx/lib/manual-link/pkgconfig/gtest_main.pc
hdf5:x64-osx: x64-osx/debug/lib/libhdf5.settings
hdf5:x64-osx: x64-osx/lib/libhdf5.settings
kf5globalaccel:x64-osx: x64-osx/debug/lib/systemd/user/plasma-kglobalaccel.service
kf5globalaccel:x64-osx: x64-osx/lib/systemd/user/plasma-kglobalaccel.service
netcdf-c:x64-osx: x64-osx/debug/lib/libnetcdf.settings
netcdf-c:x64-osx: x64-osx/lib/libnetcdf.settings
botan:x64-osx: x64-osx/debug/lib/botan-2.pc
botan:x64-osx: x64-osx/lib/botan-2.pc
libpng:x64-osx: x64-osx/debug/tools/libpng/libpng16-config
libpng:x64-osx: x64-osx/debug/tools/libpng/libpng-config
quill:x64-osx: x64-osx/debug/pkgconfig/quill.pc
openmesh:x64-osx: x64-osx/debug/libdata/pkgconfig/openmesh.pc
hpx:x64-osx: x64-osx/debug/bin/hpxcxx
gdk-pixbuf:x64-osx: x64-osx/debug/gdk-pixbuf-2.0/2.10.0/loaders.cache
glibmm:x64-osx: x64-osx/debug/lib/glibmm-2.68/proc/gmmproc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_grpc_utils.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_longrunning_operations_protos.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/googleapis_cpp_cloud_dialogflow_v2_protos.pc
poppler:x64-osx: x64-osx/debug/lib/pkgconfig/poppler.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_type_dayofweek_protos.pc
libtins:x64-osx: x64-osx/debug/lib/pkgconfig/libtins.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/googleapis_cpp_api_http_protos.pc
pcl:x64-osx: x64-osx/debug/lib/pkgconfig/pcl_2d-1.11.pc
ace:x64-osx: x64-osx/debug/lib/pkgconfig/ACE_ETCL.pc
msgpack:x64-osx: x64-osx/debug/lib/pkgconfig/msgpack.pc
libqrencode:x64-osx: x64-osx/debug/lib/pkgconfig/libqrencode.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/pubsub_client.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/googleapis_cpp_api_annotations_protos.pc
zziplib:x64-osx: x64-osx/debug/lib/pkgconfig/zzipmmapped.pc
mongo-cxx-driver:x64-osx: x64-osx/debug/lib/pkgconfig/libbsoncxx-static.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/googleapis_cpp_type_dayofweek_protos.pc
redis-plus-plus:x64-osx: x64-osx/debug/lib/pkgconfig/redis++.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/googleapis_cpp_api_auth_protos.pc
fluidlite:x64-osx: x64-osx/debug/lib/pkgconfig/fluidlite.pc
pcl:x64-osx: x64-osx/debug/lib/pkgconfig/pcl_features-1.11.pc
sais:x64-osx: x64-osx/debug/lib/pkgconfig/libsais64.pc
bde:x64-osx: x64-osx/debug/lib/pkgconfig/inteldfp.pc
orocos-kdl:x64-osx: x64-osx/debug/lib/pkgconfig/orocos_kdl.pc
libbson:x64-osx: x64-osx/debug/lib/pkgconfig/libbson-1.0.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_rpc_status_protos.pc
freeopcua:x64-osx: x64-osx/debug/lib/pkgconfig/libopcuaserver.pc
mongo-c-driver:x64-osx: x64-osx/debug/lib/pkgconfig/libmongoc-1.0.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/googleapis_cpp_api_metric_protos.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/googleapis_cpp_api_label_protos.pc
cutelyst2:x64-osx: x64-osx/debug/lib/pkgconfig/Cutelyst2Qt5UserAgent.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/googleapis_cpp_type_fraction_protos.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/googleapis_cpp_api_resource_protos.pc
pcl:x64-osx: x64-osx/debug/lib/pkgconfig/pcl_surface-1.11.pc
grpc:x64-osx: x64-osx/debug/lib/pkgconfig/grpc.pc
nlopt:x64-osx: x64-osx/debug/lib/pkgconfig/nlopt.pc
cpuid:x64-osx: x64-osx/debug/lib/pkgconfig/libcpuid.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_bigtable_protos.pc
cutelyst2:x64-osx: x64-osx/debug/lib/pkgconfig/Cutelyst2Qt5WSGI.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_api_launch_stage_protos.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_pubsub_protos.pc
libproxy:x64-osx: x64-osx/debug/lib/pkgconfig/libproxy-1.0.pc
gflags:x64-osx: x64-osx/debug/lib/pkgconfig/gflags.pc
uriparser:x64-osx: x64-osx/debug/lib/pkgconfig/liburiparser.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/googleapis_cpp_type_calendar_period_protos.pc
geogram:x64-osx: x64-osx/debug/lib/pkgconfig/geogram1.pc
arrow:x64-osx: x64-osx/debug/lib/pkgconfig/arrow-csv.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/googleapis_cpp_type_expr_protos.pc
stxxl:x64-osx: x64-osx/debug/lib/pkgconfig/stxxl_debug.pc
plplot:x64-osx: x64-osx/debug/lib/pkgconfig/plplot-c++.pc
realsense2:x64-osx: x64-osx/debug/lib/pkgconfig/realsense2.pc
blitz:x64-osx: x64-osx/debug/lib/pkgconfig/blitz.pc
amqpcpp:x64-osx: x64-osx/debug/lib/pkgconfig/amqpcpp.pc
cmocka:x64-osx: x64-osx/debug/lib/pkgconfig/cmocka.pc
grpc:x64-osx: x64-osx/debug/lib/pkgconfig/gpr.pc
openimageio:x64-osx: x64-osx/debug/lib/pkgconfig/OpenImageIO.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_type_phone_number_protos.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_api_annotations_protos.pc
rbdl:x64-osx: x64-osx/debug/lib/pkgconfig/rbdl.pc
ace:x64-osx: x64-osx/debug/lib/pkgconfig/ACE.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/storage_client.pc
pcl:x64-osx: x64-osx/debug/lib/pkgconfig/pcl_ml-1.11.pc
capnproto:x64-osx: x64-osx/debug/lib/pkgconfig/kj-async.pc
check:x64-osx: x64-osx/debug/lib/pkgconfig/check.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/bigtable_client.pc
arrow:x64-osx: x64-osx/debug/lib/pkgconfig/parquet.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_spanner_protos.pc
mosquitto:x64-osx: x64-osx/debug/lib/pkgconfig/libmosquittopp.pc
portaudio:x64-osx: x64-osx/debug/lib/pkgconfig/portaudio-2.0.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_api_resource_protos.pc
grpc:x64-osx: x64-osx/debug/lib/pkgconfig/grpc++_unsecure.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_logging_protos.pc
capnproto:x64-osx: x64-osx/debug/lib/pkgconfig/capnp-rpc.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_iam_v1_iam_policy_protos.pc
fftw3:x64-osx: x64-osx/debug/lib/pkgconfig/fftw3f.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/googleapis_cpp_type_interval_protos.pc
capnproto:x64-osx: x64-osx/debug/lib/pkgconfig/kj-http.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_cloud_dialogflow_v2_protos.pc
grpc:x64-osx: x64-osx/debug/lib/pkgconfig/grpc++.pc
kf5attica:x64-osx: x64-osx/debug/lib/pkgconfig/libKF5Attica.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_api_client_protos.pc
functions-framework-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/functions_framework_cpp.pc
freeglut:x64-osx: x64-osx/debug/lib/pkgconfig/glut.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/googleapis_cpp_cloud_dialogflow_v2beta1_protos.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_bigquery.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/googleapis.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_api_http_protos.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/googleapis_cpp_rpc_error_details_protos.pc
plplot:x64-osx: x64-osx/debug/lib/pkgconfig/plplot.pc
soxr:x64-osx: x64-osx/debug/lib/pkgconfig/soxr.pc
geographiclib:x64-osx: x64-osx/debug/lib/pkgconfig/geographiclib.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/spanner_client.pc
armadillo:x64-osx: x64-osx/debug/lib/pkgconfig/armadillo.pc
libmt32emu:x64-osx: x64-osx/debug/lib/pkgconfig/mt32emu.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_type_interval_protos.pc
aom:x64-osx: x64-osx/debug/lib/pkgconfig/aom.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/googleapis_cpp_monitoring_protos.pc
json11:x64-osx: x64-osx/debug/lib/pkgconfig/json11.pc
libkml:x64-osx: x64-osx/debug/lib/pkgconfig/libkml.pc
libraw:x64-osx: x64-osx/debug/lib/pkgconfig/libraw.pc
nanomsg:x64-osx: x64-osx/debug/lib/pkgconfig/nanomsg.pc
liblo:x64-osx: x64-osx/debug/lib/pkgconfig/liblo.pc
thrift:x64-osx: x64-osx/debug/lib/pkgconfig/thrift-nb.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/googleapis_cpp_type_localized_text_protos.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/googleapis_cpp_api_monitored_resource_protos.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_api_auth_protos.pc
bde:x64-osx: x64-osx/debug/lib/pkgconfig/bdl.pc
nanoflann:x64-osx: x64-osx/debug/lib/pkgconfig/nanoflann.pc
octomap:x64-osx: x64-osx/debug/lib/pkgconfig/octomap.pc
bde:x64-osx: x64-osx/debug/lib/pkgconfig/bsl.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/googleapis_cpp_bigtable_protos.pc
raylib:x64-osx: x64-osx/debug/lib/pkgconfig/raylib.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_storage_protos.pc
cutelyst2:x64-osx: x64-osx/debug/lib/pkgconfig/Cutelyst2Qt5UtilsValidator.pc
pcl:x64-osx: x64-osx/debug/lib/pkgconfig/pcl_tracking-1.11.pc
msgpack11:x64-osx: x64-osx/debug/lib/pkgconfig/msgpack11.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_api_distribution_protos.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/googleapis_cpp_cloud_speech_protos.pc
poppler:x64-osx: x64-osx/debug/lib/pkgconfig/poppler-cpp.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_iam.pc
freeopcua:x64-osx: x64-osx/debug/lib/pkgconfig/libopcuaclient.pc
flatbuffers:x64-osx: x64-osx/debug/lib/pkgconfig/flatbuffers.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/googleapis_cpp_iam_protos.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/googleapis_cpp_type_date_protos.pc
pcl:x64-osx: x64-osx/debug/lib/pkgconfig/pcl_search-1.11.pc
bde:x64-osx: x64-osx/debug/lib/pkgconfig/bal.pc
libgd:x64-osx: x64-osx/debug/lib/pkgconfig/gdlib.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_type_fraction_protos.pc
pcl:x64-osx: x64-osx/debug/lib/pkgconfig/pcl_recognition-1.11.pc
libconfuse:x64-osx: x64-osx/debug/lib/pkgconfig/libconfuse.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_type_month_protos.pc
libmediainfo:x64-osx: x64-osx/debug/lib/pkgconfig/libmediainfo.pc
hiredis:x64-osx: x64-osx/debug/lib/pkgconfig/hiredis.pc
blosc:x64-osx: x64-osx/debug/lib/pkgconfig/blosc.pc
arrow:x64-osx: x64-osx/debug/lib/pkgconfig/arrow-json.pc
liblemon:x64-osx: x64-osx/debug/lib/pkgconfig/lemon.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_type_localized_text_protos.pc
libgta:x64-osx: x64-osx/debug/lib/pkgconfig/gta.pc
libgit2:x64-osx: x64-osx/debug/lib/pkgconfig/libgit2.pc
libevent:x64-osx: x64-osx/debug/lib/pkgconfig/libevent.pc
hyperscan:x64-osx: x64-osx/debug/lib/pkgconfig/libhs.pc
gtest:x64-osx: x64-osx/debug/lib/pkgconfig/gmock.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_type_color_protos.pc
bde:x64-osx: x64-osx/debug/lib/pkgconfig/bbl.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_type_latlng_protos.pc
cjson:x64-osx: x64-osx/debug/lib/pkgconfig/libcjson.pc
freeopcua:x64-osx: x64-osx/debug/lib/pkgconfig/libopcuaprotocol.pc
bde:x64-osx: x64-osx/debug/lib/pkgconfig/decnumber.pc
capnproto:x64-osx: x64-osx/debug/lib/pkgconfig/kj.pc
cutelyst2:x64-osx: x64-osx/debug/lib/pkgconfig/Cutelyst2Qt5StaticSimple.pc
libevent:x64-osx: x64-osx/debug/lib/pkgconfig/libevent_extra.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_cloud_speech_protos.pc
libmikmod:x64-osx: x64-osx/debug/lib/pkgconfig/libmikmod.pc
ois:x64-osx: x64-osx/debug/lib/pkgconfig/OIS.pc
bde:x64-osx: x64-osx/debug/lib/pkgconfig/btl.pc
brpc:x64-osx: x64-osx/debug/lib/pkgconfig/brpc.pc
fftw3:x64-osx: x64-osx/debug/lib/pkgconfig/fftw3l.pc
arrow:x64-osx: x64-osx/debug/lib/pkgconfig/arrow.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/googleapis_cpp_type_money_protos.pc
bddisasm:x64-osx: x64-osx/debug/lib/pkgconfig/libbddisasm.pc
graphite2:x64-osx: x64-osx/debug/lib/pkgconfig/graphite2.pc
libsrt:x64-osx: x64-osx/debug/lib/pkgconfig/haisrt.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/googleapis_cpp_type_month_protos.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/googleapis_cpp_devtools_cloudtrace_v2_tracing_protos.pc
libftdi1:x64-osx: x64-osx/debug/lib/pkgconfig/libftdipp1.pc
mosquitto:x64-osx: x64-osx/debug/lib/pkgconfig/libmosquitto.pc
cutelyst2:x64-osx: x64-osx/debug/lib/pkgconfig/Cutelyst2Qt5Core.pc
simage:x64-osx: x64-osx/debug/lib/pkgconfig/simage.pc
mongo-c-driver:x64-osx: x64-osx/debug/lib/pkgconfig/libmongoc-ssl-1.0.pc
librdkafka:x64-osx: x64-osx/debug/lib/pkgconfig/rdkafka-static.pc
igraph:x64-osx: x64-osx/debug/lib/pkgconfig/igraph.pc
libcerf:x64-osx: x64-osx/debug/lib/pkgconfig/libcerf.pc
pcl:x64-osx: x64-osx/debug/lib/pkgconfig/pcl_octree-1.11.pc
libtorrent:x64-osx: x64-osx/debug/lib/pkgconfig/libtorrent-rasterbar.pc
libftdi1:x64-osx: x64-osx/debug/lib/pkgconfig/libftdi1.pc
pcl:x64-osx: x64-osx/debug/lib/pkgconfig/pcl_sample_consensus-1.11.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_type_money_protos.pc
muparser:x64-osx: x64-osx/debug/lib/pkgconfig/muparser.pc
libxlsxwriter:x64-osx: x64-osx/debug/lib/pkgconfig/xlsxwriter.pc
iniparser:x64-osx: x64-osx/debug/lib/pkgconfig/iniparser.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/googleapis_cpp_type_color_protos.pc
pcl:x64-osx: x64-osx/debug/lib/pkgconfig/pcl_common-1.11.pc
cutelyst2:x64-osx: x64-osx/debug/lib/pkgconfig/Cutelyst2Qt5StatusMessage.pc
thrift:x64-osx: x64-osx/debug/lib/pkgconfig/thrift.pc
collada-dom:x64-osx: x64-osx/debug/lib/pkgconfig/collada-dom-150.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_cloud_texttospeech_protos.pc
xerces-c:x64-osx: x64-osx/debug/lib/pkgconfig/xerces-c.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_monitoring_protos.pc
collada-dom:x64-osx: x64-osx/debug/lib/pkgconfig/collada-dom-141.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_devtools_cloudtrace_v2_tracing_protos.pc
orocos-kdl:x64-osx: x64-osx/debug/lib/pkgconfig/orocos-kdl.pc
libevent:x64-osx: x64-osx/debug/lib/pkgconfig/libevent_pthreads.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_rpc_error_details_protos.pc
cutelyst2:x64-osx: x64-osx/debug/lib/pkgconfig/Cutelyst2Qt5Session.pc
capnproto:x64-osx: x64-osx/debug/lib/pkgconfig/capnp-json.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_devtools_cloudtrace_v2_trace_protos.pc
cutelyst2:x64-osx: x64-osx/debug/lib/pkgconfig/Cutelyst2Qt5Authentication.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/googleapis_cpp_spanner_protos.pc
superlu:x64-osx: x64-osx/debug/lib/pkgconfig/superlu.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_type_date_protos.pc
usrsctp:x64-osx: x64-osx/debug/lib/pkgconfig/usrsctp.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/googleapis_cpp_logging_protos.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_storage.pc
librdkafka:x64-osx: x64-osx/debug/lib/pkgconfig/rdkafka++-static.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/googleapis_cpp_type_latlng_protos.pc
libevent:x64-osx: x64-osx/debug/lib/pkgconfig/libevent_core.pc
pcl:x64-osx: x64-osx/debug/lib/pkgconfig/pcl_kdtree-1.11.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_spanner.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_bigtable.pc
zziplib:x64-osx: x64-osx/debug/lib/pkgconfig/zzipfseeko.pc
pcl:x64-osx: x64-osx/debug/lib/pkgconfig/pcl_registration-1.11.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_type_calendar_period_protos.pc
sdformat6:x64-osx: x64-osx/debug/lib/pkgconfig/sdformat.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/googleapis_cpp_type_datetime_protos.pc
libwebsockets:x64-osx: x64-osx/debug/lib/pkgconfig/libwebsockets_static.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/googleapis_cpp_devtools_cloudtrace_v2_trace_protos.pc
ppconsul:x64-osx: x64-osx/debug/lib/pkgconfig/ppconsul.pc
pcl:x64-osx: x64-osx/debug/lib/pkgconfig/pcl_io-1.11.pc
plplot:x64-osx: x64-osx/debug/lib/pkgconfig/plplot-fortran.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/googleapis_cpp_api_distribution_protos.pc
libwebsockets:x64-osx: x64-osx/debug/lib/pkgconfig/libwebsockets.pc
edlib:x64-osx: x64-osx/debug/lib/pkgconfig/edlib-1.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/googleapis_cpp_cloud_texttospeech_protos.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_api_monitored_resource_protos.pc
curlpp:x64-osx: x64-osx/debug/lib/pkgconfig/curlpp.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/googleapis_cpp_iam_v1_policy_protos.pc
flann:x64-osx: x64-osx/debug/lib/pkgconfig/flann.pc
grpc:x64-osx: x64-osx/debug/lib/pkgconfig/grpc_unsecure.pc
capnproto:x64-osx: x64-osx/debug/lib/pkgconfig/capnp.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_cloud_dialogflow_v2beta1_protos.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_type_datetime_protos.pc
cutelyst2:x64-osx: x64-osx/debug/lib/pkgconfig/Cutelyst2Qt5UtilsSql.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_cloud_bigquery_protos.pc
libsrt:x64-osx: x64-osx/debug/lib/pkgconfig/srt.pc
cutelyst2:x64-osx: x64-osx/debug/lib/pkgconfig/Cutelyst2Qt5ViewJSON.pc
thrift:x64-osx: x64-osx/debug/lib/pkgconfig/thrift-z.pc
qca:x64-osx: x64-osx/debug/lib/pkgconfig/qca2.pc
libevhtp:x64-osx: x64-osx/debug/lib/pkgconfig/evhtp.pc
charls:x64-osx: x64-osx/debug/lib/pkgconfig/charls.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_iam_v1_policy_protos.pc
bde:x64-osx: x64-osx/debug/lib/pkgconfig/pcre2.pc
fftw3:x64-osx: x64-osx/debug/lib/pkgconfig/fftw3.pc
bcg729:x64-osx: x64-osx/debug/lib/pkgconfig/libbcg729.pc
oniguruma:x64-osx: x64-osx/debug/lib/pkgconfig/oniguruma.pc
blitz:x64-osx: x64-osx/debug/lib/pkgconfig/blitz-static.pc
graphqlparser:x64-osx: x64-osx/debug/lib/pkgconfig/libgraphqlparser.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/googleapis_cpp_storage_protos.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/googleapis_cpp_cloud_bigquery_protos.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_type_quaternion_protos.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/googleapis_cpp_type_phone_number_protos.pc
mongo-cxx-driver:x64-osx: x64-osx/debug/lib/pkgconfig/libmongocxx-static.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/googleapis_cpp_type_postal_address_protos.pc
ccd:x64-osx: x64-osx/debug/lib/pkgconfig/ccd.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_type_expr_protos.pc
cutelyst2:x64-osx: x64-osx/debug/lib/pkgconfig/Cutelyst2Qt5UtilsLangSelect.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_api_field_behavior_protos.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/googleapis_cpp_pubsub_protos.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/googleapis_cpp_api_client_protos.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/googleapis_cpp_type_timeofday_protos.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_iam_protos.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/googleapis_cpp_iam_v1_iam_policy_protos.pc
pcl:x64-osx: x64-osx/debug/lib/pkgconfig/pcl_filters-1.11.pc
opencc:x64-osx: x64-osx/debug/lib/pkgconfig/opencc.pc
sais:x64-osx: x64-osx/debug/lib/pkgconfig/libsais.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_common.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/googleapis_cpp_logging_type_protos.pc
folly:x64-osx: x64-osx/debug/lib/pkgconfig/libfolly.pc
pcl:x64-osx: x64-osx/debug/lib/pkgconfig/pcl_geometry-1.11.pc
clblast:x64-osx: x64-osx/debug/lib/pkgconfig/clblast.pc
itpp:x64-osx: x64-osx/debug/lib/pkgconfig/itpp.pc
ode:x64-osx: x64-osx/debug/lib/pkgconfig/ode.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_type_timeofday_protos.pc
dlib:x64-osx: x64-osx/debug/lib/pkgconfig/dlib-1.pc
pcl:x64-osx: x64-osx/debug/lib/pkgconfig/pcl_segmentation-1.11.pc
sentencepiece:x64-osx: x64-osx/debug/lib/pkgconfig/sentencepiece.pc
capnproto:x64-osx: x64-osx/debug/lib/pkgconfig/kj-test.pc
libevent:x64-osx: x64-osx/debug/lib/pkgconfig/libevent_openssl.pc
fcl:x64-osx: x64-osx/debug/lib/pkgconfig/fcl.pc
pcl:x64-osx: x64-osx/debug/lib/pkgconfig/pcl_stereo-1.11.pc
libkeyfinder:x64-osx: x64-osx/debug/lib/pkgconfig/libkeyfinder.pc
collada-dom:x64-osx: x64-osx/debug/lib/pkgconfig/collada-dom.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_api_label_protos.pc
libsbml:x64-osx: x64-osx/debug/lib/pkgconfig/libsbml.pc
arrow:x64-osx: x64-osx/debug/lib/pkgconfig/arrow-filesystem.pc
libfort:x64-osx: x64-osx/debug/lib/pkgconfig/libfort.pc
zziplib:x64-osx: x64-osx/debug/lib/pkgconfig/zziplib.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/googleapis_cpp_longrunning_operations_protos.pc
cutelyst2:x64-osx: x64-osx/debug/lib/pkgconfig/Cutelyst2Qt5UtilsPagination.pc
freeopcua:x64-osx: x64-osx/debug/lib/pkgconfig/libopcuacore.pc
bullet3:x64-osx: x64-osx/debug/lib/pkgconfig/bullet.pc
cpp-redis:x64-osx: x64-osx/debug/lib/pkgconfig/cpp_redis.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/googleapis_cpp_type_quaternion_protos.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_type_postal_address_protos.pc
libheif:x64-osx: x64-osx/debug/lib/pkgconfig/libheif.pc
zeromq:x64-osx: x64-osx/debug/lib/pkgconfig/libzmq.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_iam_v1_options_protos.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_pubsub.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/googleapis_cpp_api_field_behavior_protos.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/googleapis_cpp_api_launch_stage_protos.pc
libraw:x64-osx: x64-osx/debug/lib/pkgconfig/libraw_r.pc
coin:x64-osx: x64-osx/debug/lib/pkgconfig/Coin.pc
rtaudio:x64-osx: x64-osx/debug/lib/pkgconfig/rtaudio.pc
p-ranav-csv2:x64-osx: x64-osx/debug/lib/pkgconfig/csv2.pc
pcl:x64-osx: x64-osx/debug/lib/pkgconfig/pcl_keypoints-1.11.pc
jansson:x64-osx: x64-osx/debug/lib/pkgconfig/jansson.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_logging_type_protos.pc
ace:x64-osx: x64-osx/debug/lib/pkgconfig/ACE_ETCL_Parser.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/googleapis_cpp_rpc_status_protos.pc
rpclib:x64-osx: x64-osx/debug/lib/pkgconfig/rpclib.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/googleapis_cpp_iam_v1_options_protos.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_api_metric_protos.pc
gtest:x64-osx: x64-osx/debug/lib/pkgconfig/gtest.pc
google-cloud-cpp:x64-osx: x64-osx/debug/lib/pkgconfig/google_cloud_cpp_storage_grpc.pc
graphite2:x64-osx: x64-osx/debug/lib/libgraphite2.la
gtest:x64-osx: x64-osx/debug/lib/manual-link/pkgconfig/gmock_main.pc
gtest:x64-osx: x64-osx/debug/lib/manual-link/pkgconfig/gtest_main.pc
hdf5:x64-osx: x64-osx/debug/lib/libhdf5.settings
kf5globalaccel:x64-osx: x64-osx/debug/lib/systemd/user/plasma-kglobalaccel.service
netcdf-c:x64-osx: x64-osx/debug/lib/libnetcdf.settings
botan:x64-osx: x64-osx/debug/lib/botan-2.pc
sol2:x64-osx: x64-osx/share/pkgconfig/sol2.pc
cli:x64-osx: x64-osx/share/pkgconfig/cli.pc
polyclipping:x64-osx: x64-osx/share/pkgconfig/polyclipping.pc
easyloggingpp:x64-osx: x64-osx/share/pkgconfig/easyloggingpp.pc
cppcodec:x64-osx: x64-osx/share/pkgconfig/cppcodec-1.pc
aricpp:x64-osx: x64-osx/share/pkgconfig/aricpp.pc
glib:x64-osx: x64-osx/share/gdb/auto-load/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/glib_x64-osx/lib/libgobject-2.0.so.0.6600.4-gdb.py
glib:x64-osx: x64-osx/share/gdb/auto-load/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/glib_x64-osx/lib/libglib-2.0.so.0.6600.4-gdb.py
shapelib:x64-osx: x64-osx/share/shp/export_shp.cmake
plplot:x64-osx: x64-osx/share/plplot/examples/cmake/modules/plplot_configure.cmake
plplot:x64-osx: x64-osx/share/plplot/examples/plplot-test-interactive.sh
plplot:x64-osx: x64-osx/share/plplot/examples/fortran/Makefile
ftgl:x64-osx: x64-osx/share/ftgl/FTGL/FTGLConfig.cmake
libavif:x64-osx: x64-osx/share/libavif/libavif-config.cmake
offscale-libetcd-cpp:x64-osx: x64-osx/share/etcdcpp/etcdcppConfig.cmake
gdk-pixbuf:x64-osx: x64-osx/share/thumbnailers/gdk-pixbuf-thumbnailer.thumbnailer
kf5globalaccel:x64-osx: x64-osx/share/dbus-1/services/org.kde.kglobalaccel.service
cgns:x64-osx: x64-osx/share/cgns/cgnsBuild.defs
bcg729:x64-osx: x64-osx/share/Bcg729/cmake/Bcg729Config.cmake
ismrmrd:x64-osx: x64-osx/share/ismrmrd/cmake/ISMRMRDConfig.cmake
simage:x64-osx: x64-osx/share/Coin/conf/simage-default.cfg
mongo-c-driver:x64-osx: x64-osx/share/mongo-c-driver/uninstall.sh
ignition-msgs1:x64-osx: x64-osx/share/ignition/msgs1.yaml
ignition-transport4:x64-osx: x64-osx/share/ignition/transport4.yaml
sdformat6:x64-osx: x64-osx/share/ignition/sdformat6.yaml
log4cpp:x64-osx: x64-osx/share/log4cpp/orocos-log4cpp.pc
qca:x64-osx: x64-osx/share/qca/cmake/QcaTargets-debug.cmake
protopuf:x64-osx: x64-osx/share/protopuf/protopufConfig.cmake
ace:x64-osx: x64-osx/share/ace/bin/MakeProjectCreator/config/MPC.cfg
ace:x64-osx: x64-osx/share/ace/include/makeinclude/wrapper_macros.GNU
ace:x64-osx: x64-osx/share/ace/ace-devel.sh
minitrace:x64-osx: x64-osx/share/minitrace/minitraceConfig.cmake
minitrace:x64-osx: x64-osx/share/minitrace/minitraceConfig-release.cmake
```
</details>
Or in 
<details><summary>`... | sed 's/:.*$//' | sort | uniq -c | sort `</summary>

```
   1 aricpp
   1 cgns
   1 cli
   1 cppcodec
   1 dartsim
   1 dcmtk
   1 easyloggingpp
   1 ftgl
   1 gdcm
   1 ignition-msgs1
   1 ignition-transport4
   1 ismrmrd
   1 kf5windowsystem
   1 libavif
   1 libpmemobj-cpp
   1 log4cpp
   1 offscale-libetcd-cpp
   1 polyclipping
   1 protopuf
   1 qpid-proton
   1 shapelib
   1 sol2
   1 sparsehash
   1 symengine
   2 fontconfig
   2 lager
   2 minitrace
   2 stxxl
   3 amqpcpp
   3 aom
   3 armadillo
   3 bddisasm
   3 blosc
   3 brpc
   3 bullet3
   3 ccd
   3 charls
   3 check
   3 cjson
   3 clblast
   3 cmocka
   3 coin
   3 cpp-redis
   3 cpuid
   3 curlpp
   3 dlib
   3 edlib
   3 fcl
   3 flann
   3 flatbuffers
   3 fluidlite
   3 folly
   3 freeglut
   3 functions-framework-cpp
   3 geogram
   3 geographiclib
   3 gflags
   3 glib
   3 glibmm
   3 graphqlparser
   3 hdf5
   3 hiredis
   3 hpx
   3 hyperscan
   3 igraph
   3 iniparser
   3 itpp
   3 jansson
   3 json11
   3 kf5attica
   3 libbson
   3 libcerf
   3 libconfuse
   3 libevhtp
   3 libfort
   3 libgd
   3 libgit2
   3 libgta
   3 libheif
   3 libkeyfinder
   3 libkml
   3 liblemon
   3 liblo
   3 libmediainfo
   3 libmikmod
   3 libmt32emu
   3 libproxy
   3 libqrencode
   3 libsbml
   3 libtins
   3 libtorrent
   3 libxlsxwriter
   3 msgpack
   3 msgpack11
   3 muparser
   3 nanoflann
   3 nanomsg
   3 netcdf-c
   3 nlopt
   3 octomap
   3 ode
   3 ois
   3 oniguruma
   3 opencc
   3 openimageio
   3 openmesh
   3 p-ranav-csv2
   3 portaudio
   3 ppconsul
   3 quill
   3 raylib
   3 rbdl
   3 realsense2
   3 redis-plus-plus
   3 rpclib
   3 rtaudio
   3 sentencepiece
   3 soxr
   3 superlu
   3 uriparser
   3 usrsctp
   3 xerces-c
   3 zeromq
   4 bcg729
   4 botan
   4 gdk-pixbuf
   4 kf5globalaccel
   4 qca
   4 simage
   5 sdformat6
   6 blitz
   6 graphite2
   6 libftdi1
   6 libpng
   6 libraw
   6 librdkafka
   6 libsrt
   6 mongo-cxx-driver
   6 mosquitto
   6 orocos-kdl
   6 poppler
   6 sais
   7 libwebsockets
   7 mongo-c-driver
   9 collada-dom
   9 fftw3
   9 thrift
   9 zziplib
  12 ace
  12 freeopcua
  12 gtest
  12 plplot
  15 arrow
  15 grpc
  15 libevent
  21 capnproto
  24 bde
  36 cutelyst2
  54 pcl
 330 google-cloud-cpp
```
</details>

Reply:
Alright, then I think the correct check (for me, anyways) is:

```
if (filename_in_set(Makefile, CMakeLists.txt) check;
if (has_extension)
{
  if(extension_in_set(hpp, hxx, py, sh, h, cmake, pc)) check;
}
else
{
  if(file_has_shebang) check;
}
```

this will catch almost everything we care about, and won't result in false positives.

Additionally, we should check for `vcpkg_installed_dir`, and for these with both backslashes and forward slashes on Windows.